<!--?xml version="1.0" encoding="UTF-8"?-->
<!DOCTYPE html>
<html class="modernizr-svg modernizr-localstorage modernizr-inlinesvg modernizr-svgclippaths modernizr-flexbox js" dir="ltr" version="HTML+RDFa+MathML 1.1" xmlns:dc="http://purl.org/dc/terms/" xmlns:foaf="http://xmlns.com/foaf/0.1/" xmlns:og="http://ogp.me/ns#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:sioc="http://rdfs.org/sioc/ns#" xmlns:sioct="http://rdfs.org/sioc/types#" xmlns:skos="http://www.w3.org/2004/02/skos/core#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:mml="http://www.w3.org/1998/Math/MathML" lang="en"><head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#"><script src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/analytics.js" async=""></script><script async="" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/cbgapi.loaded_1"></script><script async="" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/cbgapi.loaded_0"></script>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<link rel="shortcut icon" href="http://physiolgenomics.physiology.org/sites/default/files/favicon_0.ico" type="image/vnd.microsoft.icon">
<meta name="issue_cover_image" content="http://physiolgenomics.physiology.org/sites/default/files/1.cover_1.gif">
<meta name="HW.identifier" content="/physiolgenomics/34/1/127.atom">
<meta name="HW.pisa" content="physiolgenomics;34/1/127">
<meta name="DC.Format" content="text/html">
<meta name="DC.Language" content="en">
<meta name="DC.Title" content="Validating the genomic signature of pediatric septic shock">
<meta name="DC.Identifier" content="10.1152/physiolgenomics.00025.2008">
<meta name="DC.Date" content="2008-06-01">
<meta name="DC.Publisher" content="American Physiological Society">
<meta name="DC.Rights" content="Copyright Â© 2008 the American Physiological Society">
<meta name="DC.Contributor" content="Natalie Cvijanovich">
<meta name="DC.Contributor" content="Thomas P. Shanley">
<meta name="DC.Contributor" content="Richard Lin">
<meta name="DC.Contributor" content="Geoffrey L. Allen">
<meta name="DC.Contributor" content="Neal J. Thomas">
<meta name="DC.Contributor" content="Paul Checchia">
<meta name="DC.Contributor" content="Nick Anas">
<meta name="DC.Contributor" content="Robert J. Freishtat">
<meta name="DC.Contributor" content="Marie Monaco">
<meta name="DC.Contributor" content="Kelli Odoms">
<meta name="DC.Contributor" content="Bhuvaneswari Sakthivel">
<meta name="DC.Contributor" content="Hector R. Wong">
<meta name="DC.Contributor" content="for the Genomics of Pediatric SIRS/Septic Shock Investigators">
<meta name="og:title" content="Validating the genomic signature of pediatric septic shock">
<meta name="og:url" content="http://physiolgenomics.physiology.org/content/34/1/127">
<meta name="og:site_name" content="Physiological Genomics">
<meta name="og:image" content="http://physiolgenomics.physiology.org/sites/default/files/1.cover_1.gif">
<meta name="og:type" content="article">
<meta name="article:published_time" content="2008-06-01">
<meta name="article:section" content="Research Article">
<meta name="citation_journal_title" content="Physiological Genomics">
<meta name="citation_publisher" content="American Physiological Society">
<meta name="citation_title" content="Validating the genomic signature of pediatric septic shock">
<meta name="citation_publication_date" content="2008/06/01">
<meta name="citation_mjid" content="physiolgenomics;34/1/127">
<meta name="citation_id" content="34/1/127">
<meta name="citation_public_url" content="http://physiolgenomics.physiology.org/content/34/1/127">
<meta name="citation_abstract_html_url" content="http://physiolgenomics.physiology.org/content/34/1/127.abstract">
<meta name="citation_full_html_url" content="http://physiolgenomics.physiology.org/content/34/1/127.full">
<meta name="citation_pdf_url" content="http://physiolgenomics.physiology.org/content/physiolgenomics/34/1/127.full.pdf">
<meta name="citation_issn" content="1094-8341">
<meta name="citation_issn" content="1531-2267">
<meta name="citation_doi" content="10.1152/physiolgenomics.00025.2008">
<meta name="citation_pmid" content="18460642">
<meta name="citation_volume" content="34">
<meta name="citation_issue" content="1">
<meta name="citation_section" content="Research Article">
<meta name="citation_firstpage" content="127">
<meta name="citation_lastpage" content="134">
<meta name="citation_author" content="Natalie Cvijanovich">
<meta name="citation_author" content="Thomas P. Shanley">
<meta name="citation_author" content="Richard Lin">
<meta name="citation_author" content="Geoffrey L. Allen">
<meta name="citation_author" content="Neal J. Thomas">
<meta name="citation_author" content="Paul Checchia">
<meta name="citation_author" content="Nick Anas">
<meta name="citation_author" content="Robert J. Freishtat">
<meta name="citation_author" content="Marie Monaco">
<meta name="citation_author" content="Kelli Odoms">
<meta name="citation_author" content="Bhuvaneswari Sakthivel">
<meta name="citation_author" content="Hector R. Wong">
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_title=Nat Rev Genet;citation_title=Microarray data analysis: from disarray to consolidation and consensus.;citation_pages=55-65;citation_volume=7;citation_year=2006;citation_issue=1;citation_pmid=16369572;citation_doi=10.1038/nrg1749">
<meta name="citation_reference" content="citation_journal_title=Shock;citation_pages=S27;citation_volume=6;citation_year=1996;citation_pmid=8828095;citation_doi=10.1097/00024382-199610001-00007">
<meta name="citation_reference" content="citation_journal_title=Applied bioinformatics;citation_journal_title=Appl Bioinformatics;citation_title=Support vector machine applications in bioinformatics.;citation_pages=67-77;citation_volume=2;citation_year=2003;citation_issue=2;citation_pmid=15130823">
<meta name="citation_reference" content="citation_journal_title=Nature; Physical Science (London);citation_journal_title=Nature; Physical Science (London);citation_title=A network-based analysis of systemic inflammation in humans.;citation_pages=1032-1037;citation_volume=437;citation_year=2005;citation_issue=7061;citation_pmid=16136080;citation_doi=10.1038/nature03985">
<meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci USA;citation_volume=102;citation_year=2005;citation_pmid=15781863;citation_doi=10.1073/pnas.0409768102">
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_title=Genome Biol;citation_title=DAVID: Database for Annotation, Visualization, and Integrated Discovery.;citation_pages=P3-P3;citation_volume=4;citation_year=2003;citation_issue=5;citation_pmid=12734009;citation_doi=10.1186/gb-2003-4-5-p3">
<meta name="citation_reference" content="citation_journal_title=Pediatric critical care medicine : a journal of the Society of Critical Care Medicine and the World Federation of Pediatric Intensive and Critical Care Societies;citation_journal_title=Pediatr Crit Care Med;citation_title=International pediatric sepsis consensus conference: definitions for sepsis and organ dysfunction in pediatrics.;citation_pages=2-8;citation_volume=6;citation_year=2005;citation_issue=1;citation_pmid=15636651;citation_doi=10.1097/01.PCC.0000149131.72248.E6">
<meta name="citation_reference" content="citation_journal_title=New England Journal of Medicine;citation_journal_title=NEJM;citation_title=The Pathophysiology and Treatment of Sepsis;citation_pages=138-150;citation_volume=348;citation_year=2003;citation_issue=2;citation_issn=0028-4793;citation_pmid=12519925;citation_doi=10.1056/NEJMra021333">
<meta name="citation_reference" content="citation_journal_title=Biostatistics;citation_journal_title=Biostatistics;citation_title=Exploration, normalization, and summaries of high density oligonucleotide array probe level data;citation_pages=249-264;citation_volume=4;citation_year=2003;citation_issue=2;citation_issn=1465-4644;citation_pmid=12925520;citation_doi=10.1093/biostatistics/4.2.249">
<meta name="citation_reference" content="citation_journal_title=Trends in genetics : TIG;citation_journal_title=Trends Genet;citation_title=Statistical issues with microarrays: processing and analysis.;citation_pages=265-271;citation_volume=18;citation_year=2002;citation_issue=5;citation_pmid=12047952;citation_doi=10.1016/S0168-9525(02)02665-3">
<meta name="citation_reference" content="citation_journal_title=The Journal of pediatrics;citation_journal_title=J Pediatr;citation_title=The Pediatric Risk of Mortality III--Acute Physiology Score (PRISM III-APS): a method of assessing physiologic instability for pediatric intensive care unit patients.;citation_pages=575-581;citation_volume=131;citation_year=1997;citation_issue=4;citation_pmid=9386662;citation_doi=10.1016/S0022-3476(97)70065-9">
<meta name="citation_reference" content="citation_journal_title=Trends in immunology;citation_journal_title=Trends Immunol;citation_title=Zinc homeostasis and immunity.;citation_pages=1-4;citation_volume=28;citation_year=2007;citation_issue=1;citation_pmid=17126599;citation_doi=10.1016/j.it.2006.11.005">
<meta name="citation_reference" content="citation_journal_title=Molecular medicine (Cambridge, Mass.);citation_journal_title=Mol Med;citation_title=Genome-level longitudinal expression of signaling pathways and gene networks in pediatric septic shock.;citation_pages=495-508;citation_volume=13;citation_year=2007;citation_issue=9-10;citation_pmid=17932561">
<meta name="citation_reference" content="citation_pages=1474;citation_year=2006">
<meta name="citation_reference" content="citation_journal_title=Critical care clinics;citation_journal_title=Crit Care Clin;citation_title=Molecular genetics in the pediatric intensive care unit.;citation_pages=577-594;citation_volume=19;citation_year=2003;citation_issue=3;citation_pmid=12848322;citation_doi=10.1016/S0749-0704(03)00004-6">
<meta name="citation_reference" content="citation_journal_title=Pediatric critical care medicine : a journal of the Society of Critical Care Medicine and the World Federation of Pediatric Intensive and Critical Care Societies;citation_journal_title=Pediatr Crit Care Med;citation_title=Scope and epidemiology of pediatric sepsis.;citation_pages=S3-S5;citation_volume=6;citation_year=2005;citation_issue=3 Suppl;citation_pmid=15857554;citation_doi=10.1097/01.PCC.0000161289.22464.C3">
<meta name="citation_reference" content="citation_journal_title=Omics : a journal of integrative biology;citation_journal_title=OMICS;citation_title=Microarray data quality - review of current developments.;citation_pages=1-13;citation_volume=11;citation_year=2007;citation_issue=1;citation_pmid=17411392;citation_doi=10.1089/omi.2006.0001">
<meta name="citation_reference" content="citation_journal_title=Pharmacogenomics;citation_journal_title=Pharmacogenomics;citation_title=Pediatric septic shock treatment: new clues from genomic profiling.;citation_pages=1287-1290;citation_volume=8;citation_year=2007;citation_issue=10;citation_pmid=17979501;citation_doi=10.2217/14622416.8.10.1287">
<meta name="citation_reference" content="citation_journal_title=Physiol Genomics;citation_volume=30;citation_year=2007;citation_pmid=17374846;citation_doi=10.1152/physiolgenomics.00024.2007">
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics 7;citation_pages=S11;citation_year=2006">
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
<meta name="generator" content="Drupal 7 (http://drupal.org)">
<link rel="canonical" href="http://physiolgenomics.physiology.org/content/34/1/127">
<link rel="shortlink" href="http://physiolgenomics.physiology.org/node/4156">
  <title>ARTICLES | Physiological Genomics</title>  
  <link type="text/css" rel="stylesheet" href="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/css_ZUFNblKMbVqoX9vOlNL81Mn-EOIKxHv1G9dUXjQvSxs.css" media="all">
<link type="text/css" rel="stylesheet" href="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/css_feghaQHdVM-ayDu-sU38yviW3WFXuh1hh0PXGI1l4zc.css" media="all">
<link type="text/css" rel="stylesheet" href="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/css_aZPkQ9mEGYU1CAYRZQPQnYzzBvF_nbuZdySUwlUvbDE.css" media="all">
<link type="text/css" rel="stylesheet" href="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/css_p09swIeqkhKVZjPrEgBiGr8woazsRcyKKuiLvOE1cHw.css" media="all">
<link type="text/css" rel="stylesheet" href="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/css_C66gTr00v8C78pnyy16Gmr-Q8G3lCnOnByxK8F8i-nc_highwire.css" media="all">
<link type="text/css" rel="stylesheet" href="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/css_i8WEgFWbt70geUCfzCwebEAGUHcaBpgqOmSuUJT2vJc.css" media="all">
<style type="text/css" media="all">
<!--/*--><![CDATA[/*><!--*/
.user-menu-dropdown-logo > .menubar-nav > .nice-menu > .menuparent a:first-child{background:url('http://physiolgenomics.physiology.org/sites/default/files/umbrella_menu_logo.png') transparent no-repeat left center;padding-left:72px;}

/*]]>*/-->
</style>
<link type="text/css" rel="stylesheet" href="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/css_Py8Mu6nwrG8vi92wY8JuYpu3EajacLcA7R6EQENx654.css" media="all">

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="http://physiolgenomics.physiology.org/sites/default/files/cdn/css/http/css_h6bj8_X8cU-XjbPX4TNPt0JYXFqUsObkSpuQOJVofyI.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/css_pcO3UeLVTwPeN6ktxiglVp42wzneGAifrHvk8z0x9GI.css" media="all">

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="http://physiolgenomics.physiology.org/sites/default/files/cdn/css/http/css_37F89u3lBDvaXVCA3-Tdk0c421Kt5FlO2IPkqEw8ICU.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/css_ztArrc3CJrjQtnO33Y-cxvEHee7IqE1Kz5XHBb7wSHE.css" media="all">
<!--<![endif]-->
  <script src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/analytics.js" async=""></script><script type="text/javascript" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/modernizr.js"></script>
<script type="text/javascript" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/js_keuIaa3l4Bij_nb95oCru9ZBPwVtKBqnjy8KO0MwZBI.js"></script>
<script type="text/javascript" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/js_ojk50nw4S5R2KRIaqHRkitSPDXkEthrFBG_z2pXg3_A.js"></script>
<script type="text/javascript" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/js_Sh2UgvOvuSU9-K8qwk3-2k4jaFfLVCvAG5OTqQOumXM.js"></script>
<script type="text/javascript" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/js_cqmAXMVJjS6tEa-jkDlkPEPPHJYsx9fo9mPCjd4Rvfw.js"></script>
<script type="text/javascript" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/highwire_at_symbol.js"></script>
<script type="text/javascript" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/highwire_article_reference_popup.js"></script>
<script type="text/javascript" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/js_u0IbvekeaenUcV_bklcmCUKsx_M-q8ECyD1eNJVx0bc.js"></script>
<script type="text/javascript" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/google_service.js"></script>
<script type="text/javascript" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/js_xk94TSfUkRosgD_bumx6L8-L4Tyj7W3mQ7_SS8_ZXPQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-2924550-1", {"cookieDomain":"auto"});ga("set", "page", location.pathname + location.search + location.hash);ga("send", "pageview");ga('create', 'UA-189672-30', 'auto', {'name': 'hwTracker'});
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
GS_googleAddAdSenseService("ca-pub-3836795673102082");
GS_googleEnableAllServices();

//--><!]]>
</script><script src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/google_ads.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
GA_googleAddSlot("ca-pub-3836795673102082", "physiolgenomics_tower_right_160x600");
GA_googleAddSlot("ca-pub-3836795673102082", "physiolgenomics_leaderboard_top_728x90");

//--><!]]>
</script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
GA_googleFetchAds();

//--><!]]>
</script>
<script type="text/javascript" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/js_MSXke3BU2A0hzWk2_lJeS0J0ZGXQAEYmgsfM2J1CqQ4.js"></script>
<script type="text/javascript" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/js_uwrfBOJCFVq42muTMjrq2bD-RUh8NNq_fZRTb6dsNb4.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"aroQtu2FjHEW5DUO1FZa1JArDtrV8LN2gG3pd4xfJww","js":{"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.js":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/jquery.ui.core.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/jquery.ui.widget.js":1,"sites\/all\/libraries\/cluetip\/jquery.cluetip.js":1,"sites\/all\/libraries\/cluetip\/lib\/jquery.bgiframe.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/jquery.ui.button.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/jquery.ui.mouse.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/jquery.ui.draggable.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/jquery.ui.position.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/jquery.ui.resizable.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/jquery.ui.dialog.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/jquery.ui.accordion.js":1,"sites\/all\/libraries\/jquery.bgiframe\/jquery.bgiframe.js":1,"sites\/all\/libraries\/jquery.hoverIntent\/jquery.hoverIntent.js":1,"sites\/all\/libraries\/superfish\/superfish.js":1,"sites\/all\/modules\/contrib\/nice_menus\/js\/nice_menus.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/jquery.zclip.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/twitter_button.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/facebook_like.js":1,"sites\/all\/modules\/contrib\/service_links\/js\/google_plus_one.js":1,"http:\/\/partner.googleadservices.com\/gampad\/google_service.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"0":1,"1":1,"2":1,"3":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"4":1,"5":1,"6":1,"7":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1},"css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.accordion.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"sites\/all\/modules\/shared\/jnl_aps_styles\/aps-shared-styles.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/contrib\/google_admanager\/google_admanager.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/libraries\/cluetip\/jquery.cluetip.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_citation\/css\/jcore_standard.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_citing_links.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/userbar-dropdown-menu.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"0":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/physiolgenomics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"4156","apath":"\/physiolgenomics\/34\/1\/127.atom","pisa":"physiolgenomics;34\/1\/127","markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"physiolgenomics;34\/1\/127"},{"requested":"long","variant":"full-text","view":"full","pisa":"physiolgenomics;34\/1\/127"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"physiolgenomics;34\/1\/127"}],"ac":{"physiolgenomics;34\/1\/127":{"access":{"full":true},"pisa_id":"physiolgenomics;34\/1\/127","atom_uri":"","jcode":"physiolgenomics"}},"modal_window_width":"560","share_modal_width":"560","moviepath":"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/ZeroClipboard.swf"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/34\/1\/127.long":true},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"ws_fl":{"width":100,"height":21},"ws_gpo":{"size":"","annotation":"","lang":"","callback":"","width":300},"color":{"logo":"http:\/\/physiolgenomics.physiology.org\/sites\/default\/files\/Journal%20Logo%20with%20%C2%AE_sm_Avenir_PG-2015.png"},"nice_menus_options":{"delay":800,"speed":"fast"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"narrow":"all and (min-width: 740px) and (min-device-width: 740px), (max-device-width: 800px) and (min-width: 740px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackUrlFragments":1},"omega":{"layouts":{"primary":"normal","order":["normal","wide"],"queries":{"normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
  <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
<script src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/osd.js" type="text/javascript"></script><script src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/button.js" async="" charset="utf-8" type="text/javascript"></script></head>
<body class="html not-front not-logged-in page-node page-node- page-node-4156 node-type-highwire-article context-content hw-default-jcode-physiolgenomics omega-mediaqueries-processed omega-equalheights-processed responsive-layout-wide hw-responsive-layout-wide"><div id="omega-media-query-dummy"><style media="all">#omega-media-query-dummy { position: relative; z-index: -1; }</style><!--[if (lt IE 9)&(!IEMobile)]><style media="all">#omega-media-query-dummy { z-index: 0; }</style><![endif]--><style media="all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)">#omega-media-query-dummy { z-index: 0; }</style><style media="all and (min-width: 1220px)">#omega-media-query-dummy { z-index: 1; }</style></div>
  <div id="skip-link">
    <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
  </div>
    <div class="page clearfix panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix user-menu-dropdown user-menu-dropdown-logo user-menu-dropdown-logo-only">
      <nav role="menubar" class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu nice-menus-processed sf-js-enabled sf-arrows" id="nice-menu-2" role="menu"><li class="menu-444 menuparent  menu-path-front first odd last" role="menuitem"><a href="http://physiolgenomics.physiology.org/" title="" class="sf-with-ul" data-icon-position="" data-hide-link-title="0">APS Journals</a><ul style="display: none;" role="menu"><li class="menu-610 menu-path-ajpcellphysiologyorg- first odd " role="menuitem"><a href="http://ajpcell.physiology.org/" class="" data-icon-position="" data-hide-link-title="0">AJP-Cell</a></li>
<li class="menu-611 menu-path-ajpendophysiologyorg-  even " role="menuitem"><a href="http://ajpendo.physiology.org/" class="" data-icon-position="" data-hide-link-title="0">AJP-Endo</a></li>
<li class="menu-612 menu-path-ajpgiphysiologyorg-  odd " role="menuitem"><a href="http://ajpgi.physiology.org/" class="" data-icon-position="" data-hide-link-title="0">AJP-GI</a></li>
<li class="menu-613 menu-path-ajpheartphysiologyorg-  even " role="menuitem"><a href="http://ajpheart.physiology.org/" class="" data-icon-position="" data-hide-link-title="0">AJP-Heart</a></li>
<li class="menu-614 menu-path-ajplungphysiologyorg-  odd " role="menuitem"><a href="http://ajplung.physiology.org/" class="" data-icon-position="" data-hide-link-title="0">AJP-Lung</a></li>
<li class="menu-615 menu-path-ajpreguphysiologyorg-  even " role="menuitem"><a href="http://ajpregu.physiology.org/" class="" data-icon-position="" data-hide-link-title="0">AJP-Regu</a></li>
<li class="menu-616 menu-path-ajprenalphysiologyorg-  odd " role="menuitem"><a href="http://ajprenal.physiology.org/" class="" data-icon-position="" data-hide-link-title="0">AJP-Renal</a></li>
<li class="menu-727 menu-path-ajplegacyphysiologyorg-  even " role="menuitem"><a href="http://ajplegacy.physiology.org/" title="" class="" data-icon-position="" data-hide-link-title="0">AJP-Legacy</a></li>
<li class="menu-617 menu-path-physiologyonlinephysiologyorg-  odd " role="menuitem"><a href="http://physiologyonline.physiology.org/" class="" data-icon-position="" data-hide-link-title="0">Physiology</a></li>
<li class="menu-618 menu-path-advanphysiologyorg-  even " role="menuitem"><a href="http://advan.physiology.org/" class="" data-icon-position="" data-hide-link-title="0">Advances</a></li>
<li class="menu-619 menu-path-japphysiologyorg-  odd " role="menuitem"><a href="http://jap.physiology.org/" class="" data-icon-position="" data-hide-link-title="0">JAPPL</a></li>
<li class="menu-620 menu-path-jnphysiologyorg  even " role="menuitem"><a href="http://jn.physiology.org/" class="" data-icon-position="" data-hide-link-title="0">JN</a></li>
<li class="menu-621 menu-path-physiolgenomicsphysiologyorg-  odd " role="menuitem"><a href="http://physiolgenomics.physiology.org/" class="" data-icon-position="" data-hide-link-title="0">PG</a></li>
<li class="menu-622 menu-path-physrevphysiologyorg-  even " role="menuitem"><a href="http://physrev.physiology.org/" class="" data-icon-position="" data-hide-link-title="0">PRV</a></li>
<li class="menu-623 menu-path-comprehensivephysiologycom-  odd " role="menuitem"><a href="http://www.comprehensivephysiology.com/" class="" data-icon-position="" data-hide-link-title="0">COMP PHYS</a></li>
<li class="menu-624 menu-path-physiologicalreportsorg-  even " role="menuitem"><a href="http://www.physiologicalreports.org/" class="" data-icon-position="" data-hide-link-title="0">PHYSIOL REP</a></li>
<li class="menu-726 menu-path-apsselectphysiologyorg  odd " role="menuitem"><a href="http://apsselect.physiology.org/" title="" class="" data-icon-position="" data-hide-link-title="0">APS Select</a></li>
<li class="menu-738 menu-path-physiologyorg  even last" role="menuitem"><a href="http://www.physiology.org/" title="" class="" data-icon-position="" data-hide-link-title="0">www.physiology.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><section class="block block-jnl-aps-styles block-the-aps-org block-jnl-aps-styles-the-aps-org even" id="block-jnl-aps-styles-the-aps-org">
  <div class="block-inner clearfix">
              <h2 class="block-title"><none></none></h2>
            
    <div class="content clearfix">
      <a href="http://www.the-aps.org/" class="aps-org-link" data-icon-position="" data-hide-link-title="0"><img class="aps-org-img" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/the-aps-org.png" alt=""></a>    </div>
  </div>
</section>  </div>
</div><div style="height: 47px;" class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-panels-mini block-aps-user-id-string block-aps-user-id-string block-panels-mini-aps-user-id-string odd block-without-title" id="block-panels-mini-aps-user-id-string">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-aps_user_id_string">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-custom pane-2">
  
      
  
  <div class="pane-content">
    <p><span>Institution: <strong>University of British Columbia Library</strong></span></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-aps-logged-in-links block-panels-mini-aps-logged-in-links even block-without-title" id="block-panels-mini-aps-logged-in-links">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-aps_logged_in_links">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-aps-logged-out-links block-panels-mini-aps-logged-out-links odd block-without-title" id="block-panels-mini-aps-logged-out-links">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-aps_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-custom pane-3">
  
      
  
  <div class="pane-content">
    <p><a href="http://physiolgenomics.physiology.org/user?destination=front">Login</a></p>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-block social-icons block-1 block-block-1 even block-without-title" id="block-block-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div id="block-follow">
	<span style="margin: 4px 2px 2px 0;"><a href="https://twitter.com/APSPhysiology"><img alt="" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/twitter-bird-dark-bgs-30px.png" style="width: 30px; height: 30px;"></a></span><span style="margin: 4px 2px 2px 0;"><a href="http://www.linkedin.com/groups/American-Physiological-Society-164204/about"><img alt="" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/LinkedIn_Logo30px.png" style="width: 30px; height: 25px;"></a>&nbsp;</span><span style="margin: 4px 2px 2px 0;"><a href="https://www.facebook.com/Physiol.genomics"><img alt="" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/FB-f-Logo__blue_29.png" style="width: 29px; height: 29px;"></a></span><span style="margin: 4px 2px 2px 8px;"><a href="http://physiolgenomics.physiology.org/rss"><img alt="" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/RSS-Icon.png" style="width: 28px; height: 28px;"></a></span></div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div><div id="zone-advertising-top-wrapper" class="zone-wrapper zone-advertising-top-wrapper clearfix">  
  <div id="zone-advertising-top" class="zone zone-advertising-top clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-ad-top" id="region-ad-top">
  <div class="region-inner region-ad-top-inner">
    <div class="block block-google-admanager block-ce8c3a9b4e77fddb0ae6ab10091d9548 block-google-admanager-ce8c3a9b4e77fddb0ae6ab10091d9548 odd block-without-title" id="block-google-admanager-ce8c3a9b4e77fddb0ae6ab10091d9548">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div id="gam-holder-physiolgenomics_leaderboard_top_728x90" class="gam-holder"><script type="text/javascript">GA_googleFillSlot("physiolgenomics_leaderboard_top_728x90");</script><script src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/ads"></script><div id="google_ads_div_physiolgenomics_leaderboard_top_728x90_ad_wrapper">
<div id="google_ads_div_physiolgenomics_leaderboard_top_728x90_ad_container" style="display:inline-block;">
<ins style="width: 728px; height: 90px; display: inline-table; position: relative; border: 0px none;"><ins style="width: 728px; height: 90px; display: block; position: relative; border: 0px none;"><iframe style="border: 0px none; position: absolute; top: 0px; left: 0px;" marginheight="0" marginwidth="0" scrolling="no" name="google_ads_iframe_physiolgenomics_leaderboard_top_728x90" id="google_ads_iframe_physiolgenomics_leaderboard_top_728x90" frameborder="0" height="90" width="728"></iframe></ins></ins></div>
</div>
<script>GA_googleCreateDomIframe("google_ads_div_physiolgenomics_leaderboard_top_728x90_ad_container" ,"physiolgenomics_leaderboard_top_728x90");</script></div>    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-branding" class="zone zone-branding clearfix container-30">
    <div class="grid-19 prefix-1 region region-branding" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="http://physiolgenomics.physiology.org/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="Physiological Genomics" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/Journal%2520Logo%2520with%2520_sm_Avenir_PG-2015.png"></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-physiolgenomics-search-box block-panels-mini-physiolgenomics-search-box odd block-without-title" id="block-panels-mini-physiolgenomics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-physiolgenomics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch">
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/34/1/127.long" method="post" id="highwire_search_quicksearch_form_1" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search" id="edit-keywords" name="keywords" size="60" maxlength="128" class="form-text" type="text">
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" id="edit-submit--2" name="op" value="Search" class="form-submit" type="submit"></div><input name="form_build_id" value="form-seMksEOftHbZrDZIWPq1XqxJ9oOIV0nJlouv8U-2TUU" type="hidden">
<input name="form_id" value="highwire_search_quicksearch_form" type="hidden">
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-4">
  
      
  
  <div class="pane-content">
    <p class="rteright"><a href="http://physiolgenomics.physiology.org/search">Advanced Search</a></p>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus aps-main-menu block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav role="menubar" class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu nice-menus-processed sf-js-enabled sf-arrows" id="nice-menu-1" role="menu"><li class="menu-449 menu-path-front first odd " role="menuitem"><a href="http://physiolgenomics.physiology.org/" title="" class="" data-icon-position="" data-hide-link-title="0">HOME</a></li>
<li class="menu-450 menuparent  menu-path-content-current active-trail  even" role="menuitem"><a href="http://physiolgenomics.physiology.org/content/current" title="" class="active sf-with-ul" data-icon-position="" data-hide-link-title="0">ARTICLES</a><ul style="display: none;" role="menu"><li class="menu-451 menu-path-content-current first odd " role="menuitem"><a href="http://physiolgenomics.physiology.org/content/current" title="" class="" data-icon-position="" data-hide-link-title="0">Current Issue</a></li>
<li class="menu-453 menu-path-content-by-year  even " role="menuitem"><a href="http://physiolgenomics.physiology.org/content/by/year" title="" class="" data-icon-position="" data-hide-link-title="0">Archives</a></li>
<li class="menu-452 menu-path-content-early-recent  odd " role="menuitem"><a href="http://physiolgenomics.physiology.org/content/early/recent" title="" class="" data-icon-position="" data-hide-link-title="0">Articles In Press </a></li>
<li class="menu-664 menu-path-collection  even last" role="menuitem"><a href="http://physiolgenomics.physiology.org/collection" title="" class="" data-icon-position="" data-hide-link-title="0">Collections</a></li>
</ul></li>
<li class="menu-454 menuparent  menu-path-node-1  odd" role="menuitem"><a href="http://physiolgenomics.physiology.org/content/authors" title="" class="sf-with-ul" data-icon-position="" data-hide-link-title="0">INFO FOR...</a><ul style="display: none;" role="menu"><li class="menu-662 menu-path-node-2783 first odd " role="menuitem"><a href="http://physiolgenomics.physiology.org/about" class="" data-icon-position="" data-hide-link-title="0">About</a></li>
<li class="menu-812 menu-path-the-apsorg-mm-publications-journals-physiol-genomics-article-types  even " role="menuitem"><a href="http://www.the-aps.org/mm/Publications/Journals/Physiol-Genomics/Article-Types" title="" class="" data-icon-position="" data-hide-link-title="0">Article Types</a></li>
<li class="menu-736 menu-path-node-22556  odd " role="menuitem"><a href="http://physiolgenomics.physiology.org/content/journal-sections" class="" data-icon-position="" data-hide-link-title="0">Journal Sections</a></li>
<li class="menu-455 menu-path-the-apsorg-mm-publications-info-for-authors  even " role="menuitem"><a href="http://www.the-aps.org/mm/Publications/Info-For-Authors" title="" class="" data-icon-position="" data-hide-link-title="0">Authors</a></li>
<li class="menu-456 menu-path-the-apsorg-mm-publications-journals-reviewer-guidelineshtml  odd " role="menuitem"><a href="http://www.the-aps.org/mm/Publications/Journals/Reviewer-Guidelines.html" title="" class="" data-icon-position="" data-hide-link-title="0">Reviewers</a></li>
<li class="menu-457 menu-path-node-2792  even " role="menuitem"><a href="http://physiolgenomics.physiology.org/subscriptions" title="" class="" data-icon-position="" data-hide-link-title="0">Subscribers</a></li>
<li class="menu-460 menu-path-node-2809  odd " role="menuitem"><a href="http://physiolgenomics.physiology.org/patient-request-for-copy-of-an-article-recently-published-in-the-american-journal-of-physiology-endocrinology-and-metabolism-physiology" class="" data-icon-position="" data-hide-link-title="0">Patients</a></li>
<li class="menu-672 menu-path-node-2817  even " role="menuitem"><a href="http://physiolgenomics.physiology.org/site/misc/addir" class="" data-icon-position="" data-hide-link-title="0">Advertisers</a></li>
<li class="menu-459 menu-path-the-apsorg-mm-hp-audiences-public-press  odd " role="menuitem"><a href="http://www.the-aps.org/mm/hp/Audiences/Public-Press" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Media</a></li>
<li class="menu-140557 menu-path-the-apsorg-mm-publications-publications-news  even " role="menuitem"><a href="http://www.the-aps.org/mm/Publications/Publications-News" data-hide-link-title="0" target="_blank" class="" data-icon-position="">News</a></li>
<li class="menu-758 menu-path-the-apsorg-mm-publications-journals-physiol-genomics-covers  odd last" role="menuitem"><a href="http://www.the-aps.org/mm/Publications/Journals/Physiol-Genomics/Covers" class="" data-icon-position="" data-hide-link-title="0">Cover Gallery</a></li>
</ul></li>
<li class="menu-461 menuparent  menu-path-node-2784  even" role="menuitem"><a href="http://physiolgenomics.physiology.org/editors-bio" class="sf-with-ul" data-icon-position="" data-hide-link-title="0">EDITORS</a><ul style="display: none;" role="menu"><li class="menu-707 menu-path-physiolgenomicsphysiologyorg-content-47-7-241 first odd " role="menuitem"><a href="http://physiolgenomics.physiology.org/content/47/7/241" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Editor's Message</a></li>
<li class="menu-462 menu-path-node-2784  even " role="menuitem"><a href="http://physiolgenomics.physiology.org/editors-bio" title="" class="" data-icon-position="" data-hide-link-title="0">Editor's Bio</a></li>
<li class="menu-464 menu-path-physiolgenomicsphysiologyorg-content-editorial-board-0  odd last" role="menuitem"><a href="http://physiolgenomics.physiology.org/content/editorial-board-0" title="" class="" data-icon-position="" data-hide-link-title="0">Editorial Board</a></li>
</ul></li>
<li class="menu-465 menuparent  menu-path-the-apsorg-mm-publications-purchasing  odd" role="menuitem"><a href="http://www.the-aps.org/mm/Publications/Purchasing" title="" class="sf-with-ul" data-icon-position="" data-hide-link-title="0">SUBSCRIBE</a><ul style="display: none;" role="menu"><li class="menu-466 menu-path-the-apsorg-mm-publications-purchasing first odd last" role="menuitem"><a href="http://www.the-aps.org/mm/Publications/Purchasing" title="" class="" data-icon-position="" data-hide-link-title="0">Subscribe</a></li>
</ul></li>
<li class="menu-663 menuparent  menu-path-pgmsubmitnet-cgi-bin-mainplex  even last" role="menuitem"><a href="http://pg.msubmit.net/cgi-bin/main.plex" title="" class="sf-with-ul" data-icon-position="" data-hide-link-title="0">SUBMIT</a><ul style="display: none;" role="menu"><li class="menu-706 menu-path-pgmsubmitnet-cgi-bin-mainplex first odd last" role="menuitem"><a href="http://pg.msubmit.net/cgi-bin/main.plex" title="" class="" data-icon-position="" data-hide-link-title="0">Submit a Manuscript</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-highwire-article-citation">
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="4156" id="node4156--3" data-pisa="physiolgenomics;34/1/127" data-pisa-master="physiolgenomics;00025.2008" data-apath="/physiolgenomics/34/1/127.atom"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-jnl-aps-full-citation clearfix has-author-tooltip">
    
  
  
      <h1 class="highwire-cite-title" id="page-title">Validating the genomic signature of pediatric septic shock</h1>  
    	<div class="highwire-cite-authors"><span class="highwire-citation-authors"><span title="" rel="#hw-article-author-popups-node4156--3 .author-tooltip-0" class="highwire-citation-author first article-author-popup-processed has-tooltip" data-delta="0"><span class="nlm-given-names">Natalie</span> <span class="nlm-surname">Cvijanovich</span></span>, <span title="" rel="#hw-article-author-popups-node4156--3 .author-tooltip-1" class="highwire-citation-author article-author-popup-processed has-tooltip" data-delta="1"><span class="nlm-given-names">Thomas P.</span> <span class="nlm-surname">Shanley</span></span>, <span title="" rel="#hw-article-author-popups-node4156--3 .author-tooltip-2" class="highwire-citation-author article-author-popup-processed has-tooltip" data-delta="2"><span class="nlm-given-names">Richard</span> <span class="nlm-surname">Lin</span></span>, <span rel="#hw-article-author-popups-node4156--3 .author-tooltip-3" class="highwire-citation-author article-author-popup-processed has-tooltip" data-delta="3"><span class="nlm-given-names">Geoffrey L.</span> <span class="nlm-surname">Allen</span></span>, <span title="" rel="#hw-article-author-popups-node4156--3 .author-tooltip-4" class="highwire-citation-author article-author-popup-processed has-tooltip" data-delta="4"><span class="nlm-given-names">Neal J.</span> <span class="nlm-surname">Thomas</span></span>, <span rel="#hw-article-author-popups-node4156--3 .author-tooltip-5" class="highwire-citation-author article-author-popup-processed has-tooltip" data-delta="5"><span class="nlm-given-names">Paul</span> <span class="nlm-surname">Checchia</span></span>, <span title="" rel="#hw-article-author-popups-node4156--3 .author-tooltip-6" class="highwire-citation-author article-author-popup-processed has-tooltip" data-delta="6"><span class="nlm-given-names">Nick</span> <span class="nlm-surname">Anas</span></span>, <span title="" rel="#hw-article-author-popups-node4156--3 .author-tooltip-7" class="highwire-citation-author article-author-popup-processed has-tooltip" data-delta="7"><span class="nlm-given-names">Robert J.</span> <span class="nlm-surname">Freishtat</span></span>, <span rel="#hw-article-author-popups-node4156--3 .author-tooltip-8" class="highwire-citation-author article-author-popup-processed has-tooltip" data-delta="8"><span class="nlm-given-names">Marie</span> <span class="nlm-surname">Monaco</span></span>, <span rel="#hw-article-author-popups-node4156--3 .author-tooltip-9" class="highwire-citation-author article-author-popup-processed has-tooltip" data-delta="9"><span class="nlm-given-names">Kelli</span> <span class="nlm-surname">Odoms</span></span>, <span title="" rel="#hw-article-author-popups-node4156--3 .author-tooltip-10" class="highwire-citation-author article-author-popup-processed has-tooltip" data-delta="10"><span class="nlm-given-names">Bhuvaneswari</span> <span class="nlm-surname">Sakthivel</span></span>, <span title="" rel="#hw-article-author-popups-node4156--3 .author-tooltip-11" class="highwire-citation-author article-author-popup-processed has-tooltip" data-delta="11"><span class="nlm-given-names">Hector R.</span> <span class="nlm-surname">Wong</span></span></span></div>
  
    	<div class="highwire-cite-metadata"><span class="highwire-cite-metadata-journal-title highwire-cite-metadata">Physiological Genomics </span><span class="highwire-cite-metadata-date highwire-cite-metadata">Published 1 June 2008 </span><span class="highwire-cite-metadata-volume highwire-cite-metadata"><span class="label">Vol.</span> 34 </span><span class="highwire-cite-metadata-issue highwire-cite-metadata"><span class="label">no. </span> 1, </span><span class="highwire-cite-metadata-pages highwire-cite-metadata"><span class="label"></span> 127-134 </span><span class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">DOI:</span> 10.1152/physiolgenomics.00025.2008 </span></div>
  
  
  
</div><div id="hw-article-author-popups-node4156--3" style="display: none;"><div style="display: none;" class="author-tooltip-0"><div class="author-tooltip-name"><span class="nlm-given-names">Natalie</span> <span class="nlm-surname">Cvijanovich</span> </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="http://physiolgenomics.physiology.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author[0]=N.%2BCvijanovich" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=Cvijanovich%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="http://physiolgenomics.physiology.org/search/author1%3ANatalie%2BCvijanovich%2B" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div style="display: none;" class="author-tooltip-1"><div class="author-tooltip-name"><span class="nlm-given-names">Thomas P.</span> <span class="nlm-surname">Shanley</span> </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="http://physiolgenomics.physiology.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author[0]=T.%20P.%2BShanley" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=Shanley%20TP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="http://physiolgenomics.physiology.org/search/author1%3AThomas%2BP.%2BShanley%2B" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div style="display: none;" class="author-tooltip-2"><div class="author-tooltip-name"><span class="nlm-given-names">Richard</span> <span class="nlm-surname">Lin</span> </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="http://physiolgenomics.physiology.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author[0]=R.%2BLin" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=Lin%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="http://physiolgenomics.physiology.org/search/author1%3ARichard%2BLin%2B" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div style="display: none;" class="author-tooltip-3"><div class="author-tooltip-name"><span class="nlm-given-names">Geoffrey L.</span> <span class="nlm-surname">Allen</span> </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="http://physiolgenomics.physiology.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author[0]=G.%20L.%2BAllen" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=Allen%20GL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="http://physiolgenomics.physiology.org/search/author1%3AGeoffrey%2BL.%2BAllen%2B" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div style="display: none;" class="author-tooltip-4"><div class="author-tooltip-name"><span class="nlm-given-names">Neal J.</span> <span class="nlm-surname">Thomas</span> </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="http://physiolgenomics.physiology.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author[0]=N.%20J.%2BThomas" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=Thomas%20NJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="http://physiolgenomics.physiology.org/search/author1%3ANeal%2BJ.%2BThomas%2B" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div style="display: none;" class="author-tooltip-5"><div class="author-tooltip-name"><span class="nlm-given-names">Paul</span> <span class="nlm-surname">Checchia</span> </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="http://physiolgenomics.physiology.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author[0]=P.%2BChecchia" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=Checchia%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="http://physiolgenomics.physiology.org/search/author1%3APaul%2BChecchia%2B" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div style="display: none;" class="author-tooltip-6"><div class="author-tooltip-name"><span class="nlm-given-names">Nick</span> <span class="nlm-surname">Anas</span> </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="http://physiolgenomics.physiology.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author[0]=N.%2BAnas" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=Anas%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="http://physiolgenomics.physiology.org/search/author1%3ANick%2BAnas%2B" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div style="display: none;" class="author-tooltip-7"><div class="author-tooltip-name"><span class="nlm-given-names">Robert J.</span> <span class="nlm-surname">Freishtat</span> </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="http://physiolgenomics.physiology.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author[0]=R.%20J.%2BFreishtat" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=Freishtat%20RJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="http://physiolgenomics.physiology.org/search/author1%3ARobert%2BJ.%2BFreishtat%2B" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div style="display: none;" class="author-tooltip-8"><div class="author-tooltip-name"><span class="nlm-given-names">Marie</span> <span class="nlm-surname">Monaco</span> </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="http://physiolgenomics.physiology.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author[0]=M.%2BMonaco" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=Monaco%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="http://physiolgenomics.physiology.org/search/author1%3AMarie%2BMonaco%2B" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div style="display: none;" class="author-tooltip-9"><div class="author-tooltip-name"><span class="nlm-given-names">Kelli</span> <span class="nlm-surname">Odoms</span> </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="http://physiolgenomics.physiology.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author[0]=K.%2BOdoms" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=Odoms%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="http://physiolgenomics.physiology.org/search/author1%3AKelli%2BOdoms%2B" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div style="display: none;" class="author-tooltip-10"><div class="author-tooltip-name"><span class="nlm-given-names">Bhuvaneswari</span> <span class="nlm-surname">Sakthivel</span> </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="http://physiolgenomics.physiology.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author[0]=B.%2BSakthivel" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=Sakthivel%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="http://physiolgenomics.physiology.org/search/author1%3ABhuvaneswari%2BSakthivel%2B" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div style="display: none;" class="author-tooltip-11"><div class="author-tooltip-name"><span class="nlm-given-names">Hector R.</span> <span class="nlm-surname">Wong</span> </div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="http://physiolgenomics.physiology.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author[0]=H.%20R.%2BWong" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=Wong%20HR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="http://physiolgenomics.physiology.org/search/author1%3AHector%2BR.%2BWong%2B" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs panels-ajax-tabs-once-processed">
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a style="cursor: pointer;" href="http://physiolgenomics.physiology.org/content/34/1/127" class="panels-ajax-tab-tab panels-ajax-tabs-once-processed panels-ajax-tabs-first-loaded" data-panel-name="physiolgenomics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:4156" data-trigger="" data-url-enabled="1">Article</a><a href="http://physiolgenomics.physiology.org/panels_ajax_tab/physiolgenomics_tab_art/node:4156/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="active"><a style="cursor: pointer;" href="http://physiolgenomics.physiology.org/content/34/1/127.figures-only" class="panels-ajax-tab-tab panels-ajax-tabs-once-processed" data-panel-name="physiolgenomics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:4156" data-trigger="figures-only" data-url-enabled="1">Figures &amp; Data</a><a href="http://physiolgenomics.physiology.org/panels_ajax_tab/physiolgenomics_tab_data/node:4156/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a style="cursor: pointer;" href="http://physiolgenomics.physiology.org/content/34/1/127.article-info" class="panels-ajax-tab-tab panels-ajax-tabs-once-processed" data-panel-name="physiolgenomics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:4156" data-trigger="article-info" data-url-enabled="1">Info</a><a href="http://physiolgenomics.physiology.org/panels_ajax_tab/physiolgenomics_tab_info/node:4156/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a style="cursor: pointer;" href="http://physiolgenomics.physiology.org/content/34/1/127.full-text.pdf+html" class="panels-ajax-tab-tab panels-ajax-tabs-once-processed" data-panel-name="physiolgenomics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:4156" data-trigger="full-text.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i> PDF</a><a href="http://physiolgenomics.physiology.org/panels_ajax_tab/physiolgenomics_tab_pdf/node:4156/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container">
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="physiolgenomics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style="display: none;"><img class="loading" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/loading.gif" alt="Loading" title="Loading"></div><div style="display: none;" class="panels-ajax-tab-wrap-physiolgenomics_tab_art"><div class="panel-display panel-1col clearfix">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup">
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view"><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2>
      <p id="p-2">We previously generated genome-wide expression data 
(microarray) from children with septic shock having the potential to 
lead the field into novel areas of investigation. Herein we seek to 
validate our data through a bioinformatic approach centered on a 
validation patient cohort. Forty-two children with a clinical diagnosis 
of septic shock and 15 normal controls served as the training data set, 
while 30 separate children with septic shock and 14 separate normal 
controls served as the test data set. Class prediction modeling using 
the training data set and the previously reported genome-wide expression
 signature of pediatric septic shock correctly identified 95â100% of 
controls and septic shock patients in the test data set, depending on 
the class prediction algorithm and the gene selection method. Subjecting
 the test data set to an identical filtering strategy as that used for 
the training data set, demonstrated 75% concordance between the two gene
 lists. Subjecting the test data set to a purely statistical filtering 
strategy, with highly stringent correction for multiple comparisons, 
demonstrated &lt;50% concordance with the previous gene filtering 
strategy. However, functional analysis of this statistics-based gene 
list demonstrated similar functional annotations and signaling pathways 
as that seen in the training data set. In particular, we validated that 
pediatric septic shock is characterized by large-scale repression of 
genes related to zinc homeostasis and lymphocyte function. These data 
demonstrate that the previously reported genome-wide expression 
signature of pediatric septic shock is applicable to a validation cohort
 of patients.</p>
    </div><ul class="kwd-group KWD"><li class="kwd">microarray</li><li class="kwd">pediatrics</li><li class="kwd">T cell function</li><li class="kwd">zinc</li></ul><p id="p-3"><span class="sc">the application of high-throughput</span>
 technologies, such as microarray, affords unprecedented opportunities 
to generate novel hypothesis and to gain a more comprehensive 
understanding of human diseases. We have suggested that complex and 
heterogeneous phenotypes, such as pediatric septic shock, are 
particularly amenable to this discovery-oriented approach (<a rel="#ref-15~.ref-cit" id="xref-ref-15-1" class="xref-bibr article-ref-popup-processed" href="#ref-15">15</a>, <a rel="#ref-18~.ref-cit" id="xref-ref-18-1" class="xref-bibr article-ref-popup-processed" href="#ref-18">18</a>). Septic shock continues to be an important child health problem with significant morbidity and mortality (<a rel="#ref-14~.ref-cit" id="xref-ref-14-1" class="xref-bibr article-ref-popup-processed" href="#ref-14">14</a>, <a rel="#ref-16~.ref-cit" id="xref-ref-16-1" class="xref-bibr article-ref-popup-processed" href="#ref-16">16</a>).
 In an ongoing translational research program focused on pediatric 
septic shock, we have begun to identify the genome-level expression 
signature of this syndrome through the application of microarray 
technology (<a rel="#ref-13~.ref-cit" id="xref-ref-13-1" class="xref-bibr article-ref-popup-processed" href="#ref-13">13</a>, <a rel="#ref-18~.ref-cit" id="xref-ref-18-2" class="xref-bibr article-ref-popup-processed" href="#ref-18">18</a>, <a rel="#ref-19~.ref-cit" id="xref-ref-19-1" class="xref-bibr article-ref-popup-processed" href="#ref-19">19</a>).
 The initial data generated through this discovery-oriented approach 
suggest that altered zinc homeostasis and lymphocyte dysfunction are 
prevalent problems in children with septic shock and thus potentially 
direct the field toward new investigative paradigms given the direct 
biological links between zinc homeostasis and lymphocyte function (<a rel="#ref-12~.ref-cit" id="xref-ref-12-1" class="xref-bibr article-ref-popup-processed" href="#ref-12">12</a>).</p><p id="p-4">Our
 current enthusiasm surrounding the genome-wide expression data in 
pediatric septic shock is tempered by the intricacies and nuances 
surrounding microarray data analysis, a methodology still in evolution (<a rel="#ref-1~.ref-cit" id="xref-ref-1-1" class="xref-bibr article-ref-popup-processed" href="#ref-1">1</a>, <a rel="#ref-5~.ref-cit" id="xref-ref-5-1" class="xref-bibr article-ref-popup-processed" href="#ref-5">5</a>, <a rel="#ref-10~.ref-cit" id="xref-ref-10-1" class="xref-bibr article-ref-popup-processed" href="#ref-10">10</a>, <a rel="#ref-17~.ref-cit" id="xref-ref-17-1" class="xref-bibr article-ref-popup-processed" href="#ref-17">17</a>).
 There are currently no definitive, absolute, standard statistical 
approaches for analyzing microarray data. The shear volume of typical 
microarray data, leading to simultaneous testing of tens of thousands of
 transcripts, has the potential to produce hundreds of false positive 
results depending on the statistical approach. Accordingly, the 
particular statistical and filtering approaches that are applied to a 
given microarray data set can yield markedly different results (<a rel="#ref-10~.ref-cit" id="xref-ref-10-2" class="xref-bibr article-ref-popup-processed" href="#ref-10">10</a>, <a rel="#ref-17~.ref-cit" id="xref-ref-17-2" class="xref-bibr article-ref-popup-processed" href="#ref-17">17</a>).
 Thus, the possibility exists that our previous observations involving 
gene expression profiling in pediatric septic shock are merely 
epiphenomena of the bioinformatic approaches used in those initial 
studies.</p><p id="p-5">Prospective validation of microarray data 
represents one potential approach for addressing these complex 
analytical issues. The ultimate goal of validation is to strengthen the 
initial conclusions derived from microarray data sets, as we have done 
with selected portions of our previous data (<a rel="#ref-13~.ref-cit" id="xref-ref-13-2" class="xref-bibr article-ref-popup-processed" href="#ref-13">13</a>, <a rel="#ref-18~.ref-cit" id="xref-ref-18-3" class="xref-bibr article-ref-popup-processed" href="#ref-18">18</a>, <a rel="#ref-19~.ref-cit" id="xref-ref-19-2" class="xref-bibr article-ref-popup-processed" href="#ref-19">19</a>). Validation of microarray data can be achieved by a variety of strategies (<a rel="#ref-1~.ref-cit" id="xref-ref-1-2" class="xref-bibr article-ref-popup-processed" href="#ref-1">1</a>, <a rel="#ref-10~.ref-cit" id="xref-ref-10-3" class="xref-bibr article-ref-popup-processed" href="#ref-10">10</a>, <a rel="#ref-17~.ref-cit" id="xref-ref-17-3" class="xref-bibr article-ref-popup-processed" href="#ref-17">17</a>).
 For example, real-time polymerase chain reaction can be used for 
confirmation of differentially expressed gene probes. Proteomics can 
also be used for data validation and has the advantage of providing gene
 product information beyond the level of transcripts. An alternative 
validation strategy involves the use of a âvalidationâ or âtestâ data 
set representing an entirely different set of samples from that used for
 derivation of the original microarray data set.</p><p id="p-6">This paper addresses two equally important and related questions: <em>1</em>) are the original observations that we reported in children with septic shock also operative in a validation cohort? and <em>2</em>)
 are these observations dependent on the bioinformatic approaches used 
to analyze the expression data? Herein we have directly addressed these 
questions by employing a training data set (our original microarray 
data) and a test data set consisting of an entirely new cohort of 
patients. We have prospectively applied validation procedures, including
 the use of statistical approaches that are distinct to that of our 
previous reports, as a means of assessing the authenticity of our 
initial observations.</p><div class="section" id="sec-1">
      <h2>METHODS</h2>
      <div id="sec-2" class="subsection">
        <div id="sec-3" class="subsection">
          <h4>Patients.</h4>
          <p id="p-7">The multi-institutional genomic and clinical 
database supporting this translational research program has been 
previously described in detail (<a rel="#ref-13~.ref-cit" id="xref-ref-13-3" class="xref-bibr article-ref-popup-processed" href="#ref-13">13</a>, <a rel="#ref-19~.ref-cit" id="xref-ref-19-3" class="xref-bibr article-ref-popup-processed" href="#ref-19">19</a>).
 Briefly, the study protocol was approved by the individual 
Institutional Review Boards of each participating institution. Children 
&lt;10 yr of age admitted to the pediatric intensive care unit (PICU) 
and meeting published, pediatric-specific criteria for septic shock were
 eligible for the study (<a rel="#ref-7~.ref-cit" id="xref-ref-7-1" class="xref-bibr article-ref-popup-processed" href="#ref-7">7</a>).
 Normal control patients were recruited from the participating 
institutions using the following exclusion criteria: a recent febrile 
illness (within 2 wk), recent use of anti-inflammatory medications 
(within 2 wk), or any history of chronic or acute disease associated 
with inflammation.</p>
        </div>
        <div id="sec-4" class="subsection">
          <h4>Sample and data collection.</h4>
          <p id="p-8">After informed consent, blood samples for RNA 
isolation were obtained within 24 h of admission to the PICU, heretofore
 referred to as â<em>day 1</em>â of septic shock. Severity of illness at
 study entry was calculated using the PRISM III score in the patients 
with septic shock (<a rel="#ref-11~.ref-cit" id="xref-ref-11-1" class="xref-bibr article-ref-popup-processed" href="#ref-11">11</a>).
 Clinical and laboratory data were collected daily while in the PICU. 
Clinical, laboratory, and biological data were entered and stored using a
 locally developed, web-based database.</p>
        </div>
        <div id="sec-5" class="subsection">
          <h4>RNA extraction and microarray hybridization.</h4>
          <p id="p-9">The data and protocols described in this 
manuscript are deposited in the National Center for Biotechnology 
Information Gene Expression Omnibus (GEO, <a href="http://www.ncbi.nlm.nih.gov/geo/">http://www.ncbi.nlm.nih.gov/geo/</a>). The training data set and the test data set can be found under GEO Series accession numbers <a class="external-ref external-ref-type-gen" href="http://physiolgenomics.physiology.org/lookup/external-ref?link_type=GEN&amp;access_num=GSE4607&amp;atom=%2Fphysiolgenomics%2F34%2F1%2F127.atom">GSE4607</a> and <a class="external-ref external-ref-type-gen" href="http://physiolgenomics.physiology.org/lookup/external-ref?link_type=GEN&amp;access_num=GSE9692&amp;atom=%2Fphysiolgenomics%2F34%2F1%2F127.atom">GSE9692</a>, respectively.</p>
          <p id="p-10">Total RNA was isolated from whole blood samples 
using the PaxGene Blood RNA System (PreAnalytiX; Qiagen/Becton Dickson, 
Valencia, CA) according the manufacturer's specifications. Microarray 
hybridization was performed by the Affymetrix Gene Chip Core facility at
 Cincinnati Children's Hospital Research Foundation as previously 
described (Human Genome U133 Plus 2.0 GeneChip; Affymetrix, Santa Clara,
 CA) (<a rel="#ref-13~.ref-cit" id="xref-ref-13-4" class="xref-bibr article-ref-popup-processed" href="#ref-13">13</a>, <a rel="#ref-19~.ref-cit" id="xref-ref-19-4" class="xref-bibr article-ref-popup-processed" href="#ref-19">19</a>).</p>
        </div>
        <div id="sec-6" class="subsection">
          <h4>Data analysis.</h4>
          <p id="p-11">Analyses were performed using one patient sample 
per chip. Image files were captured using an Affymetrix GeneChip Scanner
 3000. CEL files were subsequently preprocessed using robust 
multiple-array average (RMA) normalization using GeneSpring GX 7.3 
software (Agilent Technologies, Palo Alto, CA) (<a rel="#ref-9~.ref-cit" id="xref-ref-9-1" class="xref-bibr article-ref-popup-processed" href="#ref-9">9</a>).
 All signal intensity-based data were used after RMA normalization, 
which specifically suppresses all but significant variation among 
lower-intensity probe sets. All chips were then normalized to the 
respective median values of controls.</p>
          <p id="p-12">Differences in mRNA abundance between patient 
samples and controls were determined using GeneSpring GX 7.3. All 
statistical analyses used corrections for multiple comparisons. 
Cross-validation procedures and class prediction modeling were also 
performed using GeneSpring GX 7.3 using the default parameters of the 
software. Further details regarding derivation of differentially 
regulated gene lists, cross-validation, class prediction modeling, and 
statistical analyses are provided in <span class="sc">results</span>.</p>
          <p id="p-13">Gene lists of differentially expressed genes were
 analyzed using D.A.V.I.D. (Database for Annotation, Visualization, and 
Integrated Discovery) and the Ingenuity Pathways Analysis application 
(IPA; Ingenuity Systems, Redwood City, CA). D.A.V.I.D. allows public 
access to relational databases of functional gene annotations (<a rel="#ref-6~.ref-cit" id="xref-ref-6-1" class="xref-bibr article-ref-popup-processed" href="#ref-6">6</a>).
 In the D.A.V.I.D. analytical output, âcategoryâ refers to the original 
database or resource from which the annotations are derived, and âtermâ 
refers to the enriched annotation terms associated with the given gene 
list. The IPA application provides a tool for discovery of signaling 
pathways within the uploaded gene lists (<a rel="#ref-4~.ref-cit" id="xref-ref-4-1" class="xref-bibr article-ref-popup-processed" href="#ref-4">4</a>, <a rel="#ref-13~.ref-cit" id="xref-ref-13-5" class="xref-bibr article-ref-popup-processed" href="#ref-13">13</a>, <a rel="#ref-19~.ref-cit" id="xref-ref-19-5" class="xref-bibr article-ref-popup-processed" href="#ref-19">19</a>).
 The D.A.V.I.D. and IPA applications are both based on the established 
biomedical literature and use specific approaches to estimate 
significance (<em>P</em> values) based on nonredundant representations 
of the microarray chip and to convert the uploaded gene lists to gene 
lists containing a single value per gene. The <em>P</em> values for a 
given functional annotation or signaling pathway provide an estimate of 
the probability that a given annotation is enriched in a given gene list
 by chance alone. Because there is no clear consensus as to the most 
appropriate bioinformatic approach to use for these types of data sets, 
we consistently used the default parameters in GeneSpring, D.A.V.I.D., 
and the IPA application, unless otherwise specified.</p>
        </div>
      </div>
    </div><div class="section" id="sec-7">
      <h2>RESULTS</h2>
      <div id="sec-8" class="subsection">
        <div id="sec-9" class="subsection">
          <h4>General information for the training and test data sets.</h4>
          <p id="p-14">The training data set (42 children with septic shock and 15 controls) has been previously reported (<a rel="#ref-19~.ref-cit" id="xref-ref-19-6" class="xref-bibr article-ref-popup-processed" href="#ref-19">19</a>).
 The test data set consisted of 30 separate children with septic shock 
and 14 separate control subjects that have not been previously reported.
 The patients for the test data set were selected based on sequential 
order of enrollment into the database, with an enrollment classification
 of septic shock or control, but without regard to demographic, 
clinical, or microbiologic variables.</p>
          <p id="p-15">Summary demographic, clinical, and microbiologic data for the subjects in the test data set are provided in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Tables 1</a> and <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">2</a>.
 A total of 44 individual microarray chips, representing 14 individual 
controls and 30 individual patients with septic shock (test data set), 
were used for analysis. The microarray data represent the gene 
expression profiles of <em>day 1</em> of septic shock (i.e., within 24 h
 of presentation to the PICU). The patients with septic shock in the 
test data set were similar to the controls and the training data set (<a rel="#ref-19~.ref-cit" id="xref-ref-19-7" class="xref-bibr article-ref-popup-processed" href="#ref-19">19</a>)
 with regard to age, race, and sex. In addition, the patients with 
septic shock in the test data set had a similar severity of illness 
(PRISM score) as that of the training data set (<a rel="#ref-19~.ref-cit" id="xref-ref-19-8" class="xref-bibr article-ref-popup-processed" href="#ref-19">19</a>).
 Among the patients in the test data set there were 14 having positive 
identification of an infecting organism (47%) and 6 deaths (20% 
mortality). In comparison, the training data set had 67% of the patients
 with a positively identified infecting organism and 21% mortality (<a rel="#ref-19~.ref-cit" id="xref-ref-19-9" class="xref-bibr article-ref-popup-processed" href="#ref-19">19</a>). Finally, <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>
 demonstrates that the test data set had less heterogeneity of infecting
 organisms and higher representation of infection with gram-positive 
bacteria compared with that of the training data set (<a rel="#ref-19~.ref-cit" id="xref-ref-19-10" class="xref-bibr article-ref-popup-processed" href="#ref-19">19</a>).</p><div class="table pos-float table-expand-inline" id="T1"><div class="table-inline"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="0 first"><a href="http://physiolgenomics.physiology.org/" class="table-expand-inline highwireTablesMarkupProcessor-processed" data-table-url="/highwire/markup/11607/expansion?postprocessors=highwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" html="1" fragment="#" external="1" data-icon-position="" data-hide-link-title="0">Collapse inline</a></li><li class="1 last"><a href="http://physiolgenomics.physiology.org/highwire/markup/11607/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_figures%2Chighwire_math%2Chighwire_inline_linked_media" class="colorbox colorbox-load table-expand-popup init-colorbox-processed cboxElement" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><div class="highwire-markup-wrapper highwire-get-markup"><div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="table-expansion" id="T1"><span class="highwire-journal-article-marker-start"></span><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-47"><em>Clinical and demographic data for all subjects in test data set</em></p><div class="sb-div caption-clear"></div></div><div class="table-wrapper"><table alt="Table 1." id="table-1"><thead id="thead-1"><tr id="tr-1"><th colspan="1" rowspan="1" id="th-1" class="table-center table-vbottom" align="center" valign="bottom"></th><th colspan="1" rowspan="1" id="th-2" class="table-center table-vbottom" align="center" valign="bottom">Controls</th><th colspan="1" rowspan="1" id="th-3" class="table-center table-vbottom" align="center" valign="bottom">Septic Shock</th></tr></thead><tbody id="tbody-1"><tr id="tr-2"><td colspan="1" rowspan="1" id="td-1" class="table-left table-vtop" align="left" valign="top">Individual subjects, <em>n</em></td><td colspan="1" rowspan="1" id="td-2" class="table-center table-vtop" align="center" valign="top">14</td><td colspan="1" rowspan="1" id="td-3" class="table-center table-vtop" align="center" valign="top">30</td></tr><tr id="tr-3"><td colspan="1" rowspan="1" id="td-4" class="table-left table-vtop" align="left" valign="top">Mean age (yr) Â± SD<a id="xref-fn-5-1" class="xref-table-fn" href="#fn-5"><sup>*</sup></a></td><td colspan="1" rowspan="1" id="td-5" class="table-center table-vtop" align="center" valign="top">2.7Â±2.4</td><td colspan="1" rowspan="1" id="td-6" class="table-center table-vtop" align="center" valign="top">3.2Â±2.9</td></tr><tr id="tr-4"><td colspan="1" rowspan="1" id="td-7" class="table-left table-vtop" align="left" valign="top">Mean PRISM score Â± SD<a id="xref-fn-5-2" class="xref-table-fn" href="#fn-5"><sup>*</sup></a></td><td colspan="1" rowspan="1" id="td-8" class="table-center table-vtop" align="center" valign="top">n/a</td><td colspan="1" rowspan="1" id="td-9" class="table-center table-vtop" align="center" valign="top">18.9Â±12.3</td></tr><tr id="tr-5"><td colspan="1" rowspan="1" id="td-10" class="table-left table-vtop" align="left" valign="top">Sex (male/female)<a id="xref-fn-5-3" class="xref-table-fn" href="#fn-5"><sup>*</sup></a></td><td colspan="1" rowspan="1" id="td-11" class="table-center table-vtop" align="center" valign="top">8/6</td><td colspan="1" rowspan="1" id="td-12" class="table-center table-vtop" align="center" valign="top">16/14</td></tr><tr id="tr-6"><td colspan="1" rowspan="1" id="td-13" class="table-left table-vtop" align="left" valign="top">Race (<em>n</em>)<a id="xref-fn-5-4" class="xref-table-fn" href="#fn-5"><sup>*</sup></a></td><td colspan="1" rowspan="1" id="td-14" class="table-left table-vtop" align="left" valign="top">A. A./black (3)</td><td colspan="1" rowspan="1" id="td-15" class="table-left table-vtop" align="left" valign="top">A. A./black (2)</td></tr><tr id="tr-7"><td colspan="1" rowspan="1" id="td-16" class="table-left table-vtop" align="left" valign="top"></td><td colspan="1" rowspan="1" id="td-17" class="table-left table-vtop" align="left" valign="top">white (10)</td><td colspan="1" rowspan="1" id="td-18" class="table-left table-vtop" align="left" valign="top">white (26)</td></tr><tr id="tr-8"><td colspan="1" rowspan="1" id="td-19" class="table-left table-vtop" align="left" valign="top"></td><td colspan="1" rowspan="1" id="td-20" class="table-left table-vtop" align="left" valign="top">Asian (1)</td><td colspan="1" rowspan="1" id="td-21" class="table-left table-vtop" align="left" valign="top">unreported (2)</td></tr></tbody></table></div><div class="table-foot"><ul class="table-footnotes"><li class="fn" id="fn-4"><p id="p-48">n/a, Not applicable; A. A., African American.</p></li><li class="fn" id="fn-5"><p id="p-49"><a class="rev-xref" href="#xref-fn-5-1">âµ</a><span class="fn-label">*</span> <em>P</em> &gt; 0.05.</p></li></ul></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></div></div><div class="table pos-float table-expand-inline" id="T2"><div class="table-inline"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="0 first"><a href="http://physiolgenomics.physiology.org/" class="table-expand-inline highwireTablesMarkupProcessor-processed" data-table-url="/highwire/markup/11619/expansion?postprocessors=highwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" html="1" fragment="#" external="1" data-icon-position="" data-hide-link-title="0">Collapse inline</a></li><li class="1 last"><a href="http://physiolgenomics.physiology.org/highwire/markup/11619/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_figures%2Chighwire_math%2Chighwire_inline_linked_media" class="colorbox colorbox-load table-expand-popup init-colorbox-processed cboxElement" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><div class="highwire-markup-wrapper highwire-get-markup"><div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="table-expansion" id="T2"><span class="highwire-journal-article-marker-start"></span><div class="table-caption"><span class="table-label">Table 2.</span> <p id="p-50"><em>Microbiology data for test data set patients with septic shock</em></p><div class="sb-div caption-clear"></div></div><div class="table-wrapper"><table alt="Table 2." id="table-2"><thead id="thead-2"><tr id="tr-9"><th colspan="1" rowspan="1" id="th-4" class="table-center table-vbottom" align="center" valign="bottom">Organism (<em>n</em>)</th><th colspan="1" rowspan="1" id="th-5" class="table-center table-vbottom" align="center" valign="bottom">Primary Source of Positive Culture (<em>n</em>)</th></tr></thead><tbody id="tbody-2"><tr id="tr-10"><td colspan="1" rowspan="1" id="td-22" class="table-left table-vtop" align="left" valign="top"><em>Staphylococcus aureus</em> (5)</td><td colspan="1" rowspan="1" id="td-23" class="table-center table-vtop" align="center" valign="top">blood (10)</td></tr><tr id="tr-11"><td colspan="1" rowspan="1" id="td-24" class="table-left table-vtop" align="left" valign="top"><em>Streptococcus pyogenes</em> (2)</td><td colspan="1" rowspan="1" id="td-25" class="table-center table-vtop" align="center" valign="top">lung (1)</td></tr><tr id="tr-12"><td colspan="1" rowspan="1" id="td-26" class="table-left table-vtop" align="left" valign="top"><em>Streptococcus agalactiae</em> (2)</td><td colspan="1" rowspan="1" id="td-27" class="table-center table-vtop" align="center" valign="top">cerebral spinal fluid (1)</td></tr><tr id="tr-13"><td colspan="1" rowspan="1" id="td-28" class="table-left table-vtop" align="left" valign="top"><em>Neisseria meningitidis</em> (2)</td><td colspan="1" rowspan="1" id="td-29" class="table-center table-vtop" align="center" valign="top">other (2)</td></tr><tr id="tr-14"><td colspan="1" rowspan="1" id="td-30" class="table-left table-vtop" align="left" valign="top"><em>Streptococcus pneumoniae</em> (2)</td><td colspan="1" rowspan="1" id="td-31" class="table-center table-vtop" align="center" valign="top"></td></tr><tr id="tr-15"><td colspan="1" rowspan="1" id="td-32" class="table-left table-vtop" align="left" valign="top">Adenovirus (1)</td><td colspan="1" rowspan="1" id="td-33" class="table-center table-vtop" align="center" valign="top"></td></tr></tbody></table></div><div class="table-foot"></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></div></div>
          <p id="p-16"><a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>
 provides the total white blood cell counts and the absolute leukocyte 
subpopulation counts for the training and test data sets. The median 
total white blood cell counts, the median absolute neutrophils counts, 
and the median absolute monocyte counts were similar between the 
patients in the training and test data sets. In contrast, patients in 
the test data set had a significantly lower median absolute lymphocyte 
count compared with that of the patients in the training data set.</p><div class="table pos-float" id="T3"><div class="table-inline"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="0 first"><a href="http://physiolgenomics.physiology.org/" class="table-expand-inline highwireTablesMarkupProcessor-processed" data-table-url="/highwire/markup/11623/expansion?postprocessors=highwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" html="1" fragment="#" external="1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="1 last"><a href="http://physiolgenomics.physiology.org/highwire/markup/11623/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_figures%2Chighwire_math%2Chighwire_inline_linked_media" class="colorbox colorbox-load table-expand-popup init-colorbox-processed cboxElement" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3.</span> <p id="p-51" class="first-child"><em>Median absolute white blood cell counts for patients with septic shock in the training and test data sets (Ã10<sup>3</sup> per mm<sup>3</sup>; IQR)</em></p><div class="sb-div caption-clear"></div></div></div>
        </div>
        <div id="sec-10" class="subsection">
          <h4>Class prediction modeling based on training and test data sets.</h4>
          <p id="p-17">The initial approach to validating our microarray
 data involved class prediction modeling using the previously reported 
cohort of patients as the training data set, the new cohort of patients 
described above as the test data set, and the previously published 
genomic signature of pediatric septic shock [i.e., the list of 2,482 
gene probes found to be differentially regulated between patients with 
septic shock in the training data set and controls (<a rel="#ref-19~.ref-cit" id="xref-ref-19-11" class="xref-bibr article-ref-popup-processed" href="#ref-19">19</a>)]. As previously described, the 2,482 gene list was generated by sequential statistical and expression filters as shown in <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>. For the sake of clarity we will heretofore refer to this 2,482 gene list as âgene list A.â</p><div class="table pos-float" id="T4"><div class="table-inline"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="0 first"><a href="http://physiolgenomics.physiology.org/" class="table-expand-inline highwireTablesMarkupProcessor-processed" data-table-url="/highwire/markup/11626/expansion?postprocessors=highwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" html="1" fragment="#" external="1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="1 last"><a href="http://physiolgenomics.physiology.org/highwire/markup/11626/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_figures%2Chighwire_math%2Chighwire_inline_linked_media" class="colorbox colorbox-load table-expand-popup init-colorbox-processed cboxElement" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4.</span> <p id="p-54" class="first-child"><em>Gene list designation and derivation</em></p><div class="sb-div caption-clear"></div></div></div>
          <p id="p-18">We first conducted cross-validation procedures on
 the training data set. Two distinct class prediction algorithms were 
used for this analysis: K-nearest neighbors (KNN) and support vector 
machines (SVM) (<a rel="#ref-3~.ref-cit" id="xref-ref-3-1" class="xref-bibr article-ref-popup-processed" href="#ref-3">3</a>, <a rel="#ref-20~.ref-cit" id="xref-ref-20-1" class="xref-bibr article-ref-popup-processed" href="#ref-20">20</a>). The class options for cross-validation were either âseptic shockâ or âcontrol.â <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>
 demonstrates that cross-validation of the training data set, based on 
the KNN algorithm, correctly identified 93â95% of the subjects in the 
training data set, depending on the gene selection method. <a id="xref-table-wrap-5-2" class="xref-table" href="#T5">Table 5</a>
 also demonstrates that cross-validation of the training data set, based
 on the SVM algorithm, correctly identified 95â96% of the subjects in 
the training data set, depending on the gene selection method.</p><div class="table pos-float" id="T5"><div class="table-inline"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="0 first"><a href="http://physiolgenomics.physiology.org/" class="table-expand-inline highwireTablesMarkupProcessor-processed" data-table-url="/highwire/markup/11629/expansion?postprocessors=highwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" html="1" fragment="#" external="1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="1 last"><a href="http://physiolgenomics.physiology.org/highwire/markup/11629/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_figures%2Chighwire_math%2Chighwire_inline_linked_media" class="colorbox colorbox-load table-expand-popup init-colorbox-processed cboxElement" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5.</span> <p id="p-55" class="first-child"><em>Results of cross-validation of training data set based gene list A and on KNN or SVM algorithms</em></p><div class="sb-div caption-clear"></div></div></div>
          <p id="p-19">We next applied class prediction modeling to the 
test data set, again using both the KNN and SVM class prediction 
algorithms and all possible gene selection methods, as shown in <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a>.
 All possible combinations of gene selection methods and class 
prediction algorithms correctly predicted â¥95% of the subjects (controls
 and septic shock) in the test data set. Using the 50 predictor genes 
derived from the Golub method of gene selection (derived from the 
cross-validation procedures described above), the KNN algorithm 
correctly identified 100% of the subjects (controls and septic shock) in
 the test data set. In a similar manner, using all of the genes in gene 
list A, the SVM algorithm correctly identified 100% of the subjects in 
the test data set.</p><div class="table pos-float" id="T6"><div class="table-inline"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="0 first"><a href="http://physiolgenomics.physiology.org/" class="table-expand-inline highwireTablesMarkupProcessor-processed" data-table-url="/highwire/markup/11632/expansion?postprocessors=highwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" html="1" fragment="#" external="1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="1 last"><a href="http://physiolgenomics.physiology.org/highwire/markup/11632/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_figures%2Chighwire_math%2Chighwire_inline_linked_media" class="colorbox colorbox-load table-expand-popup init-colorbox-processed cboxElement" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6.</span> <p id="p-60" class="first-child"><em>Class prediction modeling of test data set based on cross-validation results (<a id="xref-table-wrap-5-4" class="xref-table" href="#T5">Table 5</a>) and KNN and SVM algorithms</em></p><div class="sb-div caption-clear"></div></div></div>
          <p id="p-20">We further tested the relevance of gene list A, 
to the test data set, by calculating how many of these genes were 
differentially regulated among the patients with septic shock and the 
controls in the test data set. We conducted a two-group ANOVA 
(Benjamini-Hochberg false discovery rate of 5%) using controls and 
patients with septic shock in the test data set as the comparison groups
 and gene list A. This analysis demonstrated that 2,386 of the 2,482 
genes (96%) were differentially regulated between the controls and the 
patients with septic shock in the test data set.</p>
          <p id="p-21">Collectively, these data demonstrate that the 
genome-wide expression signature originally reported for children with 
septic shock (i.e., gene list A) can accurately identify a separate test
 cohort of children with septic shock and controls with a high degree of
 sensitivity and specificity. The strength of this observation is 
further supported by the application of multiple combinations of class 
prediction algorithms and gene selection methods.</p>
        </div>
        <div id="sec-11" class="subsection">
          <h4>Functional analysis of predictor genes.</h4>
          <p id="p-22">As shown in <a id="xref-table-wrap-5-3" class="xref-table" href="#T5">Tables 5</a> and <a id="xref-table-wrap-6-2" class="xref-table" href="#T6">6</a>,
 the Golub method of gene selection appeared to be the most robust 
method for class prediction in that it was able to predict 100% of the 
septic shock patients and the controls in the test data set, based on 
the KNN class prediction algorithm. Accordingly, we uploaded the 50 
predictor genes derived from the Golub-based class prediction (see <em>Supplementary Data</em><a id="xref-fn-2-1" class="xref-fn" href="#fn-2"><sup>1</sup></a>
  for complete 50 predictor gene list) to the IPA application to 
determine if the gene list corresponded to any specific signaling 
pathways. <a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a>
 demonstrates the top 5 signaling pathways represented within the list 
of 50 predictor genes. All of these signaling pathways are consistent 
with our current paradigms surrounding the pathobiology of septic shock,
 thus supporting the biological plausibility of this 50 predictor gene 
list.</p><div class="table pos-float" id="T7"><div class="table-inline"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="0 first"><a href="http://physiolgenomics.physiology.org/" class="table-expand-inline highwireTablesMarkupProcessor-processed" data-table-url="/highwire/markup/11640/expansion?postprocessors=highwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" html="1" fragment="#" external="1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="1 last"><a href="http://physiolgenomics.physiology.org/highwire/markup/11640/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_figures%2Chighwire_math%2Chighwire_inline_linked_media" class="colorbox colorbox-load table-expand-popup init-colorbox-processed cboxElement" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7.</span> <p id="p-65" class="first-child"><em>Signaling pathways among 50 predictor genes derived from class prediction modeling and the Golub method of gene selection</em></p><div class="sb-div caption-clear"></div></div></div>
        </div>
        <div id="sec-12" class="subsection">
          <h4>Comparison of training and data set gene lists.</h4>
          <p id="p-24">The next validation step involved the generation 
of a list of differentially regulated genes between the patients with 
septic shock and controls in the test data set. To make a direct 
comparison to our previously published data, we applied an identical 
filtering approach as that used to derive the original gene list A (<a rel="#ref-19~.ref-cit" id="xref-ref-19-12" class="xref-bibr article-ref-popup-processed" href="#ref-19">19</a>).
 We conducted a two-group ANOVA (Benjamini-Hochberg false discovery rate
 of 5%) using controls and patients with septic shock in the test data 
set as the comparison groups, and all gene probes within the microarray 
(54,681 gene probes). This statistical filter yielded a working list of 
21,517 gene probes that were differentially regulated between controls 
and patients with septic shock. To further refine this 21,517 gene list,
 we next applied an expression filter that selected only the genes, 
within the above 21,517 gene list, having at least twofold expression 
difference in at least 50% of the patients with septic shock, compared 
with the median of the controls. This expression filter yielded a final 
working list of 3,296 gene probes that were differentially regulated 
between patients with septic shock in the test data set and controls (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>, gene list B).</p>
          <p id="p-25"><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> depicts a Venn analysis comparing <em>gene</em>
 lists A and B. Seventy-five percent of the genes in the training data 
set (gene list A) were present in the test data set (gene list B). These
 data demonstrate that the application of an identical filtering 
strategy, to two separate cohorts of controls and children with septic 
shock, identifies a relatively similar number (&gt;50%) of 
differentially regulated gene transcripts. These data further support 
the existence of an identifiable, characteristic genome-wide expression 
signature in children with septic shock.</p><div id="F1" class="fig pos-float odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="http://physiolgenomics.physiology.org/content/physiolgenomics/34/1/127/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Venn analysis comparing gene lists A and B. See Table 4 and text for gene list derivation." class="fragment-images colorbox-load highwireFiguresMarkupProcessor-processed cboxElement" rel="gallery-fragment-images-106406264" data-figure-caption="Venn analysis comparing gene lists A and B. See Table 4 and text for gene list derivation." data-icon-position="" data-hide-link-title="0"><img class="fragment-image" alt="Fig. 1." src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/F1.gif"></a></div></div><ul class="highwire-figure-links inline"><li class="0 first"><a href="http://physiolgenomics.physiology.org/content/physiolgenomics/34/1/127/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="1"><a href="http://physiolgenomics.physiology.org/content/physiolgenomics/34/1/127/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="2 last"><a href="http://physiolgenomics.physiology.org/highwire/powerpoint/11598" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig. 1.</span> <p id="p-45" class="first-child">Venn analysis comparing gene lists A and B. See <a id="xref-table-wrap-4-5" class="xref-table" href="#T4">Table 4</a> and text for gene list derivation.</p><div class="sb-div caption-clear"></div></div></div>
        </div>
        <div id="sec-13" class="subsection">
          <h4>Alternative gene list derivation strategy in the test data set.</h4>
          <p id="p-26">As previously stated, the particular statistical 
and filtering approaches that are applied to a given microarray data set
 can yield markedly different results (<a rel="#ref-10~.ref-cit" id="xref-ref-10-4" class="xref-bibr article-ref-popup-processed" href="#ref-10">10</a>, <a rel="#ref-17~.ref-cit" id="xref-ref-17-4" class="xref-bibr article-ref-popup-processed" href="#ref-17">17</a>).
 Our original report of differentially regulated genes between controls 
and patients with septic shock was based on sequential statistical and 
expression filters that used the default parameters in the analysis 
software (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>, gene list A) (<a rel="#ref-19~.ref-cit" id="xref-ref-19-13" class="xref-bibr article-ref-popup-processed" href="#ref-19">19</a>).
 Furthermore, the statistical filter used in the initial report 
conducted multiple testing corrections by way of the Benjamini and 
Hochberg false discovery rate.</p>
          <p id="p-27">In this analysis, we generated a list of 
differentially regulated genes between controls and the patients with 
septic shock in the test data set by applying only a statistical filter 
(ANOVA) and conducted multiple testing corrections by way of Bonferroni.
 Compared with the Benjamini and Hochberg false discovery rate, 
Bonferroni-based multiple testing correction has a much higher degree of
 stringency in the context of microarray data and is generally regarded 
as being overly stringent for analyzing microarray data (<a rel="#ref-10~.ref-cit" id="xref-ref-10-5" class="xref-bibr article-ref-popup-processed" href="#ref-10">10</a>, <a rel="#ref-17~.ref-cit" id="xref-ref-17-5" class="xref-bibr article-ref-popup-processed" href="#ref-17">17</a>). Nevertheless, we applied this stringent statistical test to the test data set as another form of validation.</p>
          <p id="p-28">Using this purely statistical approach (i.e., 
without an additional expression filter), we generated a list of 2,104 
gene probes that were differentially regulated between controls and 
patients with septic shock in the test data set (<a id="xref-table-wrap-4-4" class="xref-table" href="#T4">Table 4</a>, gene list C). <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>
 depicts a Venn analysis of gene lists B and C. Forty-one percent of the
 genes in gene list B were present in gene list C. These data illustrate
 how the particular filtering approach that is applied to a given set of
 microarray data can profoundly affect the derivation of putative 
differentially regulated genes.</p><div id="F2" class="fig pos-float odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="http://physiolgenomics.physiology.org/content/physiolgenomics/34/1/127/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Venn analysis comparing gene lists B and C. See Table 4 and text for gene list derivation." class="fragment-images colorbox-load highwireFiguresMarkupProcessor-processed cboxElement" rel="gallery-fragment-images-106406264" data-figure-caption="Venn analysis comparing gene lists B and C. See Table 4 and text for gene list derivation." data-icon-position="" data-hide-link-title="0"><img class="fragment-image" alt="Fig. 2." src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/F2.gif"></a></div></div><ul class="highwire-figure-links inline"><li class="0 first"><a href="http://physiolgenomics.physiology.org/content/physiolgenomics/34/1/127/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="1"><a href="http://physiolgenomics.physiology.org/content/physiolgenomics/34/1/127/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="2 last"><a href="http://physiolgenomics.physiology.org/highwire/powerpoint/11602" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Fig. 2.</span> <p id="p-46" class="first-child">Venn analysis comparing gene lists B and C. See <a id="xref-table-wrap-4-6" class="xref-table" href="#T4">Table 4</a> and text for gene list derivation.</p><div class="sb-div caption-clear"></div></div></div>
        </div>
        <div id="sec-14" class="subsection">
          <h4>Functional analysis of the test data set.</h4>
          <p id="p-29">Having demonstrated the impact of different 
filtering approaches on gene list derivation in our test data set, we 
next conducted functional analyses of the test data set to determine if 
the functional analyses would differ significantly from our original 
observations (<a rel="#ref-13~.ref-cit" id="xref-ref-13-6" class="xref-bibr article-ref-popup-processed" href="#ref-13">13</a>, <a rel="#ref-19~.ref-cit" id="xref-ref-19-14" class="xref-bibr article-ref-popup-processed" href="#ref-19">19</a>).
 All analyses in this section are based on gene list C and were 
conducted in an analogous manner to that of our previous reports.</p>
          <p id="p-30">To derive biological meaning from gene list C, we
 uploaded the individual lists of upregulated and downregulated genes, 
respectively, to both the D.A.V.I.D. database and the IPA application (<a rel="#ref-4~.ref-cit" id="xref-ref-4-2" class="xref-bibr article-ref-popup-processed" href="#ref-4">4</a>, <a rel="#ref-6~.ref-cit" id="xref-ref-6-2" class="xref-bibr article-ref-popup-processed" href="#ref-6">6</a>, <a rel="#ref-13~.ref-cit" id="xref-ref-13-7" class="xref-bibr article-ref-popup-processed" href="#ref-13">13</a>, <a rel="#ref-19~.ref-cit" id="xref-ref-19-15" class="xref-bibr article-ref-popup-processed" href="#ref-19">19</a>). As shown in <a id="xref-table-wrap-8-1" class="xref-table" href="#T8">Tables 8</a> and <a id="xref-table-wrap-9-1" class="xref-table" href="#T9">9</a>, the D.A.V.I.D.-dependent analyses yielded several biological relevant functional annotations within both gene lists. <a id="xref-table-wrap-8-2" class="xref-table" href="#T8">Table 8</a>,
 representing the 846 upregulated genes, is notable for multiple 
functional annotations related to host defense responses similar to that
 of our previous data (<a rel="#ref-13~.ref-cit" id="xref-ref-13-8" class="xref-bibr article-ref-popup-processed" href="#ref-13">13</a>, <a rel="#ref-19~.ref-cit" id="xref-ref-19-16" class="xref-bibr article-ref-popup-processed" href="#ref-19">19</a>). <a id="xref-table-wrap-9-2" class="xref-table" href="#T9">Table 9</a>, representing the 1,258 downregulated genes, is notable for the prevalence of zinc- and metal binding-related ontologies (see <em>Supplementary Data</em> for gene lists corresponding to zinc-related functional annotations), a principal observation of our previous data (<a rel="#ref-13~.ref-cit" id="xref-ref-13-9" class="xref-bibr article-ref-popup-processed" href="#ref-13">13</a>, <a rel="#ref-19~.ref-cit" id="xref-ref-19-17" class="xref-bibr article-ref-popup-processed" href="#ref-19">19</a>).</p><div class="table pos-float" id="T8"><div class="table-inline"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="0 first"><a href="http://physiolgenomics.physiology.org/" class="table-expand-inline highwireTablesMarkupProcessor-processed" data-table-url="/highwire/markup/11642/expansion?postprocessors=highwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" html="1" fragment="#" external="1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="1 last"><a href="http://physiolgenomics.physiology.org/highwire/markup/11642/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_figures%2Chighwire_math%2Chighwire_inline_linked_media" class="colorbox colorbox-load table-expand-popup init-colorbox-processed cboxElement" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 8.</span> <p id="p-67" class="first-child"><em>Top 20 functional annotations among 846 upregulated genes in gene list C, based on P value and listed in descending order</em></p><div class="sb-div caption-clear"></div></div></div><div class="table pos-float" id="T9"><div class="table-inline"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="0 first"><a href="http://physiolgenomics.physiology.org/" class="table-expand-inline highwireTablesMarkupProcessor-processed" data-table-url="/highwire/markup/11648/expansion?postprocessors=highwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" html="1" fragment="#" external="1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="1 last"><a href="http://physiolgenomics.physiology.org/highwire/markup/11648/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_figures%2Chighwire_math%2Chighwire_inline_linked_media" class="colorbox colorbox-load table-expand-popup init-colorbox-processed cboxElement" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 9.</span> <p id="p-69" class="first-child"><em>Top 20 functional annotations among 1,258 downregulated genes in gene list C, based on P value and listed in descending order</em></p><div class="sb-div caption-clear"></div></div></div>
          <p id="p-31">As shown in <a id="xref-table-wrap-10-1" class="xref-table" href="#T10">Tables 10</a> and <a id="xref-table-wrap-11-1" class="xref-table" href="#T11">11</a>, the IPA-dependent analyses yielded several biologically relevant signaling pathways within both gene lists. <a id="xref-table-wrap-10-2" class="xref-table" href="#T10">Table 10</a>, representing the 846 upregulated genes, is dominated by inflammation-and immunity-related signaling pathways (see <em>Supplementary Data</em>
 for gene lists corresponding to individual signaling pathways). This 
observation is consistent with the established literature focused on 
septic shock and our previous data (<a rel="#ref-2~.ref-cit" id="xref-ref-2-1" class="xref-bibr article-ref-popup-processed" href="#ref-2">2</a>, <a rel="#ref-8~.ref-cit" id="xref-ref-8-1" class="xref-bibr article-ref-popup-processed" href="#ref-8">8</a>, <a rel="#ref-13~.ref-cit" id="xref-ref-13-10" class="xref-bibr article-ref-popup-processed" href="#ref-13">13</a>, <a rel="#ref-14~.ref-cit" id="xref-ref-14-2" class="xref-bibr article-ref-popup-processed" href="#ref-14">14</a>, <a rel="#ref-19~.ref-cit" id="xref-ref-19-18" class="xref-bibr article-ref-popup-processed" href="#ref-19">19</a>). <a id="xref-table-wrap-11-2" class="xref-table" href="#T11">Table 11</a>,
 representing the 1,258 downregulated genes, is most notable for 
repression of genes corresponding to natural killer cell signaling, T 
cell receptor signaling, and antigen presentation (see <em>Supplementary Data</em> for gene lists corresponding to individual signaling pathways), which is the other principal observation of our previous data (<a rel="#ref-13~.ref-cit" id="xref-ref-13-11" class="xref-bibr article-ref-popup-processed" href="#ref-13">13</a>, <a rel="#ref-19~.ref-cit" id="xref-ref-19-19" class="xref-bibr article-ref-popup-processed" href="#ref-19">19</a>).</p><div class="table pos-float" id="T10"><div class="table-inline"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="0 first"><a href="http://physiolgenomics.physiology.org/" class="table-expand-inline highwireTablesMarkupProcessor-processed" data-table-url="/highwire/markup/11610/expansion?postprocessors=highwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" html="1" fragment="#" external="1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="1 last"><a href="http://physiolgenomics.physiology.org/highwire/markup/11610/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_figures%2Chighwire_math%2Chighwire_inline_linked_media" class="colorbox colorbox-load table-expand-popup init-colorbox-processed cboxElement" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 10.</span> <p id="p-71" class="first-child"><em>Signaling pathways among 846 upregulated genes in gene list C</em></p><div class="sb-div caption-clear"></div></div></div><div class="table pos-float" id="T11"><div class="table-inline"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="0 first"><a href="http://physiolgenomics.physiology.org/" class="table-expand-inline highwireTablesMarkupProcessor-processed" data-table-url="/highwire/markup/11614/expansion?postprocessors=highwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" html="1" fragment="#" external="1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="1 last"><a href="http://physiolgenomics.physiology.org/highwire/markup/11614/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_figures%2Chighwire_math%2Chighwire_inline_linked_media" class="colorbox colorbox-load table-expand-popup init-colorbox-processed cboxElement" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 11.</span> <p id="p-73" class="first-child"><em>Signaling pathways among the 1,258 downregulated genes in gene list C</em></p><div class="sb-div caption-clear"></div></div></div>
          <p id="p-32">Collectively, these functional analyses 
demonstrate that gene list C, derived in a completely different manner 
to that of our previous data, and representing a test cohort of 
patients, contains a large number of genes relevant to the pathobiology 
of septic shock and validate our previous observations.</p>
        </div>
      </div>
    </div><div class="section" id="sec-15">
      <h2>DISCUSSION</h2>
      <p id="p-33">Our initial reports involving the genome-level 
expression profiles of pediatric septic shock were primarily highlighted
 by three observations (<a rel="#ref-13~.ref-cit" id="xref-ref-13-12" class="xref-bibr article-ref-popup-processed" href="#ref-13">13</a>, <a rel="#ref-18~.ref-cit" id="xref-ref-18-4" class="xref-bibr article-ref-popup-processed" href="#ref-18">18</a>, <a rel="#ref-19~.ref-cit" id="xref-ref-19-20" class="xref-bibr article-ref-popup-processed" href="#ref-19">19</a>): <em>1</em>)
 pediatric septic shock is characterized by large-scale repression of 
genes that either depend on normal zinc homeostasis for normal function 
or directly participate in zinc homeostasis, and this observation is 
associated with abnormally low serum zinc concentrations in nonsurvivors
 of septic shock; <em>2</em>) pediatric septic shock is also 
characterized by large-scale repression of genes corresponding to 
adaptive immunity, T cell function in particular; and <em>3</em>) 
longitudinal studies demonstrated that both of these gene repression 
patterns persist over the first 3 days of illness. While interesting and
 holding the potential to change our current paradigms regarding the 
pathobiology of pediatric septic shock, these observations require 
validation at several levels. In an ongoing approach we are currently 
seeking functional validation of these observations by conducting 
laboratory-based experiments focused on altered zinc homeostasis in the 
context of experimental septic shock and are also conducting functional 
studies of lymphocyte function in children with septic shock. In the 
current work, we have attempted to complement our efforts at validation 
through a bioinformatics approach.</p>
      <p id="p-34">The focal point of our bioinformatic approach to 
validation was a test data set. The test data set consisted of an 
entirely new set of children with a clinical diagnosis of septic shock 
and an entirely new set of controls. The test data set patients were 
similar to the patients in our original report with respect to age and 
illness severity. In contrast, the test data set appeared to be more 
homogenous than the original cohort of patients with respect to class of
 infecting organisms. In addition, the patients with septic shock in the
 test data set had lower median absolute lymphocyte counts than the 
patients in the training data set. These potentially confounding factors
 were the result of our strategy to select patients for the test data 
set based on chronology of enrollment into the data base. Despite these 
potentially confounding factors, the key aspects of our original data 
were validated in the test data set.</p>
      <p id="p-35">One of the major strategies used to validate our 
previous data involved cross-validation procedures on the training data 
set, then subsequently conducting class prediction procedures on the 
test data set. The intent of this strategy is not to gain the ability to
 identify normal vs. septic shock in the clinical setting, which can be 
readily achieved by basic clinical examination and laboratory 
parameters. Rather, these procedures were conducted as a primary test of
 the validity of our original data. Using multiple combinations of class
 prediction algorithms and gene selection methods, we were able to 
accurately identify â¥95% of the subjects in the test data set, depending
 on the class prediction algorithm and the gene selection method. In 
fact, using the KNN algorithm and the 50 predictor genes derived from 
the Golub method of gene selection, we were able to accurately identify 
all controls and patients with septic shock in the test data set. One 
hundred percent prediction accuracy was also achieved using the SVM 
algorithm and all genes from the training data set (gene list A). In 
keeping with these observations, the majority of genes in the previously
 reported gene list were differentially regulated in the test data set 
as determined by direct statistical analysis of these genes in the 
context of the test data set. Finally, the 50 predictor genes derived 
from class prediction modeling correspond to biologically relevant 
signaling pathways in the context of septic shock. In combination, these
 data indicate that the previously reported genome-wide expression 
signature of pediatric septic shock is applicable to a test data set of 
children with septic shock and therefore speak to the validity of the 
original data.</p>
      <p id="p-36">Our current data also well illustrate how different 
filtering approaches can affect the derivation of differentially 
regulated gene lists. Using two different filtering strategies (i.e., 
gene lists B and C) we generated two putative lists of genes in the test
 data set that were differentially regulated between controls and 
patients with septic shock in the test data set. Comparison of these two
 gene lists demonstrated &lt;50% concordance, thus raising questions of 
trustworthiness. Accordingly, we submitted the gene list having the 
least concordance with our previous data (i.e., gene list C) for 
derivation of functional annotations and signaling pathway associations.
 These analyses yielded the majority of the key functional annotations 
and signaling pathways that were previously reported (<a rel="#ref-13~.ref-cit" id="xref-ref-13-13" class="xref-bibr article-ref-popup-processed" href="#ref-13">13</a>, <a rel="#ref-18~.ref-cit" id="xref-ref-18-5" class="xref-bibr article-ref-popup-processed" href="#ref-18">18</a>, <a rel="#ref-19~.ref-cit" id="xref-ref-19-21" class="xref-bibr article-ref-popup-processed" href="#ref-19">19</a>).
 Thus, elucidation and discovery of the key genes coordinately regulated
 in the context of pediatric septic shock (i.e., the genes that 
contribute most to enrichment of functional annotations and signaling 
pathways) may not be highly dependent on the filtering approach. 
Consequently, these data strongly support the observations made in our 
previous reports.</p>
      <p id="p-37">The main limitation of our current work is the 
reliance on whole blood-derived RNA for gene expression profiling, as 
previously discussed at length (<a rel="#ref-13~.ref-cit" id="xref-ref-13-14" class="xref-bibr article-ref-popup-processed" href="#ref-13">13</a>, <a rel="#ref-19~.ref-cit" id="xref-ref-19-22" class="xref-bibr article-ref-popup-processed" href="#ref-19">19</a>).
 Whole blood-derived RNA reflects a mixed population of white blood 
cells, which has the potential to profoundly confound the resulting 
microarray data. Our previous data, however, indicate that whole 
blood-derived RNA can yield consistent and biologically plausible 
microarray data in children with septic shock. For example, we have 
previously demonstrated that variation in the absolute number of 
lymphocytes does not account for the widespread repression of genes 
related to adaptive immunity that we have reported in children with 
septic shock (<a rel="#ref-13~.ref-cit" id="xref-ref-13-15" class="xref-bibr article-ref-popup-processed" href="#ref-13">13</a>).</p>
      <p id="p-38">The potential limitations of using whole 
blood-derived RNA are now further mitigated by the current data focused 
on validation in a test data set. For example, the ability to identify 
controls and patients with septic shock in the current test data set, 
with 100% accuracy, well supports the concept that whole blood-derived 
RNA can yield biologically plausible and consistent data in pediatric 
septic shock. In addition, the ability to derive functional annotations 
and signaling pathways in the test data set, which are analogous to that
 of our previous reports, further supports the assertion that using 
whole blood-derived RNA is a valid approach. Importantly, these 
functional annotation and signaling pathway data were generated in the 
context of two potentially strong negative confounders: <em>1</em>) the 
data were based on a potentially overly stringent statistical approach 
(i.e., Bonferroni correction for multiple comparisons), and <em>2</em>) the patients with septic shock in the test data set had lower median lymphocyte counts than that of the training data set.</p>
      <p id="p-39">In conclusion, we have demonstrated that the 
previously reported genome-wide expression signature of pediatric septic
 shock is applicable to a test cohort of children with septic shock at 
multiple levels. We have also validated, through an entirely different 
filtering approach, that <em>day 1</em> of pediatric septic shock is 
characterized by large-scale repression of zinc- and lymphocyte-related 
genes. Given the importance of zinc homeostasis to normal lymphocyte 
function, these data have formed a hypothesis surrounding altered zinc 
homeostasis leading to altered lymphocyte function in children with 
septic shock as previously proposed (<a rel="#ref-18~.ref-cit" id="xref-ref-18-6" class="xref-bibr article-ref-popup-processed" href="#ref-18">18</a>). This hypothesis is readily testable at the translational and experimental levels.</p>
    </div><div class="section" id="sec-16">
      <h2>CONTRIBUTORS</h2>
      <p id="p-40">H. R. Wong and T. P. Shanley conceived the original 
genomic database focused on pediatric septic shock. H. R. Wong directed 
the overall study, including analysis of the data. H. R. Wong had full 
access to all of the data in the study and takes responsibility for the 
integrity of the data and the accuracy of the data analysis. N. 
Cvijanovich, T. P. Shanley, R. Lin, G. L. Allen, N. J. Thomas, P. 
Checchia, N. Anas, and R. J. Freishtat serve as the local principal 
investigators at the sites contributing the largest number of patients 
to the database. M. Monaco, and K. Odoms manage the clinical database 
and all clinical samples generated through this program. B. Sakthivel 
provides bioinformatic support to the program. H. R. Wong, N. 
Cvijanovich, and T. P. Shanley drafted the manuscript. All authors 
reviewed and helped revise the manuscript and approved the final version
 of the manuscript.</p>
    </div><div class="section" id="sec-17">
      <h2>GRANTS</h2>
      <p id="p-41">This work was supported by National Institute of General Medical Sciences Grant RO1 GM-064619.</p>
    </div><div class="section ack" id="ack-1"><h2>Acknowledgments</h2>
      <p id="p-44">Additional Genomics of Pediatric SIRS/Septic Shock 
Investigators: Allan Doctor (St. Louis Children's Hospital, Washington 
University School of Medicine, St. Louis, MO); Douglas Willson 
(University of Virginia, Charlottesville, VA); Meena Kalyanarman (Newark
 Beth Israel Medical Center, Newark, NJ); Nancy M. Tofil (The University
 of Alabama at Birmingham, Birmingham, AL); Scott Penfil (DuPont 
Hospital for Children, Wilmington, DE ); Julie Simon (Children's 
Hospital and Research Center Oakland, Oakland, CA); Joseph Hess (Penn 
State Children's Hospital, Hershey, PA); Margaret Winkler (The 
University of Alabama at Birmingham, Birmingham, AL); Gwenn McLaughlin, 
MD (Jackson Memorial Hospital, Miami, FL); Cheri Landers (Kentucky 
Children's Hospital, Lexington, KY); Gary Kohn, MD (Morristown Memorial 
Hospital, Morristown, NJ).</p>
    </div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">âµ</a><span class="fn-label">*</span> Additional members of the Genomics of Pediatric SIRS/Septic Shock Investigators appear in the <span class="sc">acknowledgments</span>.</p></li><li class="fn" id="fn-2"><p id="p-23"><a class="rev-xref" href="#xref-fn-2-1">âµ</a><span class="fn-label">1</span> The online version of this article contains supplemental material.</p></li><li class="fn fn-group-article-title" id="fn-3"><p id="p-42">Address
 for reprint requests and other correspondence: H. R. Wong, Div. of 
Critical Care Medicine, Cincinnati Children's Hospital Medical Center, 
3333 Burnet Ave., Cincinnati, OH 45229-3039 (e-mail: <span class="em-link"><span class="em-addr"><a href="mailto:hector.wong@cchmc.org">hector.wong@cchmc.org</a></span></span>).</p><p id="p-43">The
 costs of publication of this article were defrayed in part by the 
payment of page charges. The article must therefore be hereby marked â<em>advertisement</em>â in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.</p></li></ul></div><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2008 the American Physiological Society</li></ul><div class="section ref-list" id="ref-list-1"><h2>REFERENCES</h2><ol class="cit-list"><li><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference 1 in text" id="ref-1">âµ</a>
        
        <div class="cit ref-cit ref-other" id="cit-34.1.127.1" data-doi="10.1038/nrg1749"><div class="cit-metadata"><cite><span class="named-content arthw-firstauthor" id="named-content-1"><span class="named-content arthw-snm" id="named-content-2">Allison</span> <span class="named-content arthw-fnm" id="named-content-3">DB</span></span><strong>, Cui X, Page GP, Sabripour M.</strong> Microarray data analysis: from disarray to consolidation and consensus. <span class="cit-source">Nat Rev Genet</span> <span class="cit-vol">7</span>: <span class="cit-fpage">55</span>â65, <span class="cit-pub-date">2006</span>.</cite></div><div class="cit-extra"><a href="http://physiolgenomics.physiology.org/highwire/openurl?query=rft.jtitle=Nature%20reviews.%20Genetics&amp;rft.stitle=Nat%20Rev%20Genet&amp;rft.aulast=Jha&amp;rft.auinit1=A.&amp;rft.volume=7&amp;rft.issue=1&amp;rft.spage=55&amp;rft.epage=65&amp;rft.atitle=Microarray%20data%20analysis:%20from%20disarray%20to%20consolidation%20and%20consensus.&amp;rft_id=info:doi/10.1038/nrg1749&amp;rft_id=info:pmid/16369572&amp;rft.genre=article&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;ctx_ver=Z39.88-2004&amp;url_ver=Z39.88-2004&amp;url_ctx_fmt=info:ofi/fmt:kev:mtx:ctx&amp;redirect_url=http://gw2jh3xr2c.search.serialssolutions.com" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url insertImage-processed"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/5510b14b-ba7f-4b26-b710-c206bd73a987-20120614.jpeg"> </a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=10.1038/nrg1749&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=16369572&amp;link_type=MED&amp;atom=%2Fphysiolgenomics%2F34%2F1%2F127.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>Medline</span></a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=000234139800016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div>
      </li><li><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference 2 in text" id="ref-2">âµ</a>
        
        <div class="cit ref-cit ref-other" id="cit-34.1.127.2" data-doi="10.1097/00024382-199610001-00007"><div class="cit-metadata"><cite><span class="named-content arthw-firstauthor" id="named-content-4"><span class="named-content arthw-snm" id="named-content-5">Ayala</span> <span class="named-content arthw-fnm" id="named-content-6">A</span></span><strong>, Chaudry IH.</strong> Immune dysfunction in murine polymicrobial sepsis: mediators, macrophages, lymphocytes and apoptosis. <span class="cit-source">Shock</span> <span class="cit-vol">6</span>, <em>Suppl</em> 1: <span class="cit-fpage">S27</span>âS38, <span class="cit-pub-date">1996</span>.</cite></div><div class="cit-extra"><a href="http://physiolgenomics.physiology.org/highwire/openurl?query=rft.jtitle=Shock&amp;rft.volume=6&amp;rft.spage=S27&amp;rft_id=info:doi/10.1097/00024382-199610001-00007&amp;rft_id=info:pmid/8828095&amp;rft.genre=article&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;ctx_ver=Z39.88-2004&amp;url_ver=Z39.88-2004&amp;url_ctx_fmt=info:ofi/fmt:kev:mtx:ctx&amp;redirect_url=http://gw2jh3xr2c.search.serialssolutions.com" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url insertImage-processed"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/5510b14b-ba7f-4b26-b710-c206bd73a987-20120614.jpeg"> </a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=10.1097/00024382-199610001-00007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=8828095&amp;link_type=MED&amp;atom=%2Fphysiolgenomics%2F34%2F1%2F127.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>Medline</span></a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=A1996UY63300007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div>
      </li><li><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference 3 in text" id="ref-3">âµ</a>
        
        <div class="cit ref-cit ref-other" id="cit-34.1.127.3"><div class="cit-metadata"><cite><span class="named-content arthw-firstauthor" id="named-content-7"><span class="named-content arthw-snm" id="named-content-8">Byvatov</span> <span class="named-content arthw-fnm" id="named-content-9">E</span></span><strong>, Schneider G.</strong> Support vector machine applications in bioinformatics. <span class="cit-source">Appl Bioinformatics</span> <span class="cit-vol">2</span>: <span class="cit-fpage">67</span>â77, <span class="cit-pub-date">2003</span>.</cite></div><div class="cit-extra"><a href="http://physiolgenomics.physiology.org/highwire/openurl?query=rft.jtitle=Applied%20bioinformatics&amp;rft.stitle=Appl%20Bioinformatics&amp;rft.aulast=Byvatov&amp;rft.auinit1=E.&amp;rft.volume=2&amp;rft.issue=2&amp;rft.spage=67&amp;rft.epage=77&amp;rft.atitle=Support%20vector%20machine%20applications%20in%20bioinformatics.&amp;rft_id=info:pmid/15130823&amp;rft.genre=article&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;ctx_ver=Z39.88-2004&amp;url_ver=Z39.88-2004&amp;url_ctx_fmt=info:ofi/fmt:kev:mtx:ctx&amp;redirect_url=http://gw2jh3xr2c.search.serialssolutions.com" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url insertImage-processed"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/5510b14b-ba7f-4b26-b710-c206bd73a987-20120614.jpeg"> </a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=15130823&amp;link_type=MED&amp;atom=%2Fphysiolgenomics%2F34%2F1%2F127.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>Medline</span></a></div></div>
      </li><li><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference 4 in text" id="ref-4">âµ</a>
        
        <div class="cit ref-cit ref-other" id="cit-34.1.127.4" data-doi="10.1038/nature03985"><div class="cit-metadata"><cite><span class="named-content arthw-firstauthor" id="named-content-10"><span class="named-content arthw-snm" id="named-content-11">Calvano</span> <span class="named-content arthw-fnm" id="named-content-12">SE</span></span><strong>,
 Xiao W, Richards DR, Felciano RM, Baker HV, Cho RJ, Chen RO, Brownstein
 BH, Cobb JP, Tschoeke SK, Miller-Graziano C, Moldawer LL, Mindrinos MN,
 Davis RW, Tompkins RG, Lowry SF.</strong> A network-based analysis of systemic inflammation in humans. <span class="cit-source">Nature</span> <span class="cit-vol">437</span>: <span class="cit-fpage">1032</span>â1037, <span class="cit-pub-date">2005</span>.</cite></div><div class="cit-extra"><a href="http://physiolgenomics.physiology.org/highwire/openurl?query=rft.jtitle=Nature;%20Physical%20Science%20%28London%29&amp;rft.stitle=Nature;%20Physical%20Science%20%28London%29&amp;rft.aulast=Calvano&amp;rft.auinit1=S.%20E.&amp;rft.volume=437&amp;rft.issue=7061&amp;rft.spage=1032&amp;rft.epage=1037&amp;rft.atitle=A%20network-based%20analysis%20of%20systemic%20inflammation%20in%20humans.&amp;rft_id=info:doi/10.1038/nature03985&amp;rft_id=info:pmid/16136080&amp;rft.genre=article&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;ctx_ver=Z39.88-2004&amp;url_ver=Z39.88-2004&amp;url_ctx_fmt=info:ofi/fmt:kev:mtx:ctx&amp;redirect_url=http://gw2jh3xr2c.search.serialssolutions.com" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url insertImage-processed"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/5510b14b-ba7f-4b26-b710-c206bd73a987-20120614.jpeg"> </a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=10.1038/nature03985&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=16136080&amp;link_type=MED&amp;atom=%2Fphysiolgenomics%2F34%2F1%2F127.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>Medline</span></a></div></div>
      </li><li><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference 5 in text" id="ref-5">âµ</a>
        
        <div class="cit ref-cit ref-other" id="cit-34.1.127.5" data-doi="10.1073/pnas.0409768102"><div class="cit-metadata"><cite><span class="named-content arthw-firstauthor" id="named-content-13"><span class="named-content arthw-snm" id="named-content-14">Cobb</span> <span class="named-content arthw-fnm" id="named-content-15">JP</span></span><strong>,
 Mindrinos MN, Miller-Graziano C, Calvano SE, Baker HV, Xiao W, 
Laudanski K, Brownstein BH, Elson CM, Hayden DL, Herndon DN, Lowry SF, 
Maier RV, Schoenfeld DA, Moldawer LL, Davis RW, Tompkins RG, Bankey P, 
Billiar T, Camp D, Chaudry I, Freeman B, Gamelli R, Gibran N, Harbrecht 
B, Heagy W, Heimbach D, Horton J, Hunt J, Lederer J, Mannick J, McKinley
 B, Minei J, Moore E, Moore F, Munford R, Nathens A, O'Keefe G, Purdue 
G, Rahme L, Remick D, Sailors M, Shapiro M, Silver G, Smith R, 
Stephanopoulos G, Stormo G, Toner M, Warren S, West M, Wolfe S, Young V.</strong> Application of genome-wide expression analysis to human health and disease. <span class="cit-source">Proc Natl Acad Sci USA</span> <span class="cit-vol">102</span>: <span class="cit-fpage">4801</span>â4806, <span class="cit-pub-date">2005</span>.</cite></div><div class="cit-extra"><a href="http://physiolgenomics.physiology.org/highwire/openurl?query=rft.jtitle=Proc%20Natl%20Acad%20Sci%20USA&amp;rft_id=info:doi/10.1073/pnas.0409768102&amp;rft_id=info:pmid/15781863&amp;rft.genre=article&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;ctx_ver=Z39.88-2004&amp;url_ver=Z39.88-2004&amp;url_ctx_fmt=info:ofi/fmt:kev:mtx:ctx&amp;redirect_url=http://gw2jh3xr2c.search.serialssolutions.com" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url insertImage-processed"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/5510b14b-ba7f-4b26-b710-c206bd73a987-20120614.jpeg"> </a><a href="http://physiolgenomics.physiology.org/lookup/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=102/13/4801&amp;atom=%2Fphysiolgenomics%2F34%2F1%2F127.atom" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div>
      </li><li><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference 6 in text" id="ref-6">âµ</a>
        
        <div class="cit ref-cit ref-other" id="cit-34.1.127.6" data-doi="10.1186/gb-2003-4-5-p3"><div class="cit-metadata"><cite><span class="named-content arthw-firstauthor" id="named-content-16"><span class="named-content arthw-snm" id="named-content-17">Dennis</span> <span class="named-content arthw-fnm" id="named-content-18">G</span></span><strong> Jr, Sherman BT, Hosack DA, Yang J, Gao W, Lane HC, Lempicki RA.</strong> DAVID: Database for Annotation, Visualization, and Integrated Discovery. <span class="cit-source">Genome Biol</span> <span class="cit-vol">4</span>: <span class="cit-fpage">P3</span>, <span class="cit-pub-date">2003</span>.</cite></div><div class="cit-extra"><a href="http://physiolgenomics.physiology.org/highwire/openurl?query=rft.jtitle=Genome%20biology&amp;rft.stitle=Genome%20Biol&amp;rft.aulast=Dennis&amp;rft.auinit1=G.&amp;rft.volume=4&amp;rft.issue=5&amp;rft.spage=P3&amp;rft.epage=P3&amp;rft.atitle=DAVID:%20Database%20for%20Annotation,%20Visualization,%20and%20Integrated%20Discovery.&amp;rft_id=info:doi/10.1186/gb-2003-4-5-p3&amp;rft_id=info:pmid/12734009&amp;rft.genre=article&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;ctx_ver=Z39.88-2004&amp;url_ver=Z39.88-2004&amp;url_ctx_fmt=info:ofi/fmt:kev:mtx:ctx&amp;redirect_url=http://gw2jh3xr2c.search.serialssolutions.com" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url insertImage-processed"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/5510b14b-ba7f-4b26-b710-c206bd73a987-20120614.jpeg"> </a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=10.1186/gb-2003-4-5-p3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=12734009&amp;link_type=MED&amp;atom=%2Fphysiolgenomics%2F34%2F1%2F127.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>Medline</span></a></div></div>
      </li><li><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference 7 in text" id="ref-7">âµ</a>
        
        <div class="cit ref-cit ref-other" id="cit-34.1.127.7" data-doi="10.1097/01.PCC.0000149131.72248.E6"><div class="cit-metadata"><cite><span class="named-content arthw-firstauthor" id="named-content-19"><span class="named-content arthw-snm" id="named-content-20">Goldstein</span> <span class="named-content arthw-fnm" id="named-content-21">B</span></span><strong>, Giroir B, Randolph A.</strong> International pediatric sepsis consensus conference: definitions for sepsis and organ dysfunction in pediatrics. <span class="cit-source">Pediatr Crit Care Med</span> <span class="cit-vol">6</span>: <span class="cit-fpage">2</span>â8, <span class="cit-pub-date">2005</span>.</cite></div><div class="cit-extra"><a href="http://physiolgenomics.physiology.org/highwire/openurl?query=rft.jtitle=Pediatric%20critical%20care%20medicine%20:%20%20a%20journal%20of%20the%20Society%20of%20Critical%20Care%20Medicine%20and%20the%20World%20Federation%20of%20Pediatric%20Intensive%20and%20Critical%20Care%20Societies&amp;rft.stitle=Pediatr%20Crit%20Care%20Med&amp;rft.aulast=Goldstein&amp;rft.auinit1=B.&amp;rft.volume=6&amp;rft.issue=1&amp;rft.spage=2&amp;rft.epage=8&amp;rft.atitle=International%20pediatric%20sepsis%20consensus%20conference:%20definitions%20for%20sepsis%20and%20organ%20dysfunction%20in%20pediatrics.&amp;rft_id=info:doi/10.1097/01.PCC.0000149131.72248.E6&amp;rft_id=info:pmid/15636651&amp;rft.genre=article&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;ctx_ver=Z39.88-2004&amp;url_ver=Z39.88-2004&amp;url_ctx_fmt=info:ofi/fmt:kev:mtx:ctx&amp;redirect_url=http://gw2jh3xr2c.search.serialssolutions.com" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url insertImage-processed"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/5510b14b-ba7f-4b26-b710-c206bd73a987-20120614.jpeg"> </a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=10.1097/01.PCC.0000149131.72248.E6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=15636651&amp;link_type=MED&amp;atom=%2Fphysiolgenomics%2F34%2F1%2F127.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>Medline</span></a></div></div>
      </li><li><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference 8 in text" id="ref-8">âµ</a>
        
        <div class="cit ref-cit ref-other" id="cit-34.1.127.8" data-doi="10.1056/NEJMra021333"><div class="cit-metadata"><cite><span class="named-content arthw-firstauthor" id="named-content-22"><span class="named-content arthw-snm" id="named-content-23">Hotchkiss</span> <span class="named-content arthw-fnm" id="named-content-24">RS</span></span><strong>, Karl IE.</strong> The pathophysiology and treatment of sepsis. <span class="cit-source">N Engl J Med</span> <span class="cit-vol">348</span>: <span class="cit-fpage">138</span>â150, <span class="cit-pub-date">2003</span>.</cite></div><div class="cit-extra"><a href="http://physiolgenomics.physiology.org/highwire/openurl?query=rft.jtitle=New%20England%20Journal%20of%20Medicine&amp;rft.stitle=NEJM&amp;rft.issn=0028-4793&amp;rft.aulast=Hotchkiss&amp;rft.auinit1=R.%20S.&amp;rft.volume=348&amp;rft.issue=2&amp;rft.spage=138&amp;rft.epage=150&amp;rft.atitle=The%20Pathophysiology%20and%20Treatment%20of%20Sepsis&amp;rft_id=info:doi/10.1056/NEJMra021333&amp;rft_id=info:pmid/12519925&amp;rft.genre=article&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;ctx_ver=Z39.88-2004&amp;url_ver=Z39.88-2004&amp;url_ctx_fmt=info:ofi/fmt:kev:mtx:ctx&amp;redirect_url=http://gw2jh3xr2c.search.serialssolutions.com" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url insertImage-processed"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/5510b14b-ba7f-4b26-b710-c206bd73a987-20120614.jpeg"> </a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=10.1056/NEJMra021333&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=12519925&amp;link_type=MED&amp;atom=%2Fphysiolgenomics%2F34%2F1%2F127.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>Medline</span></a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=000180263900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div>
      </li><li><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference 9 in text" id="ref-9">âµ</a>
        
        <div class="cit ref-cit ref-other" id="cit-34.1.127.9" data-doi="10.1093/biostatistics/4.2.249"><div class="cit-metadata"><cite><span class="named-content arthw-firstauthor" id="named-content-25"><span class="named-content arthw-snm" id="named-content-26">Irizarry</span> <span class="named-content arthw-fnm" id="named-content-27">RA</span></span><strong>, Hobbs B, Collin F, Beazer-Barclay YD, Antonellis KJ, Scherf U, Speed TP.</strong> Exploration, normalization, and summaries of high density oligonucleotide array probe level data. <span class="cit-source">Biostatistics</span> <span class="cit-vol">4</span>: <span class="cit-fpage">249</span>â264, <span class="cit-pub-date">2003</span>.</cite></div><div class="cit-extra"><a href="http://physiolgenomics.physiology.org/highwire/openurl?query=rft.jtitle=Biostatistics&amp;rft.stitle=Biostatistics&amp;rft.issn=1465-4644&amp;rft.aulast=Irizarry&amp;rft.auinit1=R.%20A.&amp;rft.volume=4&amp;rft.issue=2&amp;rft.spage=249&amp;rft.epage=264&amp;rft.atitle=Exploration,%20normalization,%20and%20summaries%20of%20high%20density%20oligonucleotide%20array%20probe%20level%20data&amp;rft_id=info:doi/10.1093/biostatistics/4.2.249&amp;rft_id=info:pmid/12925520&amp;rft.genre=article&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;ctx_ver=Z39.88-2004&amp;url_ver=Z39.88-2004&amp;url_ctx_fmt=info:ofi/fmt:kev:mtx:ctx&amp;redirect_url=http://gw2jh3xr2c.search.serialssolutions.com" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url insertImage-processed"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/5510b14b-ba7f-4b26-b710-c206bd73a987-20120614.jpeg"> </a><a href="http://physiolgenomics.physiology.org/lookup/ijlink?linkType=ABST&amp;journalCode=biosts&amp;resid=4/2/249&amp;atom=%2Fphysiolgenomics%2F34%2F1%2F127.atom" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div>
      </li><li><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference 10 in text" id="ref-10">âµ</a>
        
        <div class="cit ref-cit ref-other" id="cit-34.1.127.10" data-doi="10.1016/S0168-9525(02)02665-3"><div class="cit-metadata"><cite><span class="named-content arthw-firstauthor" id="named-content-28"><span class="named-content arthw-snm" id="named-content-29">Nadon</span> <span class="named-content arthw-fnm" id="named-content-30">R</span></span><strong>, Shoemaker J.</strong> Statistical issues with microarrays: processing and analysis. <span class="cit-source">Trends Genet</span> <span class="cit-vol">18</span>: <span class="cit-fpage">265</span>â271, <span class="cit-pub-date">2002</span>.</cite></div><div class="cit-extra"><a href="http://physiolgenomics.physiology.org/highwire/openurl?query=rft.jtitle=Trends%20in%20genetics%20:%20%20TIG&amp;rft.stitle=Trends%20Genet&amp;rft.aulast=Nadon&amp;rft.auinit1=R.&amp;rft.volume=18&amp;rft.issue=5&amp;rft.spage=265&amp;rft.epage=271&amp;rft.atitle=Statistical%20issues%20with%20microarrays:%20processing%20and%20analysis.&amp;rft_id=info:doi/10.1016/S0168-9525%2802%2902665-3&amp;rft_id=info:pmid/12047952&amp;rft.genre=article&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;ctx_ver=Z39.88-2004&amp;url_ver=Z39.88-2004&amp;url_ctx_fmt=info:ofi/fmt:kev:mtx:ctx&amp;redirect_url=http://gw2jh3xr2c.search.serialssolutions.com" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url insertImage-processed"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/5510b14b-ba7f-4b26-b710-c206bd73a987-20120614.jpeg"> </a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=10.1016/S0168-9525%2802%2902665-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=12047952&amp;link_type=MED&amp;atom=%2Fphysiolgenomics%2F34%2F1%2F127.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>Medline</span></a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=000175235200011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div>
      </li><li><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference 11 in text" id="ref-11">âµ</a>
        
        <div class="cit ref-cit ref-other" id="cit-34.1.127.11" data-doi="10.1016/S0022-3476(97)70065-9"><div class="cit-metadata"><cite><span class="named-content arthw-firstauthor" id="named-content-31"><span class="named-content arthw-snm" id="named-content-32">Pollack</span> <span class="named-content arthw-fnm" id="named-content-33">MM</span></span><strong>, Patel KM, Ruttimann UE.</strong>
 The Pediatric Risk of Mortality IIIâAcute Physiology Score (PRISM 
III-APS): a method of assessing physiologic instability for pediatric 
intensive care unit patients. <span class="cit-source">J Pediatr</span> <span class="cit-vol">131</span>: <span class="cit-fpage">575</span>â581, <span class="cit-pub-date">1997</span>.</cite></div><div class="cit-extra"><a href="http://physiolgenomics.physiology.org/highwire/openurl?query=rft.jtitle=The%20Journal%20of%20pediatrics&amp;rft.stitle=J%20Pediatr&amp;rft.aulast=Pollack&amp;rft.auinit1=M.%20M.&amp;rft.volume=131&amp;rft.issue=4&amp;rft.spage=575&amp;rft.epage=581&amp;rft.atitle=The%20Pediatric%20Risk%20of%20Mortality%20III--Acute%20Physiology%20Score%20%28PRISM%20III-APS%29:%20a%20method%20of%20assessing%20physiologic%20instability%20for%20pediatric%20intensive%20care%20unit%20patients.&amp;rft_id=info:doi/10.1016/S0022-3476%2897%2970065-9&amp;rft_id=info:pmid/9386662&amp;rft.genre=article&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;ctx_ver=Z39.88-2004&amp;url_ver=Z39.88-2004&amp;url_ctx_fmt=info:ofi/fmt:kev:mtx:ctx&amp;redirect_url=http://gw2jh3xr2c.search.serialssolutions.com" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url insertImage-processed"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/5510b14b-ba7f-4b26-b710-c206bd73a987-20120614.jpeg"> </a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=10.1016/S0022-3476%2897%2970065-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=9386662&amp;link_type=MED&amp;atom=%2Fphysiolgenomics%2F34%2F1%2F127.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>Medline</span></a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=A1997YG84300018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div>
      </li><li><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference 12 in text" id="ref-12">âµ</a>
        
        <div class="cit ref-cit ref-other" id="cit-34.1.127.12" data-doi="10.1016/j.it.2006.11.005"><div class="cit-metadata"><cite><span class="named-content arthw-firstauthor" id="named-content-34"><span class="named-content arthw-snm" id="named-content-35">Rink</span> <span class="named-content arthw-fnm" id="named-content-36">L</span></span><strong>, Haase H.</strong> Zinc homeostasis and immunity. <span class="cit-source">Trends Immunol</span> <span class="cit-vol">28</span>: <span class="cit-fpage">1</span>â4, <span class="cit-pub-date">2007</span>.</cite></div><div class="cit-extra"><a href="http://physiolgenomics.physiology.org/highwire/openurl?query=rft.jtitle=Trends%20in%20immunology&amp;rft.stitle=Trends%20Immunol&amp;rft.aulast=Rink&amp;rft.auinit1=L.&amp;rft.volume=28&amp;rft.issue=1&amp;rft.spage=1&amp;rft.epage=4&amp;rft.atitle=Zinc%20homeostasis%20and%20immunity.&amp;rft_id=info:doi/10.1016/j.it.2006.11.005&amp;rft_id=info:pmid/17126599&amp;rft.genre=article&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;ctx_ver=Z39.88-2004&amp;url_ver=Z39.88-2004&amp;url_ctx_fmt=info:ofi/fmt:kev:mtx:ctx&amp;redirect_url=http://gw2jh3xr2c.search.serialssolutions.com" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url insertImage-processed"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/5510b14b-ba7f-4b26-b710-c206bd73a987-20120614.jpeg"> </a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=10.1016/j.it.2006.11.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=17126599&amp;link_type=MED&amp;atom=%2Fphysiolgenomics%2F34%2F1%2F127.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>Medline</span></a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=000243707900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div>
      </li><li><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference 13 in text" id="ref-13">âµ</a>
        
        <div class="cit ref-cit ref-other" id="cit-34.1.127.13"><div class="cit-metadata"><cite><span class="named-content arthw-firstauthor" id="named-content-37"><span class="named-content arthw-snm" id="named-content-38">Shanley</span> <span class="named-content arthw-fnm" id="named-content-39">TP</span></span><strong>,
 Cvijanovich N, Lin R, Allen GL, Thomas NJ, Doctor A, Kalyanaraman M, 
Tofil NM, Penfil S, Monaco M, Odoms K, Barnes M, Sakthivel B, Aronow BJ,
 Wong HR.</strong> Genome-level longitudinal expression of signaling pathways and gene networks in pediatric septic shock. <span class="cit-source">Mol Med</span> <span class="cit-vol">13</span>: <span class="cit-fpage">495</span>â508, <span class="cit-pub-date">2007</span>.</cite></div><div class="cit-extra"><a href="http://physiolgenomics.physiology.org/highwire/openurl?query=rft.jtitle=Molecular%20medicine%20%28Cambridge,%20Mass.%29&amp;rft.stitle=Mol%20Med&amp;rft.aulast=Shanley&amp;rft.auinit1=T.%20P.&amp;rft.volume=13&amp;rft.issue=9-10&amp;rft.spage=495&amp;rft.epage=508&amp;rft.atitle=Genome-level%20longitudinal%20expression%20of%20signaling%20pathways%20and%20gene%20networks%20in%20pediatric%20septic%20shock.&amp;rft_id=info:pmid/17932561&amp;rft.genre=article&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;ctx_ver=Z39.88-2004&amp;url_ver=Z39.88-2004&amp;url_ctx_fmt=info:ofi/fmt:kev:mtx:ctx&amp;redirect_url=http://gw2jh3xr2c.search.serialssolutions.com" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url insertImage-processed"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/5510b14b-ba7f-4b26-b710-c206bd73a987-20120614.jpeg"> </a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=17932561&amp;link_type=MED&amp;atom=%2Fphysiolgenomics%2F34%2F1%2F127.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>Medline</span></a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=000250280400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div>
      </li><li><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference 14 in text" id="ref-14">âµ</a>
        
        <div class="cit ref-cit ref-other" id="cit-34.1.127.14"><div class="cit-metadata"><cite><span class="named-content arthw-firstauthor" id="named-content-40"><span class="named-content arthw-snm" id="named-content-41">Shanley</span> <span class="named-content arthw-fnm" id="named-content-42">TP</span></span><strong>, Hallstrom C, Wong Sepsis HR.</strong> In: <em>Pediatric Critical Care Medicine</em> (3rd ed.), edited by Fuhrman BP and Zimmerman JJ. St. Louis, MO: Mosby, <span class="cit-pub-date">2006</span>, p. <span class="cit-fpage">1474</span>â1493.</cite></div><div class="cit-extra"></div></div>
      </li><li><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference 15 in text" id="ref-15">âµ</a>
        
        <div class="cit ref-cit ref-other" id="cit-34.1.127.15" data-doi="10.1016/S0749-0704(03)00004-6"><div class="cit-metadata"><cite><span class="named-content arthw-firstauthor" id="named-content-43"><span class="named-content arthw-snm" id="named-content-44">Shanley</span> <span class="named-content arthw-fnm" id="named-content-45">TP</span></span><strong>, Wong HR.</strong> Molecular genetics in the pediatric intensive care unit. <span class="cit-source">Crit Care Clin</span> <span class="cit-vol">19</span>: <span class="cit-fpage">577</span>â594, <span class="cit-pub-date">2003</span>.</cite></div><div class="cit-extra"><a href="http://physiolgenomics.physiology.org/highwire/openurl?query=rft.jtitle=Critical%20care%20clinics&amp;rft.stitle=Crit%20Care%20Clin&amp;rft.aulast=Shanley&amp;rft.auinit1=T.%20P.&amp;rft.volume=19&amp;rft.issue=3&amp;rft.spage=577&amp;rft.epage=594&amp;rft.atitle=Molecular%20genetics%20in%20the%20pediatric%20intensive%20care%20unit.&amp;rft_id=info:doi/10.1016/S0749-0704%2803%2900004-6&amp;rft_id=info:pmid/12848322&amp;rft.genre=article&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;ctx_ver=Z39.88-2004&amp;url_ver=Z39.88-2004&amp;url_ctx_fmt=info:ofi/fmt:kev:mtx:ctx&amp;redirect_url=http://gw2jh3xr2c.search.serialssolutions.com" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url insertImage-processed"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/5510b14b-ba7f-4b26-b710-c206bd73a987-20120614.jpeg"> </a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=10.1016/S0749-0704%2803%2900004-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=12848322&amp;link_type=MED&amp;atom=%2Fphysiolgenomics%2F34%2F1%2F127.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>Medline</span></a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=000183662900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div>
      </li><li><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference 16 in text" id="ref-16">âµ</a>
        
        <div class="cit ref-cit ref-other" id="cit-34.1.127.16" data-doi="10.1097/01.PCC.0000161289.22464.C3"><div class="cit-metadata"><cite><span class="named-content arthw-firstauthor" id="named-content-46"><span class="named-content arthw-snm" id="named-content-47">Watson</span> <span class="named-content arthw-fnm" id="named-content-48">RS</span></span><strong>, Carcillo JA.</strong> Scope and epidemiology of pediatric sepsis. <span class="cit-source">Pediatr Crit Care Med</span> <span class="cit-vol">6</span>: <span class="cit-fpage">S3</span>â5, <span class="cit-pub-date">2005</span>.</cite></div><div class="cit-extra"><a href="http://physiolgenomics.physiology.org/highwire/openurl?query=rft.jtitle=Pediatric%20critical%20care%20medicine%20:%20%20a%20journal%20of%20the%20Society%20of%20Critical%20Care%20Medicine%20and%20the%20World%20Federation%20of%20Pediatric%20Intensive%20and%20Critical%20Care%20Societies&amp;rft.stitle=Pediatr%20Crit%20Care%20Med&amp;rft.aulast=Watson&amp;rft.auinit1=R.%20S.&amp;rft.volume=6&amp;rft.issue=3%20Suppl&amp;rft.spage=S3&amp;rft.epage=S5&amp;rft.atitle=Scope%20and%20epidemiology%20of%20pediatric%20sepsis.&amp;rft_id=info:doi/10.1097/01.PCC.0000161289.22464.C3&amp;rft_id=info:pmid/15857554&amp;rft.genre=article&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;ctx_ver=Z39.88-2004&amp;url_ver=Z39.88-2004&amp;url_ctx_fmt=info:ofi/fmt:kev:mtx:ctx&amp;redirect_url=http://gw2jh3xr2c.search.serialssolutions.com" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url insertImage-processed"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/5510b14b-ba7f-4b26-b710-c206bd73a987-20120614.jpeg"> </a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=10.1097/01.PCC.0000161289.22464.C3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=15857554&amp;link_type=MED&amp;atom=%2Fphysiolgenomics%2F34%2F1%2F127.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>Medline</span></a></div></div>
      </li><li><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference 17 in text" id="ref-17">âµ</a>
        
        <div class="cit ref-cit ref-other" id="cit-34.1.127.17" data-doi="10.1089/omi.2006.0001"><div class="cit-metadata"><cite><span class="named-content arthw-firstauthor" id="named-content-49"><span class="named-content arthw-snm" id="named-content-50">Wilkes</span> <span class="named-content arthw-fnm" id="named-content-51">T</span></span><strong>, Laux H, Foy CA.</strong> Microarray data quality - review of current developments. <span class="cit-source">Omics</span> <span class="cit-vol">11</span>: <span class="cit-fpage">1</span>â13, <span class="cit-pub-date">2007</span>.</cite></div><div class="cit-extra"><a href="http://physiolgenomics.physiology.org/highwire/openurl?query=rft.jtitle=Omics%20:%20%20a%20journal%20of%20integrative%20biology&amp;rft.stitle=OMICS&amp;rft.aulast=Wilkes&amp;rft.auinit1=T.&amp;rft.volume=11&amp;rft.issue=1&amp;rft.spage=1&amp;rft.epage=13&amp;rft.atitle=Microarray%20data%20quality%20-%20review%20of%20current%20developments.&amp;rft_id=info:doi/10.1089/omi.2006.0001&amp;rft_id=info:pmid/17411392&amp;rft.genre=article&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;ctx_ver=Z39.88-2004&amp;url_ver=Z39.88-2004&amp;url_ctx_fmt=info:ofi/fmt:kev:mtx:ctx&amp;redirect_url=http://gw2jh3xr2c.search.serialssolutions.com" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url insertImage-processed"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/5510b14b-ba7f-4b26-b710-c206bd73a987-20120614.jpeg"> </a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=10.1089/omi.2006.0001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=17411392&amp;link_type=MED&amp;atom=%2Fphysiolgenomics%2F34%2F1%2F127.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>Medline</span></a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=000245740000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div>
      </li><li><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference 18 in text" id="ref-18">âµ</a>
        
        <div class="cit ref-cit ref-other" id="cit-34.1.127.18" data-doi="10.2217/14622416.8.10.1287"><div class="cit-metadata"><cite><span class="named-content arthw-firstauthor" id="named-content-52"><span class="named-content arthw-snm" id="named-content-53">Wong</span> <span class="named-content arthw-fnm" id="named-content-54">HR</span></span><strong>.</strong> Pediatric septic shock treatment: new clues from genomic profiling. <span class="cit-source">Pharmacogenomics</span> <span class="cit-vol">8</span>: <span class="cit-fpage">1287</span>â1290, <span class="cit-pub-date">2007</span>.</cite></div><div class="cit-extra"><a href="http://physiolgenomics.physiology.org/highwire/openurl?query=rft.jtitle=Pharmacogenomics&amp;rft.stitle=Pharmacogenomics&amp;rft.aulast=Wong&amp;rft.auinit1=H.%20R.&amp;rft.volume=8&amp;rft.issue=10&amp;rft.spage=1287&amp;rft.epage=1290&amp;rft.atitle=Pediatric%20septic%20shock%20treatment:%20new%20clues%20from%20genomic%20profiling.&amp;rft_id=info:doi/10.2217/14622416.8.10.1287&amp;rft_id=info:pmid/17979501&amp;rft.genre=article&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;ctx_ver=Z39.88-2004&amp;url_ver=Z39.88-2004&amp;url_ctx_fmt=info:ofi/fmt:kev:mtx:ctx&amp;redirect_url=http://gw2jh3xr2c.search.serialssolutions.com" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url insertImage-processed"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/5510b14b-ba7f-4b26-b710-c206bd73a987-20120614.jpeg"> </a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=10.2217/14622416.8.10.1287&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=17979501&amp;link_type=MED&amp;atom=%2Fphysiolgenomics%2F34%2F1%2F127.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>Medline</span></a><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=000250979200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div>
      </li><li><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference 19 in text" id="ref-19">âµ</a>
        
        <div class="cit ref-cit ref-other" id="cit-34.1.127.19" data-doi="10.1152/physiolgenomics.00024.2007"><div class="cit-metadata"><cite><span class="named-content arthw-firstauthor" id="named-content-55"><span class="named-content arthw-snm" id="named-content-56">Wong</span> <span class="named-content arthw-fnm" id="named-content-57">HR</span></span><strong>,
 Shanley TP, Sakthivel B, Cvijanovich N, Lin R, Allen GL, Thomas NJ, 
Doctor A, Kalyanaraman M, Tofil NM, Penfil S, Monaco M, Tagavilla MA, 
Odoms K, Dunsmore K, Barnes M, Aronow BJ.</strong> Genome-level expression profiles in pediatric septic shock indicate a role for altered zinc homeostasis in poor outcome. <span class="cit-source">Physiol Genomics</span> <span class="cit-vol">30</span>: <span class="cit-fpage">146</span>â155, <span class="cit-pub-date">2007</span>.</cite></div><div class="cit-extra"><a href="http://physiolgenomics.physiology.org/highwire/openurl?query=rft.jtitle=Physiol%20Genomics&amp;rft_id=info:doi/10.1152/physiolgenomics.00024.2007&amp;rft_id=info:pmid/17374846&amp;rft.genre=article&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;ctx_ver=Z39.88-2004&amp;url_ver=Z39.88-2004&amp;url_ctx_fmt=info:ofi/fmt:kev:mtx:ctx&amp;redirect_url=http://gw2jh3xr2c.search.serialssolutions.com" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url insertImage-processed"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/5510b14b-ba7f-4b26-b710-c206bd73a987-20120614.jpeg"> </a><a href="http://physiolgenomics.physiology.org/lookup/ijlink?linkType=ABST&amp;journalCode=physiolgenomics&amp;resid=30/2/146&amp;atom=%2Fphysiolgenomics%2F34%2F1%2F127.atom" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div>
      </li><li><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference 20 in text" id="ref-20">âµ</a>
        
        <div class="cit ref-cit ref-other" id="cit-34.1.127.20"><div class="cit-metadata"><cite><span class="named-content arthw-firstauthor" id="named-content-58"><span class="named-content arthw-snm" id="named-content-59">Yao</span> <span class="named-content arthw-fnm" id="named-content-60">Z</span></span><strong>, Ruzzo WL.</strong> A regression-based K nearest neighbor algorithm for gene function prediction from heterogeneous data. <span class="cit-source">BMC Bioinformatics 7</span>, <em>Suppl</em> 1: <span class="cit-fpage">S11</span>, <span class="cit-pub-date">2006</span>.</cite></div><div class="cit-extra"></div></div>
      </li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="http://physiolgenomics.physiology.org/content/34/1/127.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div><div class="panels-ajax-tab-wrap-physiolgenomics_tab_data"><!--?xml version="1.0" encoding="UTF-8" ?-->
    
  


<link type="text/css" rel="stylesheet" href="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/css_Bew06AqfLkraZEZ_KZtqo5ScVCqjDNMCihAZ6L6k4OI.css" media="all">
<link rel="stylesheet" type="text/css" href="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/onecol.css"><div class="panels-ajax-tab-panel panels-ajax-tab-panel-physiolgenomics-tab-data"><div class="panel-display panel-1col clearfix">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-fig-data">
  
        <h2 class="pane-title">Article Figures &amp; Data</h2>
    
  
  <div class="pane-content">
    <div class="elements-frag-data highwire-markup" id="fig-data"><div id="fig-data-figures" class="group frag-figure"><div class="fig-data-title-jump clearfix"><h3 class="fig-data-group-title">Figures</h3><div class="fig-data-jump-links"><ul class="fig-data-jump-links-list links inline"><li class="table first"><a href="#fig-data-tables" class="fig-data-jump-link fig-data-jump-link-table link-icon"><i class="icon-caret-down"></i> <span class="title">Tables</span></a></li><li class="additional-files last"><a href="#fig-data-additional-files" class="fig-data-jump-link fig-data-jump-link-additional-files link-icon"><i class="icon-caret-down"></i> <span class="title">Additional Files</span></a></li></ul></div></div><div class="item-list"><ul class="fig-data-list clearfix" id="fragments-fig"><li class="first"><div class="element-fig-data clearfix -caption"><div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="fig-expansion" id="F1"><span class="highwire-journal-article-marker-start"></span><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="http://physiolgenomics.physiology.org/content/physiolgenomics/34/1/127/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Venn analysis comparing gene lists A and B. See Table 4 and text for gene list derivation." class="fragment-images colorbox-load highwireFiguresMarkupProcessor-processed cboxElement" rel="gallery-fragment-images-752214206" data-figure-caption="Venn analysis comparing gene lists A and B. See Table 4 and text for gene list derivation." data-icon-position="" data-hide-link-title="0"><img class="fragment-image" alt="Fig. 1." src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/F1.gif"></a></div></div><ul class="highwire-figure-links inline"><li class="0 first"><a href="http://physiolgenomics.physiology.org/content/physiolgenomics/34/1/127/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 1." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="1"><a href="http://physiolgenomics.physiology.org/content/physiolgenomics/34/1/127/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="2 last"><a href="http://physiolgenomics.physiology.org/highwire/powerpoint/11598" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig. 1.</span> <p id="p-45">Venn analysis comparing gene lists A and B. See <span id="xref-table-wrap-4-5" class="xref-table">Table 4</span> and text for gene list derivation.</p><div class="sb-div caption-clear"></div></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></li><li class="last"><div class="element-fig-data clearfix -caption"><div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="fig-expansion" id="F2"><span class="highwire-journal-article-marker-start"></span><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="http://physiolgenomics.physiology.org/content/physiolgenomics/34/1/127/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Venn analysis comparing gene lists B and C. See Table 4 and text for gene list derivation." class="fragment-images colorbox-load highwireFiguresMarkupProcessor-processed cboxElement" rel="gallery-fragment-images-752214206" data-figure-caption="Venn analysis comparing gene lists B and C. See Table 4 and text for gene list derivation." data-icon-position="" data-hide-link-title="0"><img class="fragment-image" alt="Fig. 2." src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/F2.gif"></a></div></div><ul class="highwire-figure-links inline"><li class="0 first"><a href="http://physiolgenomics.physiology.org/content/physiolgenomics/34/1/127/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Fig. 2." data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="1"><a href="http://physiolgenomics.physiology.org/content/physiolgenomics/34/1/127/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="2 last"><a href="http://physiolgenomics.physiology.org/highwire/powerpoint/11602" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Fig. 2.</span> <p id="p-46">Venn analysis comparing gene lists B and C. See <span id="xref-table-wrap-4-6" class="xref-table">Table 4</span> and text for gene list derivation.</p><div class="sb-div caption-clear"></div></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></li></ul></div></div><div id="fig-data-tables" class="group frag-table"><div class="fig-data-title-jump clearfix"><h3 class="fig-data-group-title">Tables</h3><div class="fig-data-jump-links"><ul class="fig-data-jump-links-list links inline"><li class="figure first"><a href="#fig-data-figures" class="fig-data-jump-link fig-data-jump-link-figure link-icon"><i class="icon-caret-up"></i> <span class="title">Figures</span></a></li><li class="additional-files last"><a href="#fig-data-additional-files" class="fig-data-jump-link fig-data-jump-link-additional-files link-icon"><i class="icon-caret-down"></i> <span class="title">Additional Files</span></a></li></ul></div></div><div class="item-list"><ul class="fig-data-list clearfix" id="fragments-table"><li class="first"><div class="element-fig-data clearfix -caption"><div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="table-expansion" id="T1"><span class="highwire-journal-article-marker-start"></span><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-47"><em>Clinical and demographic data for all subjects in test data set</em></p><div class="sb-div caption-clear"></div></div><table id="table-1"><thead id="thead-1"><tr id="tr-1"><th colspan="1" rowspan="1" id="th-1" class="table-center table-vbottom" align="center" valign="bottom"></th><th colspan="1" rowspan="1" id="th-2" class="table-center table-vbottom" align="center" valign="bottom">Controls</th><th colspan="1" rowspan="1" id="th-3" class="table-center table-vbottom" align="center" valign="bottom">Septic Shock</th></tr></thead><tbody id="tbody-1"><tr id="tr-2"><td colspan="1" rowspan="1" id="td-1" class="table-left table-vtop" align="left" valign="top">Individual subjects, <em>n</em></td><td colspan="1" rowspan="1" id="td-2" class="table-center table-vtop" align="center" valign="top">14</td><td colspan="1" rowspan="1" id="td-3" class="table-center table-vtop" align="center" valign="top">30</td></tr><tr id="tr-3"><td colspan="1" rowspan="1" id="td-4" class="table-left table-vtop" align="left" valign="top">Mean age (yr) Â± SD<a id="xref-fn-5-1" class="xref-table-fn" href="#fn-5"><sup>*</sup></a></td><td colspan="1" rowspan="1" id="td-5" class="table-center table-vtop" align="center" valign="top">2.7Â±2.4</td><td colspan="1" rowspan="1" id="td-6" class="table-center table-vtop" align="center" valign="top">3.2Â±2.9</td></tr><tr id="tr-4"><td colspan="1" rowspan="1" id="td-7" class="table-left table-vtop" align="left" valign="top">Mean PRISM score Â± SD<a id="xref-fn-5-2" class="xref-table-fn" href="#fn-5"><sup>*</sup></a></td><td colspan="1" rowspan="1" id="td-8" class="table-center table-vtop" align="center" valign="top">n/a</td><td colspan="1" rowspan="1" id="td-9" class="table-center table-vtop" align="center" valign="top">18.9Â±12.3</td></tr><tr id="tr-5"><td colspan="1" rowspan="1" id="td-10" class="table-left table-vtop" align="left" valign="top">Sex (male/female)<a id="xref-fn-5-3" class="xref-table-fn" href="#fn-5"><sup>*</sup></a></td><td colspan="1" rowspan="1" id="td-11" class="table-center table-vtop" align="center" valign="top">8/6</td><td colspan="1" rowspan="1" id="td-12" class="table-center table-vtop" align="center" valign="top">16/14</td></tr><tr id="tr-6"><td colspan="1" rowspan="1" id="td-13" class="table-left table-vtop" align="left" valign="top">Race (<em>n</em>)<a id="xref-fn-5-4" class="xref-table-fn" href="#fn-5"><sup>*</sup></a></td><td colspan="1" rowspan="1" id="td-14" class="table-left table-vtop" align="left" valign="top">A. A./black (3)</td><td colspan="1" rowspan="1" id="td-15" class="table-left table-vtop" align="left" valign="top">A. A./black (2)</td></tr><tr id="tr-7"><td colspan="1" rowspan="1" id="td-16" class="table-left table-vtop" align="left" valign="top"></td><td colspan="1" rowspan="1" id="td-17" class="table-left table-vtop" align="left" valign="top">white (10)</td><td colspan="1" rowspan="1" id="td-18" class="table-left table-vtop" align="left" valign="top">white (26)</td></tr><tr id="tr-8"><td colspan="1" rowspan="1" id="td-19" class="table-left table-vtop" align="left" valign="top"></td><td colspan="1" rowspan="1" id="td-20" class="table-left table-vtop" align="left" valign="top">Asian (1)</td><td colspan="1" rowspan="1" id="td-21" class="table-left table-vtop" align="left" valign="top">unreported (2)</td></tr></tbody></table><div class="table-foot"><ul class="table-footnotes"><li class="fn" id="fn-4"><p id="p-48">n/a, Not applicable; A. A., African American.</p></li><li class="fn" id="fn-5"><p id="p-49"><a class="rev-xref" href="#xref-fn-5-1">âµ</a><span class="fn-label">*</span> <em>P</em> &gt; 0.05.</p></li></ul></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></li><li><div class="element-fig-data clearfix -caption"><div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="table-expansion" id="T2"><span class="highwire-journal-article-marker-start"></span><div class="table-caption"><span class="table-label">Table 2.</span> <p id="p-50"><em>Microbiology data for test data set patients with septic shock</em></p><div class="sb-div caption-clear"></div></div><table id="table-2"><thead id="thead-2"><tr id="tr-9"><th colspan="1" rowspan="1" id="th-4" class="table-center table-vbottom" align="center" valign="bottom">Organism (<em>n</em>)</th><th colspan="1" rowspan="1" id="th-5" class="table-center table-vbottom" align="center" valign="bottom">Primary Source of Positive Culture (<em>n</em>)</th></tr></thead><tbody id="tbody-2"><tr id="tr-10"><td colspan="1" rowspan="1" id="td-22" class="table-left table-vtop" align="left" valign="top"><em>Staphylococcus aureus</em> (5)</td><td colspan="1" rowspan="1" id="td-23" class="table-center table-vtop" align="center" valign="top">blood (10)</td></tr><tr id="tr-11"><td colspan="1" rowspan="1" id="td-24" class="table-left table-vtop" align="left" valign="top"><em>Streptococcus pyogenes</em> (2)</td><td colspan="1" rowspan="1" id="td-25" class="table-center table-vtop" align="center" valign="top">lung (1)</td></tr><tr id="tr-12"><td colspan="1" rowspan="1" id="td-26" class="table-left table-vtop" align="left" valign="top"><em>Streptococcus agalactiae</em> (2)</td><td colspan="1" rowspan="1" id="td-27" class="table-center table-vtop" align="center" valign="top">cerebral spinal fluid (1)</td></tr><tr id="tr-13"><td colspan="1" rowspan="1" id="td-28" class="table-left table-vtop" align="left" valign="top"><em>Neisseria meningitidis</em> (2)</td><td colspan="1" rowspan="1" id="td-29" class="table-center table-vtop" align="center" valign="top">other (2)</td></tr><tr id="tr-14"><td colspan="1" rowspan="1" id="td-30" class="table-left table-vtop" align="left" valign="top"><em>Streptococcus pneumoniae</em> (2)</td><td colspan="1" rowspan="1" id="td-31" class="table-center table-vtop" align="center" valign="top"></td></tr><tr id="tr-15"><td colspan="1" rowspan="1" id="td-32" class="table-left table-vtop" align="left" valign="top">Adenovirus (1)</td><td colspan="1" rowspan="1" id="td-33" class="table-center table-vtop" align="center" valign="top"></td></tr></tbody></table><div class="table-foot"></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></li><li><div class="element-fig-data clearfix -caption"><div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="table-expansion" id="T3"><span class="highwire-journal-article-marker-start"></span><div class="table-caption"><span class="table-label">Table 3.</span> <p id="p-51"><em>Median absolute white blood cell counts for patients with septic shock in the training and test data sets (Ã10<sup>3</sup> per mm<sup>3</sup>; IQR)</em></p><div class="sb-div caption-clear"></div></div><table id="table-3"><thead id="thead-3"><tr id="tr-16"><th colspan="1" rowspan="1" id="th-6" class="table-center table-vbottom" align="center" valign="bottom"></th><th colspan="1" rowspan="1" id="th-7" class="table-center table-vbottom" align="center" valign="bottom">Training Data Set</th><th colspan="1" rowspan="1" id="th-8" class="table-center table-vbottom" align="center" valign="bottom">Test Data Set</th></tr></thead><tbody id="tbody-3"><tr id="tr-17"><td colspan="1" rowspan="1" id="td-34" class="table-left table-vtop" align="left" valign="top">Total WBC</td><td colspan="1" rowspan="1" char="." id="td-35" class="table-char table-vtop" align="char" valign="top">11.6 (4.1â22.6)</td><td colspan="1" rowspan="1" char="." id="td-36" class="table-char table-vtop" align="char" valign="top">6.4 (3.5â16.9)</td></tr><tr id="tr-18"><td colspan="1" rowspan="1" id="td-37" class="table-left table-vtop" align="left" valign="top">Neutrophils (mature + immature)</td><td colspan="1" rowspan="1" char="." id="td-38" class="table-char table-vtop" align="char" valign="top">7.1 (2.6â16.5)</td><td colspan="1" rowspan="1" char="." id="td-39" class="table-char table-vtop" align="char" valign="top">4.5 (1.7â11.7)</td></tr><tr id="tr-19"><td colspan="1" rowspan="1" id="td-40" class="table-left table-vtop" align="left" valign="top">Lymphocytes<a id="xref-fn-7-1" class="xref-table-fn" href="#fn-7"><sup>*</sup></a></td><td colspan="1" rowspan="1" char="." id="td-41" class="table-char table-vtop" align="char" valign="top">1.8 (1.3â3.1)</td><td colspan="1" rowspan="1" char="." id="td-42" class="table-char table-vtop" align="char" valign="top">0.9 (0.6â1.3)</td></tr><tr id="tr-20"><td colspan="1" rowspan="1" id="td-43" class="table-left table-vtop" align="left" valign="top">Monocytes</td><td colspan="1" rowspan="1" char="." id="td-44" class="table-char table-vtop" align="char" valign="top">0.4 (0.1â1.4)</td><td colspan="1" rowspan="1" char="." id="td-45" class="table-char table-vtop" align="char" valign="top">0.2 (0.1â0.7)</td></tr></tbody></table><div class="table-foot"><ul class="table-footnotes"><li class="fn" id="fn-6"><p id="p-52">WBC, white blood cell; IQR, interquartile range.</p></li><li class="fn" id="fn-7"><p id="p-53"><a class="rev-xref" href="#xref-fn-7-1">âµ</a><span class="fn-label">*</span> <em>P</em> &lt; 0.05 by Mann-Whitney rank sum test.</p></li></ul></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></li><li><div class="element-fig-data clearfix -caption"><div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="table-expansion" id="T4"><span class="highwire-journal-article-marker-start"></span><div class="table-caption"><span class="table-label">Table 4.</span> <p id="p-54"><em>Gene list designation and derivation</em></p><div class="sb-div caption-clear"></div></div><table id="table-4"><thead id="thead-4"><tr id="tr-21"><th colspan="1" rowspan="1" id="th-9" class="table-center table-vbottom" align="center" valign="bottom">Gene list</th><th colspan="1" rowspan="1" id="th-10" class="table-center table-vbottom" align="center" valign="bottom"># of Genes</th><th colspan="1" rowspan="1" id="th-11" class="table-center table-vbottom" align="center" valign="bottom">Data Set</th><th colspan="1" rowspan="1" id="th-12" class="table-center table-vbottom" align="center" valign="bottom">Statistical Filter</th><th colspan="1" rowspan="1" id="th-13" class="table-center table-vbottom" align="center" valign="bottom">Expression Filter</th></tr></thead><tbody id="tbody-4"><tr id="tr-22"><td colspan="1" rowspan="1" id="td-46" class="table-center table-vtop" align="center" valign="top">A</td><td colspan="1" rowspan="1" char="." id="td-47" class="table-char table-vtop" align="char" valign="top">2,482</td><td colspan="1" rowspan="1" id="td-48" class="table-left table-vtop" align="left" valign="top">training</td><td colspan="1" rowspan="1" id="td-49" class="table-left table-vtop" align="left" valign="top">ANOVA, <em>t</em>-test Benjamini-Hochberg false discovery rate (5%)</td><td colspan="1" rowspan="1" id="td-50" class="table-left table-vtop" align="left" valign="top">â¥2-fold expression difference between the median of controls and at least 50% of the patients with septic shock</td></tr><tr id="tr-23"><td colspan="1" rowspan="1" id="td-51" class="table-center table-vtop" align="center" valign="top">B</td><td colspan="1" rowspan="1" char="." id="td-52" class="table-char table-vtop" align="char" valign="top">3,296</td><td colspan="1" rowspan="1" id="td-53" class="table-left table-vtop" align="left" valign="top">test</td><td colspan="1" rowspan="1" id="td-54" class="table-left table-vtop" align="left" valign="top">same as gene list A</td><td colspan="1" rowspan="1" id="td-55" class="table-left table-vtop" align="left" valign="top">same as gene list A</td></tr><tr id="tr-24"><td colspan="1" rowspan="1" id="td-56" class="table-center table-vtop" align="center" valign="top">C</td><td colspan="1" rowspan="1" char="." id="td-57" class="table-char table-vtop" align="char" valign="top">2,104</td><td colspan="1" rowspan="1" id="td-58" class="table-left table-vtop" align="left" valign="top">test</td><td colspan="1" rowspan="1" id="td-59" class="table-left table-vtop" align="left" valign="top">ANOVA, <em>t</em>-test Bonferroni correction for multiple comparisons (<em>P</em> = 0.05)</td><td colspan="1" rowspan="1" id="td-60" class="table-left table-vtop" align="left" valign="top">none</td></tr></tbody></table><div class="table-foot"></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></li><li><div class="element-fig-data clearfix -caption"><div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="table-expansion" id="T5"><span class="highwire-journal-article-marker-start"></span><div class="table-caption"><span class="table-label">Table 5.</span> <p id="p-55"><em>Results of cross-validation of training data set based gene list A and on KNN or SVM algorithms</em></p><div class="sb-div caption-clear"></div></div><table id="table-5"><thead id="thead-5"><tr id="tr-25"><th colspan="1" rowspan="1" id="th-14" class="table-center table-vbottom" align="center" valign="bottom">Gene Selection Method (<em>Algorithm</em>)</th><th colspan="1" rowspan="1" id="th-15" class="table-center table-vbottom" align="center" valign="bottom"># Correct Predictions</th><th colspan="1" rowspan="1" id="th-16" class="table-center table-vbottom" align="center" valign="bottom"># Incorrect Predictions</th><th colspan="1" rowspan="1" id="th-17" class="table-center table-vbottom" align="center" valign="bottom"># Not Predicted</th></tr></thead><tbody id="tbody-5"><tr id="tr-26"><td colspan="1" rowspan="1" id="td-61" class="table-left table-vtop" align="left" valign="top">Fisher's exact test<a id="xref-fn-9-1" class="xref-table-fn" href="#fn-9"><sup>*</sup></a></td><td colspan="1" rowspan="1" id="td-62" class="table-center table-vtop" align="center" valign="top"></td><td colspan="1" rowspan="1" id="td-63" class="table-center table-vtop" align="center" valign="top"></td><td colspan="1" rowspan="1" id="td-64" class="table-center table-vtop" align="center" valign="top"></td></tr><tr id="tr-27"><td colspan="1" rowspan="1" id="td-65" class="table-left table-vtop" align="left" valign="top">ââââ<em>KNN</em></td><td colspan="1" rowspan="1" char="." id="td-66" class="table-char table-vtop" align="char" valign="top">53</td><td colspan="1" rowspan="1" id="td-67" class="table-left table-vtop" align="left" valign="top">1 (SS predicted as C)</td><td colspan="1" rowspan="1" id="td-68" class="table-left table-vtop" align="left" valign="top">3 (2 SS; 1 C)</td></tr><tr id="tr-28"><td colspan="1" rowspan="1" id="td-69" class="table-left table-vtop" align="left" valign="top">ââââ<em>SVM</em></td><td colspan="1" rowspan="1" char="." id="td-70" class="table-char table-vtop" align="char" valign="top">54</td><td colspan="1" rowspan="1" id="td-71" class="table-left table-vtop" align="left" valign="top">3 (3 C predicted as SS)</td><td colspan="1" rowspan="1" id="td-72" class="table-center table-vtop" align="center" valign="top">n/a</td></tr><tr id="tr-29"><td colspan="1" rowspan="1" id="td-73" class="table-left table-vtop" align="left" valign="top"></td><td colspan="1" rowspan="1" id="td-74" class="table-center table-vtop" align="center" valign="top"></td><td colspan="1" rowspan="1" id="td-75" class="table-left table-vtop" align="left" valign="top">100% sensitivity</td><td colspan="1" rowspan="1" id="td-76" class="table-left table-vtop" align="left" valign="top"></td></tr><tr id="tr-30"><td colspan="1" rowspan="1" id="td-77" class="table-left table-vtop" align="left" valign="top"></td><td colspan="1" rowspan="1" id="td-78" class="table-center table-vtop" align="center" valign="top"></td><td colspan="1" rowspan="1" id="td-79" class="table-left table-vtop" align="left" valign="top">80% specificity</td><td colspan="1" rowspan="1" id="td-80" class="table-left table-vtop" align="left" valign="top"></td></tr><tr id="tr-31"><td colspan="1" rowspan="1" id="td-81" class="table-left table-vtop" align="left" valign="top">Golub method<a id="xref-fn-10-1" class="xref-table-fn" href="#fn-10"><sup>â </sup></a></td><td colspan="1" rowspan="1" id="td-82" class="table-center table-vtop" align="center" valign="top"></td><td colspan="1" rowspan="1" id="td-83" class="table-left table-vtop" align="left" valign="top"></td><td colspan="1" rowspan="1" id="td-84" class="table-left table-vtop" align="left" valign="top"></td></tr><tr id="tr-32"><td colspan="1" rowspan="1" id="td-85" class="table-left table-vtop" align="left" valign="top">ââââ<em>KNN</em></td><td colspan="1" rowspan="1" char="." id="td-86" class="table-char table-vtop" align="char" valign="top">54</td><td colspan="1" rowspan="1" id="td-87" class="table-left table-vtop" align="left" valign="top">1 (1 C predicted as SS)</td><td colspan="1" rowspan="1" id="td-88" class="table-left table-vtop" align="left" valign="top">2 (2 SS)</td></tr><tr id="tr-33"><td colspan="1" rowspan="1" id="td-89" class="table-left table-vtop" align="left" valign="top">ââââ<em>SVM</em></td><td colspan="1" rowspan="1" char="." id="td-90" class="table-char table-vtop" align="char" valign="top">54</td><td colspan="1" rowspan="1" id="td-91" class="table-left table-vtop" align="left" valign="top">3 (3 C predicted as SS)</td><td colspan="1" rowspan="1" id="td-92" class="table-center table-vtop" align="center" valign="top">n/a</td></tr><tr id="tr-34"><td colspan="1" rowspan="1" id="td-93" class="table-left table-vtop" align="left" valign="top"></td><td colspan="1" rowspan="1" id="td-94" class="table-center table-vtop" align="center" valign="top"></td><td colspan="1" rowspan="1" id="td-95" class="table-left table-vtop" align="left" valign="top">100% sensitivity</td><td colspan="1" rowspan="1" id="td-96" class="table-left table-vtop" align="left" valign="top"></td></tr><tr id="tr-35"><td colspan="1" rowspan="1" id="td-97" class="table-left table-vtop" align="left" valign="top"></td><td colspan="1" rowspan="1" id="td-98" class="table-center table-vtop" align="center" valign="top"></td><td colspan="1" rowspan="1" id="td-99" class="table-left table-vtop" align="left" valign="top">80% specificity</td><td colspan="1" rowspan="1" id="td-100" class="table-left table-vtop" align="left" valign="top"></td></tr><tr id="tr-36"><td colspan="1" rowspan="1" id="td-101" class="table-left table-vtop" align="left" valign="top">All genes in list<a id="xref-fn-11-1" class="xref-table-fn" href="#fn-11"><sup>â¡</sup></a></td><td colspan="1" rowspan="1" id="td-102" class="table-center table-vtop" align="center" valign="top"></td><td colspan="1" rowspan="1" id="td-103" class="table-left table-vtop" align="left" valign="top"></td><td colspan="1" rowspan="1" id="td-104" class="table-left table-vtop" align="left" valign="top"></td></tr><tr id="tr-37"><td colspan="1" rowspan="1" id="td-105" class="table-left table-vtop" align="left" valign="top">ââââ<em>KNN</em></td><td colspan="1" rowspan="1" char="." id="td-106" class="table-char table-vtop" align="char" valign="top">53</td><td colspan="1" rowspan="1" id="td-107" class="table-left table-vtop" align="left" valign="top">1 (1 SS predicted as C)</td><td colspan="1" rowspan="1" id="td-108" class="table-left table-vtop" align="left" valign="top">3 (3 SS)</td></tr><tr id="tr-38"><td colspan="1" rowspan="1" id="td-109" class="table-left table-vtop" align="left" valign="top">ââââ<em>SVM</em></td><td colspan="1" rowspan="1" char="." id="td-110" class="table-char table-vtop" align="char" valign="top">55</td><td colspan="1" rowspan="1" id="td-111" class="table-left table-vtop" align="left" valign="top">2 (2 C predicted as SS)</td><td colspan="1" rowspan="1" id="td-112" class="table-center table-vtop" align="center" valign="top">n/a</td></tr><tr id="tr-39"><td colspan="1" rowspan="1" id="td-113" class="table-left table-vtop" align="left" valign="top"></td><td colspan="1" rowspan="1" id="td-114" class="table-center table-vtop" align="center" valign="top"></td><td colspan="1" rowspan="1" id="td-115" class="table-left table-vtop" align="left" valign="top">100% sensitivity</td><td colspan="1" rowspan="1" id="td-116" class="table-left table-vtop" align="left" valign="top"></td></tr><tr id="tr-40"><td colspan="1" rowspan="1" id="td-117" class="table-left table-vtop" align="left" valign="top"></td><td colspan="1" rowspan="1" id="td-118" class="table-center table-vtop" align="center" valign="top"></td><td colspan="1" rowspan="1" id="td-119" class="table-left table-vtop" align="left" valign="top">87% specificity</td><td colspan="1" rowspan="1" id="td-120" class="table-left table-vtop" align="left" valign="top"></td></tr></tbody></table><div class="table-foot"><ul class="table-footnotes"><li class="fn" id="fn-8"><p id="p-56">The class options for the model were septic shock (SS) or control (C). KNN, K-nearest neighbors; SVM, support vector machines.</p></li><li class="fn" id="fn-9"><p id="p-57"><a class="rev-xref" href="#xref-fn-9-1">âµ</a><span class="fn-label">*</span> GeneSpring parameters (default): 50 predictor genes, 10 neighbors, 0.2 decision cutoff for <em>P</em> value ratio.</p></li><li class="fn" id="fn-10"><p id="p-58"><a class="rev-xref" href="#xref-fn-10-1">âµ</a><span class="fn-label">â </span> GeneSpring parameters (default): 50 predictor genes, 10 neighbors, 0.2 decision cutoff for <em>P</em> value ratio.</p></li><li class="fn" id="fn-11"><p id="p-59"><a class="rev-xref" href="#xref-fn-11-1">âµ</a><span class="fn-label">â¡</span> GeneSpring parameters (default): 10 neighbors, 0.2 decision cutoff for <em>P</em> value ratio.</p></li></ul></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></li><li><div class="element-fig-data clearfix -caption"><div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="table-expansion" id="T6"><span class="highwire-journal-article-marker-start"></span><div class="table-caption"><span class="table-label">Table 6.</span> <p id="p-60"><em>Class prediction modeling of test data set based on cross-validation results (<span id="xref-table-wrap-5-4" class="xref-table">Table 5</span>) and KNN and SVM algorithms</em></p><div class="sb-div caption-clear"></div></div><table id="table-6"><thead id="thead-6"><tr id="tr-41"><th colspan="1" rowspan="1" id="th-18" class="table-center table-vbottom" align="center" valign="bottom">Gene Selection Method (<em>Algorithm</em>)</th><th colspan="1" rowspan="1" id="th-19" class="table-center table-vbottom" align="center" valign="bottom"># Correct Predictions</th><th colspan="1" rowspan="1" id="th-20" class="table-center table-vbottom" align="center" valign="bottom"># Incorrect Predictions</th><th colspan="1" rowspan="1" id="th-21" class="table-center table-vbottom" align="center" valign="bottom"># Not Predicted</th></tr></thead><tbody id="tbody-6"><tr id="tr-42"><td colspan="1" rowspan="1" id="td-121" class="table-left table-vtop" align="left" valign="top">Fisher's exact test<a id="xref-fn-13-1" class="xref-table-fn" href="#fn-13"><sup>*</sup></a></td><td colspan="1" rowspan="1" id="td-122" class="table-center table-vtop" align="center" valign="top"></td><td colspan="1" rowspan="1" id="td-123" class="table-center table-vtop" align="center" valign="top"></td><td colspan="1" rowspan="1" id="td-124" class="table-center table-vtop" align="center" valign="top"></td></tr><tr id="tr-43"><td colspan="1" rowspan="1" id="td-125" class="table-left table-vtop" align="left" valign="top">ââââ<em>KNN</em></td><td colspan="1" rowspan="1" char="." id="td-126" class="table-char table-vtop" align="char" valign="top">43</td><td colspan="1" rowspan="1" id="td-127" class="table-left table-vtop" align="left" valign="top">1 (SS predicted as C)</td><td colspan="1" rowspan="1" id="td-128" class="table-center table-vtop" align="center" valign="top">0</td></tr><tr id="tr-44"><td colspan="1" rowspan="1" id="td-129" class="table-left table-vtop" align="left" valign="top">ââââ<em>SVM</em></td><td colspan="1" rowspan="1" char="." id="td-130" class="table-char table-vtop" align="char" valign="top">42</td><td colspan="1" rowspan="1" id="td-131" class="table-left table-vtop" align="left" valign="top">2 (2 C predicted as SS)</td><td colspan="1" rowspan="1" id="td-132" class="table-center table-vtop" align="center" valign="top">n/a</td></tr><tr id="tr-45"><td colspan="1" rowspan="1" id="td-133" class="table-left table-vtop" align="left" valign="top"></td><td colspan="1" rowspan="1" id="td-134" class="table-center table-vtop" align="center" valign="top"></td><td colspan="1" rowspan="1" id="td-135" class="table-left table-vtop" align="left" valign="top">100% sensitivity</td><td colspan="1" rowspan="1" id="td-136" class="table-center table-vtop" align="center" valign="top"></td></tr><tr id="tr-46"><td colspan="1" rowspan="1" id="td-137" class="table-left table-vtop" align="left" valign="top"></td><td colspan="1" rowspan="1" id="td-138" class="table-center table-vtop" align="center" valign="top"></td><td colspan="1" rowspan="1" id="td-139" class="table-left table-vtop" align="left" valign="top">86% specificity</td><td colspan="1" rowspan="1" id="td-140" class="table-center table-vtop" align="center" valign="top"></td></tr><tr id="tr-47"><td colspan="1" rowspan="1" id="td-141" class="table-left table-vtop" align="left" valign="top">Golub method<a id="xref-fn-14-1" class="xref-table-fn" href="#fn-14"><sup>â </sup></a></td><td colspan="1" rowspan="1" id="td-142" class="table-center table-vtop" align="center" valign="top"></td><td colspan="1" rowspan="1" id="td-143" class="table-left table-vtop" align="left" valign="top"></td><td colspan="1" rowspan="1" id="td-144" class="table-center table-vtop" align="center" valign="top"></td></tr><tr id="tr-48"><td colspan="1" rowspan="1" id="td-145" class="table-left table-vtop" align="left" valign="top">ââââ<em>KNN</em></td><td colspan="1" rowspan="1" char="." id="td-146" class="table-char table-vtop" align="char" valign="top">44</td><td colspan="1" rowspan="1" id="td-147" class="table-left table-vtop" align="left" valign="top">0</td><td colspan="1" rowspan="1" id="td-148" class="table-center table-vtop" align="center" valign="top">0</td></tr><tr id="tr-49"><td colspan="1" rowspan="1" id="td-149" class="table-left table-vtop" align="left" valign="top">ââââ<em>SVM</em></td><td colspan="1" rowspan="1" char="." id="td-150" class="table-char table-vtop" align="char" valign="top">42</td><td colspan="1" rowspan="1" id="td-151" class="table-left table-vtop" align="left" valign="top">2 (2 C predicted as SS)</td><td colspan="1" rowspan="1" id="td-152" class="table-center table-vtop" align="center" valign="top">n/a</td></tr><tr id="tr-50"><td colspan="1" rowspan="1" id="td-153" class="table-left table-vtop" align="left" valign="top"></td><td colspan="1" rowspan="1" id="td-154" class="table-center table-vtop" align="center" valign="top"></td><td colspan="1" rowspan="1" id="td-155" class="table-left table-vtop" align="left" valign="top">100% sensitivity</td><td colspan="1" rowspan="1" id="td-156" class="table-center table-vtop" align="center" valign="top"></td></tr><tr id="tr-51"><td colspan="1" rowspan="1" id="td-157" class="table-left table-vtop" align="left" valign="top"></td><td colspan="1" rowspan="1" id="td-158" class="table-center table-vtop" align="center" valign="top"></td><td colspan="1" rowspan="1" id="td-159" class="table-left table-vtop" align="left" valign="top">86% specificity</td><td colspan="1" rowspan="1" id="td-160" class="table-center table-vtop" align="center" valign="top"></td></tr><tr id="tr-52"><td colspan="1" rowspan="1" id="td-161" class="table-left table-vtop" align="left" valign="top">All genes in list<a id="xref-fn-15-1" class="xref-table-fn" href="#fn-15"><sup>â¡</sup></a></td><td colspan="1" rowspan="1" id="td-162" class="table-center table-vtop" align="center" valign="top"></td><td colspan="1" rowspan="1" id="td-163" class="table-left table-vtop" align="left" valign="top"></td><td colspan="1" rowspan="1" id="td-164" class="table-center table-vtop" align="center" valign="top"></td></tr><tr id="tr-53"><td colspan="1" rowspan="1" id="td-165" class="table-left table-vtop" align="left" valign="top">ââââ<em>KNN</em></td><td colspan="1" rowspan="1" char="." id="td-166" class="table-char table-vtop" align="char" valign="top">42</td><td colspan="1" rowspan="1" id="td-167" class="table-left table-vtop" align="left" valign="top">1 (SS predicted as C)</td><td colspan="1" rowspan="1" id="td-168" class="table-center table-vtop" align="center" valign="top">1 (SS)</td></tr><tr id="tr-54"><td colspan="1" rowspan="1" id="td-169" class="table-left table-vtop" align="left" valign="top">ââââ<em>SVM</em></td><td colspan="1" rowspan="1" char="." id="td-170" class="table-char table-vtop" align="char" valign="top">44</td><td colspan="1" rowspan="1" id="td-171" class="table-left table-vtop" align="left" valign="top">0</td><td colspan="1" rowspan="1" id="td-172" class="table-center table-vtop" align="center" valign="top">n/a</td></tr><tr id="tr-55"><td colspan="1" rowspan="1" id="td-173" class="table-left table-vtop" align="left" valign="top"></td><td colspan="1" rowspan="1" id="td-174" class="table-center table-vtop" align="center" valign="top"></td><td colspan="1" rowspan="1" id="td-175" class="table-left table-vtop" align="left" valign="top">100% sensitivity</td><td colspan="1" rowspan="1" id="td-176" class="table-center table-vtop" align="center" valign="top"></td></tr><tr id="tr-56"><td colspan="1" rowspan="1" id="td-177" class="table-left table-vtop" align="left" valign="top"></td><td colspan="1" rowspan="1" id="td-178" class="table-center table-vtop" align="center" valign="top"></td><td colspan="1" rowspan="1" id="td-179" class="table-left table-vtop" align="left" valign="top">100% specificity</td><td colspan="1" rowspan="1" id="td-180" class="table-center table-vtop" align="center" valign="top"></td></tr></tbody></table><div class="table-foot"><ul class="table-footnotes"><li class="fn" id="fn-12"><p id="p-61">The class options for the model were SS or C.</p></li><li class="fn" id="fn-13"><p id="p-62"><a class="rev-xref" href="#xref-fn-13-1">âµ</a><span class="fn-label">*</span> GeneSpring parameters (default): 50 predictor genes, 10 neighbors, 0.2 decision cutoff for <em>P</em> value ratio.</p></li><li class="fn" id="fn-14"><p id="p-63"><a class="rev-xref" href="#xref-fn-14-1">âµ</a><span class="fn-label">â </span> GeneSpring parameters (default): 50 predictor genes, 10 neighbors, 0.2 decision cutoff for <em>P</em> value ratio.</p></li><li class="fn" id="fn-15"><p id="p-64"><a class="rev-xref" href="#xref-fn-15-1">âµ</a><span class="fn-label">â¡</span> GeneSpring parameters (default): 10 neighbors, 0.2 decision cutoff for <em>P</em> value ratio.</p></li></ul></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></li><li><div class="element-fig-data clearfix -caption"><div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="table-expansion" id="T7"><span class="highwire-journal-article-marker-start"></span><div class="table-caption"><span class="table-label">Table 7.</span> <p id="p-65"><em>Signaling pathways among 50 predictor genes derived from class prediction modeling and the Golub method of gene selection</em></p><div class="sb-div caption-clear"></div></div><table id="table-7"><thead id="thead-7"><tr id="tr-57"><th colspan="1" rowspan="1" id="th-22" class="table-center table-vbottom" align="center" valign="bottom">Signaling Pathway</th><th colspan="1" rowspan="1" id="th-23" class="table-center table-vbottom" align="center" valign="bottom"># of Genes</th><th colspan="1" rowspan="1" id="th-24" class="table-center table-vbottom" align="center" valign="bottom"><em>P</em> Value</th></tr></thead><tbody id="tbody-7"><tr id="tr-58"><td colspan="1" rowspan="1" id="td-181" class="table-left table-vtop" align="left" valign="top">NF-ÎºB signaling</td><td colspan="1" rowspan="1" char="." id="td-182" class="table-char table-vtop" align="char" valign="top">3</td><td colspan="1" rowspan="1" id="td-183" class="table-center table-vtop" align="center" valign="top">6.2E-3</td></tr><tr id="tr-59"><td colspan="1" rowspan="1" id="td-184" class="table-left table-vtop" align="left" valign="top">Leukocyte extravasation signaling</td><td colspan="1" rowspan="1" char="." id="td-185" class="table-char table-vtop" align="char" valign="top">3</td><td colspan="1" rowspan="1" id="td-186" class="table-center table-vtop" align="center" valign="top">1.3E-2</td></tr><tr id="tr-60"><td colspan="1" rowspan="1" id="td-187" class="table-left table-vtop" align="left" valign="top">Liver X receptor/retinoid X receptor signaling</td><td colspan="1" rowspan="1" char="." id="td-188" class="table-char table-vtop" align="char" valign="top">2</td><td colspan="1" rowspan="1" id="td-189" class="table-center table-vtop" align="center" valign="top">1.3E-2</td></tr><tr id="tr-61"><td colspan="1" rowspan="1" id="td-190" class="table-left table-vtop" align="left" valign="top">T cell receptor signaling</td><td colspan="1" rowspan="1" char="." id="td-191" class="table-char table-vtop" align="char" valign="top">2</td><td colspan="1" rowspan="1" id="td-192" class="table-center table-vtop" align="center" valign="top">2.6E-2</td></tr><tr id="tr-62"><td colspan="1" rowspan="1" id="td-193" class="table-left table-vtop" align="left" valign="top">p38 MAP kinase signaling</td><td colspan="1" rowspan="1" char="." id="td-194" class="table-char table-vtop" align="char" valign="top">2</td><td colspan="1" rowspan="1" id="td-195" class="table-center table-vtop" align="center" valign="top">2.6E-2</td></tr></tbody></table><div class="table-foot"><ul class="table-footnotes"><li class="fn" id="fn-16"><p id="p-66">See
 text for details. The analysis is derived from the Ingenuity Pathways 
Analysis default parameters and the signaling pathways represent the top
 5 most significant <em>P</em> values (listed in descending order).</p></li></ul></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></li><li><div class="element-fig-data clearfix -caption"><div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="table-expansion" id="T8"><span class="highwire-journal-article-marker-start"></span><div class="table-caption"><span class="table-label">Table 8.</span> <p id="p-67"><em>Top 20 functional annotations among 846 upregulated genes in gene list C, based on P value and listed in descending order</em></p><div class="sb-div caption-clear"></div></div><table id="table-8"><thead id="thead-8"><tr id="tr-63"><th colspan="1" rowspan="1" id="th-25" class="table-center table-vbottom" align="center" valign="bottom">Category</th><th colspan="1" rowspan="1" id="th-26" class="table-center table-vbottom" align="center" valign="bottom">Term</th><th colspan="1" rowspan="1" id="th-27" class="table-center table-vbottom" align="center" valign="bottom"># of Genes</th><th colspan="1" rowspan="1" id="th-28" class="table-center table-vbottom" align="center" valign="bottom"><em>P</em> Value</th></tr></thead><tbody id="tbody-8"><tr id="tr-64"><td colspan="1" rowspan="1" id="td-196" class="table-left table-vtop" align="left" valign="top">SP_PIR_KEYWORDS</td><td colspan="1" rowspan="1" id="td-197" class="table-left table-vtop" align="left" valign="top">membrane</td><td colspan="1" rowspan="1" char="." id="td-198" class="table-char table-vtop" align="char" valign="top">164</td><td colspan="1" rowspan="1" id="td-199" class="table-center table-vtop" align="center" valign="top">2.9E-16</td></tr><tr id="tr-65"><td colspan="1" rowspan="1" id="td-200" class="table-left table-vtop" align="left" valign="top">GOTERM_BP_ALL</td><td colspan="1" rowspan="1" id="td-201" class="table-left table-vtop" align="left" valign="top">response to other organism</td><td colspan="1" rowspan="1" char="." id="td-202" class="table-char table-vtop" align="char" valign="top">52</td><td colspan="1" rowspan="1" id="td-203" class="table-center table-vtop" align="center" valign="top">7.7E-12</td></tr><tr id="tr-66"><td colspan="1" rowspan="1" id="td-204" class="table-left table-vtop" align="left" valign="top">GOTERM_BP_ALL</td><td colspan="1" rowspan="1" id="td-205" class="table-left table-vtop" align="left" valign="top">response to external stimulus</td><td colspan="1" rowspan="1" char="." id="td-206" class="table-char table-vtop" align="char" valign="top">47</td><td colspan="1" rowspan="1" id="td-207" class="table-center table-vtop" align="center" valign="top">1.9E-11</td></tr><tr id="tr-67"><td colspan="1" rowspan="1" id="td-208" class="table-left table-vtop" align="left" valign="top">GOTERM_BP_ALL</td><td colspan="1" rowspan="1" id="td-209" class="table-left table-vtop" align="left" valign="top">response to pest/pathogen/parasite</td><td colspan="1" rowspan="1" char="." id="td-210" class="table-char table-vtop" align="char" valign="top">49</td><td colspan="1" rowspan="1" id="td-211" class="table-center table-vtop" align="center" valign="top">3.3E-11</td></tr><tr id="tr-68"><td colspan="1" rowspan="1" id="td-212" class="table-left table-vtop" align="left" valign="top">SP_PIR_KEYWORDS</td><td colspan="1" rowspan="1" id="td-213" class="table-left table-vtop" align="left" valign="top">glycoprotein</td><td colspan="1" rowspan="1" char="." id="td-214" class="table-char table-vtop" align="char" valign="top">136</td><td colspan="1" rowspan="1" id="td-215" class="table-center table-vtop" align="center" valign="top">7.9E-11</td></tr><tr id="tr-69"><td colspan="1" rowspan="1" id="td-216" class="table-left table-vtop" align="left" valign="top">GOTERM_BP_ALL</td><td colspan="1" rowspan="1" id="td-217" class="table-left table-vtop" align="left" valign="top">response to wounding</td><td colspan="1" rowspan="1" char="." id="td-218" class="table-char table-vtop" align="char" valign="top">38</td><td colspan="1" rowspan="1" id="td-219" class="table-center table-vtop" align="center" valign="top">5.3E-10</td></tr><tr id="tr-70"><td colspan="1" rowspan="1" id="td-220" class="table-left table-vtop" align="left" valign="top">SP_PIR_KEYWORDS</td><td colspan="1" rowspan="1" id="td-221" class="table-left table-vtop" align="left" valign="top">lipoprotein</td><td colspan="1" rowspan="1" char="." id="td-222" class="table-char table-vtop" align="char" valign="top">40</td><td colspan="1" rowspan="1" id="td-223" class="table-center table-vtop" align="center" valign="top">5.6E-10</td></tr><tr id="tr-71"><td colspan="1" rowspan="1" id="td-224" class="table-left table-vtop" align="left" valign="top">SP_PIR_KEYWORDS</td><td colspan="1" rowspan="1" id="td-225" class="table-left table-vtop" align="left" valign="top">direct protein sequencing</td><td colspan="1" rowspan="1" char="." id="td-226" class="table-char table-vtop" align="char" valign="top">94</td><td colspan="1" rowspan="1" id="td-227" class="table-center table-vtop" align="center" valign="top">6.9E-10</td></tr><tr id="tr-72"><td colspan="1" rowspan="1" id="td-228" class="table-left table-vtop" align="left" valign="top">SP_PIR_KEYWORDS</td><td colspan="1" rowspan="1" id="td-229" class="table-left table-vtop" align="left" valign="top">phosphorylation</td><td colspan="1" rowspan="1" char="." id="td-230" class="table-char table-vtop" align="char" valign="top">87</td><td colspan="1" rowspan="1" id="td-231" class="table-center table-vtop" align="center" valign="top">2.4E-9</td></tr><tr id="tr-73"><td colspan="1" rowspan="1" id="td-232" class="table-left table-vtop" align="left" valign="top">GOTERM_BP_ALL</td><td colspan="1" rowspan="1" id="td-233" class="table-left table-vtop" align="left" valign="top">inflammatory response</td><td colspan="1" rowspan="1" char="." id="td-234" class="table-char table-vtop" align="char" valign="top">24</td><td colspan="1" rowspan="1" id="td-235" class="table-center table-vtop" align="center" valign="top">6.9E-8</td></tr><tr id="tr-74"><td colspan="1" rowspan="1" id="td-236" class="table-left table-vtop" align="left" valign="top">SP_PIR_KEYWORDS</td><td colspan="1" rowspan="1" id="td-237" class="table-left table-vtop" align="left" valign="top">signal</td><td colspan="1" rowspan="1" char="." id="td-238" class="table-char table-vtop" align="char" valign="top">101</td><td colspan="1" rowspan="1" id="td-239" class="table-center table-vtop" align="center" valign="top">7.4E-8</td></tr><tr id="tr-75"><td colspan="1" rowspan="1" id="td-240" class="table-left table-vtop" align="left" valign="top">GOTERM_BP_ALL</td><td colspan="1" rowspan="1" id="td-241" class="table-left table-vtop" align="left" valign="top">response to stress</td><td colspan="1" rowspan="1" char="." id="td-242" class="table-char table-vtop" align="char" valign="top">65</td><td colspan="1" rowspan="1" id="td-243" class="table-center table-vtop" align="center" valign="top">1.1E-7</td></tr><tr id="tr-76"><td colspan="1" rowspan="1" id="td-244" class="table-left table-vtop" align="left" valign="top">SP_PIR_KEYWORDS</td><td colspan="1" rowspan="1" id="td-245" class="table-left table-vtop" align="left" valign="top">transmembrane</td><td colspan="1" rowspan="1" char="." id="td-246" class="table-char table-vtop" align="char" valign="top">134</td><td colspan="1" rowspan="1" id="td-247" class="table-center table-vtop" align="center" valign="top">1.3E-7</td></tr><tr id="tr-77"><td colspan="1" rowspan="1" id="td-248" class="table-left table-vtop" align="left" valign="top">GOTERM_BP_ALL</td><td colspan="1" rowspan="1" id="td-249" class="table-left table-vtop" align="left" valign="top">response to biotic stimulus</td><td colspan="1" rowspan="1" char="." id="td-250" class="table-char table-vtop" align="char" valign="top">73</td><td colspan="1" rowspan="1" id="td-251" class="table-center table-vtop" align="center" valign="top">1.5E-7</td></tr><tr id="tr-78"><td colspan="1" rowspan="1" id="td-252" class="table-left table-vtop" align="left" valign="top">GOTERM_CC_ALL</td><td colspan="1" rowspan="1" id="td-253" class="table-left table-vtop" align="left" valign="top">membrane</td><td colspan="1" rowspan="1" char="." id="td-254" class="table-char table-vtop" align="char" valign="top">216</td><td colspan="1" rowspan="1" id="td-255" class="table-center table-vtop" align="center" valign="top">3.0E-7</td></tr><tr id="tr-79"><td colspan="1" rowspan="1" id="td-256" class="table-left table-vtop" align="left" valign="top">GOTERM_BP_ALL</td><td colspan="1" rowspan="1" id="td-257" class="table-left table-vtop" align="left" valign="top">intracellular signaling cascade</td><td colspan="1" rowspan="1" char="." id="td-258" class="table-char table-vtop" align="char" valign="top">63</td><td colspan="1" rowspan="1" id="td-259" class="table-center table-vtop" align="center" valign="top">9.2E-7</td></tr><tr id="tr-80"><td colspan="1" rowspan="1" id="td-260" class="table-left table-vtop" align="left" valign="top">SP_PIR_KEYWORDS</td><td colspan="1" rowspan="1" id="td-261" class="table-left table-vtop" align="left" valign="top">alternative splicing</td><td colspan="1" rowspan="1" char="." id="td-262" class="table-char table-vtop" align="char" valign="top">130</td><td colspan="1" rowspan="1" id="td-263" class="table-center table-vtop" align="center" valign="top">1.4E-6</td></tr><tr id="tr-81"><td colspan="1" rowspan="1" id="td-264" class="table-left table-vtop" align="left" valign="top">GOTERM_MF_ALL</td><td colspan="1" rowspan="1" id="td-265" class="table-left table-vtop" align="left" valign="top">catalytic activity</td><td colspan="1" rowspan="1" char="." id="td-266" class="table-char table-vtop" align="char" valign="top">208</td><td colspan="1" rowspan="1" id="td-267" class="table-center table-vtop" align="center" valign="top">1.6E-6</td></tr><tr id="tr-82"><td colspan="1" rowspan="1" id="td-268" class="table-left table-vtop" align="left" valign="top">GOTERM_BP_ALL</td><td colspan="1" rowspan="1" id="td-269" class="table-left table-vtop" align="left" valign="top">immune response</td><td colspan="1" rowspan="1" char="." id="td-270" class="table-char table-vtop" align="char" valign="top">63</td><td colspan="1" rowspan="1" id="td-271" class="table-center table-vtop" align="center" valign="top">1.7E-6</td></tr><tr id="tr-83"><td colspan="1" rowspan="1" id="td-272" class="table-left table-vtop" align="left" valign="top">SP_PIR_KEYWORDS</td><td colspan="1" rowspan="1" id="td-273" class="table-left table-vtop" align="left" valign="top">transferase</td><td colspan="1" rowspan="1" char="." id="td-274" class="table-char table-vtop" align="char" valign="top">60</td><td colspan="1" rowspan="1" id="td-275" class="table-center table-vtop" align="center" valign="top">1.9E-6</td></tr></tbody></table><div class="table-foot"><ul class="table-footnotes"><li class="fn" id="fn-17"><p id="p-68">The
 analysis is based on the default parameters in D.A.V.I.D. âCategoryâ 
refers to the original database or resource from which the annotations 
were derived, and âTermâ refers to the enriched annotation terms 
associated with the gene list.</p></li></ul></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></li><li><div class="element-fig-data clearfix -caption"><div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="table-expansion" id="T9"><span class="highwire-journal-article-marker-start"></span><div class="table-caption"><span class="table-label">Table 9.</span> <p id="p-69"><em>Top 20 functional annotations among 1,258 downregulated genes in gene list C, based on P value and listed in descending order</em></p><div class="sb-div caption-clear"></div></div><table id="table-9"><thead id="thead-9"><tr id="tr-84"><th colspan="1" rowspan="1" id="th-29" class="table-center table-vbottom" align="center" valign="bottom">Category</th><th colspan="1" rowspan="1" id="th-30" class="table-center table-vbottom" align="center" valign="bottom">Term</th><th colspan="1" rowspan="1" id="th-31" class="table-center table-vbottom" align="center" valign="bottom"># of Genes</th><th colspan="1" rowspan="1" id="th-32" class="table-center table-vbottom" align="center" valign="bottom"><em>P</em> Value</th></tr></thead><tbody id="tbody-9"><tr id="tr-85"><td colspan="1" rowspan="1" id="td-276" class="table-left table-vtop" align="left" valign="top">SP_PIR_KEYWORDS</td><td colspan="1" rowspan="1" id="td-277" class="table-left table-vtop" align="left" valign="top">zinc-finger</td><td colspan="1" rowspan="1" char="." id="td-278" class="table-char table-vtop" align="char" valign="top">137</td><td colspan="1" rowspan="1" id="td-279" class="table-center table-vtop" align="center" valign="top">2.7E-29</td></tr><tr id="tr-86"><td colspan="1" rowspan="1" id="td-280" class="table-left table-vtop" align="left" valign="top">SP_PIR_KEYWORDS</td><td colspan="1" rowspan="1" id="td-281" class="table-left table-vtop" align="left" valign="top">nuclear protein</td><td colspan="1" rowspan="1" char="." id="td-282" class="table-char table-vtop" align="char" valign="top">209</td><td colspan="1" rowspan="1" id="td-283" class="table-center table-vtop" align="center" valign="top">2.6E-26</td></tr><tr id="tr-87"><td colspan="1" rowspan="1" id="td-284" class="table-left table-vtop" align="left" valign="top">SP_PIR_KEYWORDS</td><td colspan="1" rowspan="1" id="td-285" class="table-left table-vtop" align="left" valign="top">zinc</td><td colspan="1" rowspan="1" char="." id="td-286" class="table-char table-vtop" align="char" valign="top">144</td><td colspan="1" rowspan="1" id="td-287" class="table-center table-vtop" align="center" valign="top">4.0E-24</td></tr><tr id="tr-88"><td colspan="1" rowspan="1" id="td-288" class="table-left table-vtop" align="left" valign="top">SP_PIR_KEYWORDS</td><td colspan="1" rowspan="1" id="td-289" class="table-left table-vtop" align="left" valign="top">transcription</td><td colspan="1" rowspan="1" char="." id="td-290" class="table-char table-vtop" align="char" valign="top">124</td><td colspan="1" rowspan="1" id="td-291" class="table-center table-vtop" align="center" valign="top">1.1E-22</td></tr><tr id="tr-89"><td colspan="1" rowspan="1" id="td-292" class="table-left table-vtop" align="left" valign="top">SP_PIR_KEYWORDS</td><td colspan="1" rowspan="1" id="td-293" class="table-left table-vtop" align="left" valign="top">transcription regulation</td><td colspan="1" rowspan="1" char="." id="td-294" class="table-char table-vtop" align="char" valign="top">122</td><td colspan="1" rowspan="1" id="td-295" class="table-center table-vtop" align="center" valign="top">5.0E-21</td></tr><tr id="tr-90"><td colspan="1" rowspan="1" id="td-296" class="table-left table-vtop" align="left" valign="top">SP_PIR_KEYWORDS</td><td colspan="1" rowspan="1" id="td-297" class="table-left table-vtop" align="left" valign="top">DNA binding</td><td colspan="1" rowspan="1" char="." id="td-298" class="table-char table-vtop" align="char" valign="top">119</td><td colspan="1" rowspan="1" id="td-299" class="table-center table-vtop" align="center" valign="top">2.7E-19</td></tr><tr id="tr-91"><td colspan="1" rowspan="1" id="td-300" class="table-left table-vtop" align="left" valign="top">GOTERM_MF_ALL</td><td colspan="1" rowspan="1" id="td-301" class="table-left table-vtop" align="left" valign="top">zind ion binding</td><td colspan="1" rowspan="1" char="." id="td-302" class="table-char table-vtop" align="char" valign="top">161</td><td colspan="1" rowspan="1" id="td-303" class="table-center table-vtop" align="center" valign="top">4.7E-19</td></tr><tr id="tr-92"><td colspan="1" rowspan="1" id="td-304" class="table-left table-vtop" align="left" valign="top">SP_PIR_KEYWORDS</td><td colspan="1" rowspan="1" id="td-305" class="table-left table-vtop" align="left" valign="top">metal-binding</td><td colspan="1" rowspan="1" char="." id="td-306" class="table-char table-vtop" align="char" valign="top">149</td><td colspan="1" rowspan="1" id="td-307" class="table-center table-vtop" align="center" valign="top">3.2E-17</td></tr><tr id="tr-93"><td colspan="1" rowspan="1" id="td-308" class="table-left table-vtop" align="left" valign="top">GOTERM_MF_ALL</td><td colspan="1" rowspan="1" id="td-309" class="table-left table-vtop" align="left" valign="top">transition metal binding</td><td colspan="1" rowspan="1" char="." id="td-310" class="table-char table-vtop" align="char" valign="top">174</td><td colspan="1" rowspan="1" id="td-311" class="table-center table-vtop" align="center" valign="top">7.9E-16</td></tr><tr id="tr-94"><td colspan="1" rowspan="1" id="td-312" class="table-left table-vtop" align="left" valign="top">GOTERM_BP_ALL</td><td colspan="1" rowspan="1" id="td-313" class="table-left table-vtop" align="left" valign="top">transcription</td><td colspan="1" rowspan="1" char="." id="td-314" class="table-char table-vtop" align="char" valign="top">171</td><td colspan="1" rowspan="1" id="td-315" class="table-center table-vtop" align="center" valign="top">9.4E-16</td></tr><tr id="tr-95"><td colspan="1" rowspan="1" id="td-316" class="table-left table-vtop" align="left" valign="top">GOTERM_BP_ALL</td><td colspan="1" rowspan="1" id="td-317" class="table-left table-vtop" align="left" valign="top">regulation of transcription</td><td colspan="1" rowspan="1" char="." id="td-318" class="table-char table-vtop" align="char" valign="top">163</td><td colspan="1" rowspan="1" id="td-319" class="table-center table-vtop" align="center" valign="top">7.1E-15</td></tr><tr id="tr-96"><td colspan="1" rowspan="1" id="td-320" class="table-left table-vtop" align="left" valign="top">GOTERM_MF_AL</td><td colspan="1" rowspan="1" id="td-321" class="table-left table-vtop" align="left" valign="top">nucleic acid binding</td><td colspan="1" rowspan="1" char="." id="td-322" class="table-char table-vtop" align="char" valign="top">218</td><td colspan="1" rowspan="1" id="td-323" class="table-center table-vtop" align="center" valign="top">1.3E-14</td></tr><tr id="tr-97"><td colspan="1" rowspan="1" id="td-324" class="table-left table-vtop" align="left" valign="top">GOTERM_BP_ALL</td><td colspan="1" rowspan="1" id="td-325" class="table-left table-vtop" align="left" valign="top">regulation of biological process</td><td colspan="1" rowspan="1" char="." id="td-326" class="table-char table-vtop" align="char" valign="top">225</td><td colspan="1" rowspan="1" id="td-327" class="table-center table-vtop" align="center" valign="top">3.8E-13</td></tr><tr id="tr-98"><td colspan="1" rowspan="1" id="td-328" class="table-left table-vtop" align="left" valign="top">GOTERM_BP_ALL</td><td colspan="1" rowspan="1" id="td-329" class="table-left table-vtop" align="left" valign="top">regulation of cellular process</td><td colspan="1" rowspan="1" char="." id="td-330" class="table-char table-vtop" align="char" valign="top">214</td><td colspan="1" rowspan="1" id="td-331" class="table-center table-vtop" align="center" valign="top">4.4E-13</td></tr><tr id="tr-99"><td colspan="1" rowspan="1" id="td-332" class="table-left table-vtop" align="left" valign="top">GOTERM_BP_ALL</td><td colspan="1" rowspan="1" id="td-333" class="table-left table-vtop" align="left" valign="top">regulation of cellular metabolism</td><td colspan="1" rowspan="1" char="." id="td-334" class="table-char table-vtop" align="char" valign="top">165</td><td colspan="1" rowspan="1" id="td-335" class="table-center table-vtop" align="center" valign="top">6.8E-13</td></tr><tr id="tr-100"><td colspan="1" rowspan="1" id="td-336" class="table-left table-vtop" align="left" valign="top">GOTERM_BP_ALL</td><td colspan="1" rowspan="1" id="td-337" class="table-left table-vtop" align="left" valign="top">regulation of metabolism</td><td colspan="1" rowspan="1" char="." id="td-338" class="table-char table-vtop" align="char" valign="top">168</td><td colspan="1" rowspan="1" id="td-339" class="table-center table-vtop" align="center" valign="top">9.1E-13</td></tr><tr id="tr-101"><td colspan="1" rowspan="1" id="td-340" class="table-left table-vtop" align="left" valign="top">GOTERM_CC_ALL</td><td colspan="1" rowspan="1" id="td-341" class="table-left table-vtop" align="left" valign="top">nucleus</td><td colspan="1" rowspan="1" char="." id="td-342" class="table-char table-vtop" align="char" valign="top">239</td><td colspan="1" rowspan="1" id="td-343" class="table-center table-vtop" align="center" valign="top">9.0E-12</td></tr><tr id="tr-102"><td colspan="1" rowspan="1" id="td-344" class="table-left table-vtop" align="left" valign="top">GOTERM_MF_AL</td><td colspan="1" rowspan="1" id="td-345" class="table-left table-vtop" align="left" valign="top">cation binding</td><td colspan="1" rowspan="1" char="." id="td-346" class="table-char table-vtop" align="char" valign="top">196</td><td colspan="1" rowspan="1" id="td-347" class="table-center table-vtop" align="center" valign="top">2.6E-9</td></tr><tr id="tr-103"><td colspan="1" rowspan="1" id="td-348" class="table-left table-vtop" align="left" valign="top">GOTERM_MF_ALL</td><td colspan="1" rowspan="1" id="td-349" class="table-left table-vtop" align="left" valign="top">ion binding</td><td colspan="1" rowspan="1" char="." id="td-350" class="table-char table-vtop" align="char" valign="top">201</td><td colspan="1" rowspan="1" id="td-351" class="table-center table-vtop" align="center" valign="top">5.1E-8</td></tr><tr id="tr-104"><td colspan="1" rowspan="1" id="td-352" class="table-left table-vtop" align="left" valign="top">GOTERM_MF_AL</td><td colspan="1" rowspan="1" id="td-353" class="table-left table-vtop" align="left" valign="top">metal ion binding</td><td colspan="1" rowspan="1" char="." id="td-354" class="table-char table-vtop" align="char" valign="top">201</td><td colspan="1" rowspan="1" id="td-355" class="table-center table-vtop" align="center" valign="top">5.1E-8</td></tr></tbody></table><div class="table-foot"><ul class="table-footnotes"><li class="fn" id="fn-18"><p id="p-70">The
 analysis is based on the default parameters in D.A.V.I.D. âCategoryâ 
refers to the original database or resource from which the annotations 
were derived, and âTermâ refers to the enriched annotation terms 
associated with the gene list.</p></li></ul></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></li><li><div class="element-fig-data clearfix -caption"><div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="table-expansion" id="T10"><span class="highwire-journal-article-marker-start"></span><div class="table-caption"><span class="table-label">Table 10.</span> <p id="p-71"><em>Signaling pathways among 846 upregulated genes in gene list C</em></p><div class="sb-div caption-clear"></div></div><table id="table-10"><thead id="thead-10"><tr id="tr-105"><th colspan="1" rowspan="1" id="th-33" class="table-center table-vbottom" align="center" valign="bottom">Signaling Pathway</th><th colspan="1" rowspan="1" id="th-34" class="table-center table-vbottom" align="center" valign="bottom"># of Genes</th><th colspan="1" rowspan="1" id="th-35" class="table-center table-vbottom" align="center" valign="bottom"><em>P</em> Value</th></tr></thead><tbody id="tbody-10"><tr id="tr-106"><td colspan="1" rowspan="1" id="td-356" class="table-left table-vtop" align="left" valign="top">Toll-like receptor signaling</td><td colspan="1" rowspan="1" char="." id="td-357" class="table-char table-vtop" align="char" valign="top">10</td><td colspan="1" rowspan="1" id="td-358" class="table-center table-vtop" align="center" valign="top">2.6E-6</td></tr><tr id="tr-107"><td colspan="1" rowspan="1" id="td-359" class="table-left table-vtop" align="left" valign="top">Interleukin-10 signaling</td><td colspan="1" rowspan="1" char="." id="td-360" class="table-char table-vtop" align="char" valign="top">10</td><td colspan="1" rowspan="1" id="td-361" class="table-center table-vtop" align="center" valign="top">2.8E-5</td></tr><tr id="tr-108"><td colspan="1" rowspan="1" id="td-362" class="table-left table-vtop" align="left" valign="top">NF-ÎºB signaling</td><td colspan="1" rowspan="1" char="." id="td-363" class="table-char table-vtop" align="char" valign="top">17</td><td colspan="1" rowspan="1" id="td-364" class="table-center table-vtop" align="center" valign="top">3.6E-4</td></tr><tr id="tr-109"><td colspan="1" rowspan="1" id="td-365" class="table-left table-vtop" align="left" valign="top">Acute phase response signaling</td><td colspan="1" rowspan="1" char="." id="td-366" class="table-char table-vtop" align="char" valign="top">14</td><td colspan="1" rowspan="1" id="td-367" class="table-center table-vtop" align="center" valign="top">1.6E-3</td></tr><tr id="tr-110"><td colspan="1" rowspan="1" id="td-368" class="table-left table-vtop" align="left" valign="top">p38 MAP kinase signaling</td><td colspan="1" rowspan="1" char="." id="td-369" class="table-char table-vtop" align="char" valign="top">9</td><td colspan="1" rowspan="1" id="td-370" class="table-center table-vtop" align="center" valign="top">5.2E-3</td></tr><tr id="tr-111"><td colspan="1" rowspan="1" id="td-371" class="table-left table-vtop" align="left" valign="top">Complement system</td><td colspan="1" rowspan="1" char="." id="td-372" class="table-char table-vtop" align="char" valign="top">5</td><td colspan="1" rowspan="1" id="td-373" class="table-center table-vtop" align="center" valign="top">5.4E-3</td></tr><tr id="tr-112"><td colspan="1" rowspan="1" id="td-374" class="table-left table-vtop" align="left" valign="top">Hepatic cholestasis</td><td colspan="1" rowspan="1" char="." id="td-375" class="table-char table-vtop" align="char" valign="top">11</td><td colspan="1" rowspan="1" id="td-376" class="table-center table-vtop" align="center" valign="top">5.5E-3</td></tr><tr id="tr-113"><td colspan="1" rowspan="1" id="td-377" class="table-left table-vtop" align="left" valign="top">LXR/RXRÎ± activation</td><td colspan="1" rowspan="1" char="." id="td-378" class="table-char table-vtop" align="char" valign="top">7</td><td colspan="1" rowspan="1" id="td-379" class="table-center table-vtop" align="center" valign="top">6.8E-3</td></tr><tr id="tr-114"><td colspan="1" rowspan="1" id="td-380" class="table-left table-vtop" align="left" valign="top">Interleukin-6 signaling</td><td colspan="1" rowspan="1" char="." id="td-381" class="table-char table-vtop" align="char" valign="top">8</td><td colspan="1" rowspan="1" id="td-382" class="table-center table-vtop" align="center" valign="top">9.6E-3</td></tr><tr id="tr-115"><td colspan="1" rowspan="1" id="td-383" class="table-left table-vtop" align="left" valign="top">PPARÎ±/RXRÎ± activation</td><td colspan="1" rowspan="1" char="." id="td-384" class="table-char table-vtop" align="char" valign="top">12</td><td colspan="1" rowspan="1" id="td-385" class="table-center table-vtop" align="center" valign="top">1.2E-2</td></tr></tbody></table><div class="table-foot"><ul class="table-footnotes"><li class="fn" id="fn-19"><p id="p-72">The
 analysis is derived from the Ingenuity Pathways Analysis default 
parameters, and the signaling pathways represent the top 10 most 
significant <em>P</em> values (listed in descending order).</p></li></ul></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></li><li class="last"><div class="element-fig-data clearfix -caption"><div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="table-expansion" id="T11"><span class="highwire-journal-article-marker-start"></span><div class="table-caption"><span class="table-label">Table 11.</span> <p id="p-73"><em>Signaling pathways among the 1,258 downregulated genes in gene list C</em></p><div class="sb-div caption-clear"></div></div><table id="table-11"><thead id="thead-11"><tr id="tr-116"><th colspan="1" rowspan="1" id="th-36" class="table-center table-vbottom" align="center" valign="bottom">Signaling Pathway</th><th colspan="1" rowspan="1" id="th-37" class="table-center table-vbottom" align="center" valign="bottom"># of Genes</th><th colspan="1" rowspan="1" id="th-38" class="table-center table-vbottom" align="center" valign="bottom"><em>P</em> Value</th></tr></thead><tbody id="tbody-11"><tr id="tr-117"><td colspan="1" rowspan="1" id="td-386" class="table-left table-vtop" align="left" valign="top">Natural killer cell signaling</td><td colspan="1" rowspan="1" char="." id="td-387" class="table-char table-vtop" align="char" valign="top">18</td><td colspan="1" rowspan="1" id="td-388" class="table-center table-vtop" align="center" valign="top">1.0E-7</td></tr><tr id="tr-118"><td colspan="1" rowspan="1" id="td-389" class="table-left table-vtop" align="left" valign="top">T-cell receptor signaling</td><td colspan="1" rowspan="1" char="." id="td-390" class="table-char table-vtop" align="char" valign="top">15</td><td colspan="1" rowspan="1" id="td-391" class="table-center table-vtop" align="center" valign="top">3.0E-6</td></tr><tr id="tr-119"><td colspan="1" rowspan="1" id="td-392" class="table-left table-vtop" align="left" valign="top">Antigen presentation pathway</td><td colspan="1" rowspan="1" char="." id="td-393" class="table-char table-vtop" align="char" valign="top">7</td><td colspan="1" rowspan="1" id="td-394" class="table-center table-vtop" align="center" valign="top">4.6E-4</td></tr><tr id="tr-120"><td colspan="1" rowspan="1" id="td-395" class="table-left table-vtop" align="left" valign="top">Interleukin-4 signaling</td><td colspan="1" rowspan="1" char="." id="td-396" class="table-char table-vtop" align="char" valign="top">8</td><td colspan="1" rowspan="1" id="td-397" class="table-center table-vtop" align="center" valign="top">3.7E-3</td></tr></tbody></table><div class="table-foot"><ul class="table-footnotes"><li class="fn" id="fn-20"><p id="p-74">The
 analysis is derived from the Ingenuity Pathways Analysis default 
parameters, and the signaling pathways represent the top 5 most 
significant <em>P</em> values (listed in descending order).</p></li></ul></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></li></ul></div></div><div id="fig-data-additional-files" class="group frag-additional-files"><div class="fig-data-title-jump clearfix"><h3 class="fig-data-group-title">Additional Files</h3><div class="fig-data-jump-links"><ul class="fig-data-jump-links-list links inline"><li class="figure first"><a href="#fig-data-figures" class="fig-data-jump-link fig-data-jump-link-figure link-icon"><i class="icon-caret-up"></i> <span class="title">Figures</span></a></li><li class="table last"><a href="#fig-data-tables" class="fig-data-jump-link fig-data-jump-link-table link-icon"><i class="icon-caret-up"></i> <span class="title">Tables</span></a></li></ul></div></div><div class="item-list"><ul class="fig-data-list clearfix" id="fragments-additional-data"><li class="first last"><div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div><span class="highwire-journal-article-marker-start"></span><div class="auto-clean"><span style="font-family: Verdana,Arial,Helvetica,sans-serif; font-size: 83.33%">
         
            <h2>Supplemental Tables</h2>
            <p>Four tables in Excel format.</p>
            <p><strong>Files in this Data Supplement:</strong></p>
            <ul><li><a href="http://physiolgenomics.physiology.org/highwire/filestream/19592/field_highwire_adjunct_files/0/Supplementary_data_50_predictor_gene_list_Cvijanovich_et_al.xls" class="" data-icon-position="" data-hide-link-title="0">Table S1</a>
                  		-
                  		50 predictor gene list
               </li><li><a href="http://physiolgenomics.physiology.org/highwire/filestream/19592/field_highwire_adjunct_files/1/Supplementary_data_gene_lists_downregulated_signaling_pathways_Cvijanovich_et_al.xls" class="" data-icon-position="" data-hide-link-title="0">Table S2</a>
                  		-
                  		downregulated signaling pathways
               </li><li><a href="http://physiolgenomics.physiology.org/highwire/filestream/19592/field_highwire_adjunct_files/2/Supplementary_data_gene_lists_upregulated_signaling_pathways_Cvijanovich_et_al.xls" class="" data-icon-position="" data-hide-link-title="0">Table S3</a>
                  		-
                  		upregulated signaling pathways
               </li><li><a href="http://physiolgenomics.physiology.org/highwire/filestream/19592/field_highwire_adjunct_files/3/Supplementary_data_gene_lists_zinc_related_functional_annotations_Cvijanovich_et_al.xls" class="" data-icon-position="" data-hide-link-title="0">Table S4</a>
                  		-
                  		zinc-related functional annotations
               </li></ul>
         </span>
      
   </div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></li></ul></div></div></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager">
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="http://physiolgenomics.physiology.org/content/34/1/112" title="The in vivo gene expression signature of oxidative stress" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous</span></a></span><span class="pager-next"><a href="http://physiolgenomics.physiology.org/content/34/1/22" title="Alternative splicing and exon duplication generates 10 unique porcine 5-HT4 receptor splice variants including a functional homofusion variant " rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top">
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-physiolgen-issue-arch-art pane-style-alt-content">
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_physiolgen_issue_arch_art" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"></div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-variant-list no-margin-bottom">
  
      
  
  <div class="pane-content">
    <ul class="links variants-list menu"><li class="toc first"><a href="http://physiolgenomics.physiology.org/content/34/1.toc" class="highwire-variant-link variant-toc " title="Table of Contents" rel="alternative" type="text/html" data-icon-position="" data-hide-link-title="0">Table of Contents</a></li><li class="enclosure-back-matter"><a href="http://physiolgenomics.physiology.org/content/physiolgenomics/34/1/local/back-matter.pdf" class="highwire-variant-link variant-enclosure" title="Back Matter (PDF)" rel="alternative" data-icon-position="" data-hide-link-title="0">Back Matter (PDF)</a></li><li class="enclosure-ed-board"><a href="http://physiolgenomics.physiology.org/content/physiolgenomics/34/1/local/ed-board.pdf" class="highwire-variant-link variant-enclosure" title="Ed Board (PDF)" rel="alternative" data-icon-position="" data-hide-link-title="0">Ed Board (PDF)</a></li><li class="enclosure-front-matter last"><a href="http://physiolgenomics.physiology.org/content/physiolgenomics/34/1/local/front-matter.pdf" class="highwire-variant-link variant-enclosure" title="Front Matter (PDF)" rel="alternative" data-icon-position="" data-hide-link-title="0">Front Matter (PDF)</a></li></ul>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav pane-style-alt-content">
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float highwire-article-nav-processed highwire-nav-float-processed"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="parent first odd"><a href="http://physiolgenomics.physiology.org/content/34/1/127" class="highwire-article-nav-jumplink" data-panel-ajax-tab="physiolgenomics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="physiolgenomics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="physiolgenomics_tab_art" data-icon-position="" data-hide-link-title="0">METHODS</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="physiolgenomics_tab_art" data-icon-position="" data-hide-link-title="0">RESULTS</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="physiolgenomics_tab_art" data-icon-position="" data-hide-link-title="0">DISCUSSION</a></li><li class="odd"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="physiolgenomics_tab_art" data-icon-position="" data-hide-link-title="0">CONTRIBUTORS</a></li><li class="even"><a href="#sec-17" class="highwire-article-nav-jumplink" data-panel-ajax-tab="physiolgenomics_tab_art" data-icon-position="" data-hide-link-title="0">GRANTS</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="physiolgenomics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="physiolgenomics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="physiolgenomics_tab_art" data-icon-position="" data-hide-link-title="0">REFERENCES</a></li></ul></div></li><li class="even"><a href="http://physiolgenomics.physiology.org/content/34/1/127.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="physiolgenomics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures &amp; Data</a></li><li class="odd"><a href="http://physiolgenomics.physiology.org/content/34/1/127.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="physiolgenomics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info</a></li><li class="last even"><a href="http://physiolgenomics.physiology.org/content/34/1/127.full-text.pdf%2Bhtml" class="highwire-article-nav-jumplink" data-panel-ajax-tab="physiolgenomics_tab_pdf" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i> PDF</a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-article-add-link pane-style-alt-content no-margin-bottom">
  
      
  
  <div class="pane-content">
    <a href="http://physiolgenomics.physiology.org/alerts/article/add/4156?type=citedby&amp;destination=node/4156" class="highwire-article-alert link-icon"><i class="icon-bullhorn"></i> <span class="title">Alert me when this article is cited</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-article-add-link pane-style-alt-content">
  
      
  
  <div class="pane-content">
    <a href="http://physiolgenomics.physiology.org/alerts/article/add/4156?type=correction&amp;destination=node/4156" class="highwire-article-alert link-icon"><i class="icon-bullhorn"></i> <span class="title">Alert me if a correction is posted</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-physiolgenomics-art-tools pane-style-alt-content">
  
      
  
  <div class="pane-content">
    <div id="mini-panel-physiolgenomics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-physiolgenomics-art-email">
  
      
  
  <div class="pane-content">
    <div class="minipanel-dialog-wrapper"><div class="minipanel-dialog-link-link"><a href="http://physiolgenomics.physiology.org/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger minipanel-dialog-link-trigger-processed" title="" data-icon-position="" data-hide-link-title="0"><i class="icon-envelope"></i> Email</a></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link">
  
      
  
  <div class="pane-content">
    <a href="http://physiolgenomics.physiology.org/content/34/1/127.full-text.pdf" target="_blank" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-physiolgenomics-cite-tool">
  
      
  
  <div class="pane-content">
    <div class="minipanel-dialog-wrapper"><div class="minipanel-dialog-link-link"><a href="http://physiolgenomics.physiology.org/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon minipanel-dialog-link-trigger-processed" title=""><i class="icon-globe"></i> <span class="title">Citation Tools</span></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-permission-link">
  
      
  
  <div class="pane-content">
    <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=amphysiosoc&amp;publication=1094-8341&amp;title=Validating%20the%20genomic%20signature%20of%20pediatric%20septic%20shock&amp;publicationDate=2008-06-01&amp;author=Natalie%20Cvijanovich%2CThomas%20P.%20Shanley%2CRichard%20Lin%2CGeoffrey%20L.%20Allen%2CNeal%20J.%20Thomas%2CPaul%20Checchia%2CNick%20Anas%2CRobert%20J.%20Freishtat%2CMarie%20Monaco%2CKelli%20Odoms%2CBhuvaneswari%20Sakthivel%2CHector%20R.%20Wong&amp;contentID=physiolgenomics%3B34/1/127&amp;volumeNum=34&amp;issueNum=1&amp;startPage=127&amp;endPage=134&amp;numPages=8&amp;copyright=&amp;orderBeanReset=true" target="_blank" class="highwire-permission-link link-icon"><i class="icon-copyright"></i> <span class="title">Request Permissions</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
  
      
  
  <div class="pane-content">
    <a href="http://physiolgenomics.physiology.org/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-physiolgenomics-share highwire_clipboard_form_ajax_shareit">
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-physiolgenomics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation">
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="4156" id="node4156--2" data-pisa="physiolgenomics;34/1/127" data-pisa-master="physiolgenomics;00025.2008" data-apath="/physiolgenomics/34/1/127.atom"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-jcore-standard clearfix">
    
  
  
  
    	<div class="highwire-cite-authors"><span class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Natalie</span> <span class="nlm-surname">Cvijanovich</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Thomas P.</span> <span class="nlm-surname">Shanley</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Richard</span> <span class="nlm-surname">Lin</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Geoffrey L.</span> <span class="nlm-surname">Allen</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Neal J.</span> <span class="nlm-surname">Thomas</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Paul</span> <span class="nlm-surname">Checchia</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Nick</span> <span class="nlm-surname">Anas</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Robert J.</span> <span class="nlm-surname">Freishtat</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Marie</span> <span class="nlm-surname">Monaco</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Kelli</span> <span class="nlm-surname">Odoms</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Bhuvaneswari</span> <span class="nlm-surname">Sakthivel</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Hector R.</span> <span class="nlm-surname">Wong</span></span></span></div>
  
    	<div class="highwire-cite-metadata"><span class="highwire-cite-metadata-journal highwire-cite-metadata">Physiological Genomics </span><span class="highwire-cite-metadata-date highwire-cite-metadata">Jun 2008, </span><span class="highwire-cite-metadata-volume highwire-cite-metadata">34 </span><span class="highwire-cite-metadata-issue highwire-cite-metadata">(1) </span><span class="highwire-cite-metadata-pages highwire-cite-metadata">127-134; </span><span class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">DOI:</span> 10.1152/physiolgenomics.00025.2008 </span></div>
  
  
  
</div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy">
  
      
  
  <div class="pane-content">
    <div class="clipboard-copy">
  <span class="label-url">Permalink:</span>
  <span class="input-text-url">
    <input id="dynamic" value="http://physiolgenomics.physiology.org/content/34/1/127" size="50" type="text">  </span>
  <span class="copy-button button">
    <a href="http://physiolgenomics.physiology.org/" id="copy-dynamic" class="" data-icon-position="" data-hide-link-title="0">Copy</a>  </span>
</div>  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links">
  
      
  
  <div class="pane-content">
    <div class="service-links"><a data-log-redirect-set="true" href="http://physiolgenomics.physiology.org/highwire_log/share/delicious?link=http%3A%2F%2Fdel.icio.us%2Fpost%3Furl%3Dhttp%253A%2F%2Fphysiolgenomics.physiology.org%2Fcontent%2F34%2F1%2F127%26title%3DValidating%2520the%2520genomic%2520signature%2520of%2520pediatric%2520septic%2520shock" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/delicious-32.png" alt="del.icio.us logo"></a> <a data-log-redirect-set="true" href="http://physiolgenomics.physiology.org/highwire_log/share/digg?link=http%3A%2F%2Fdigg.com%2Fsubmit%3Fphase%3D2%26url%3Dhttp%253A%2F%2Fphysiolgenomics.physiology.org%2Fcontent%2F34%2F1%2F127%26title%3DValidating%2520the%2520genomic%2520signature%2520of%2520pediatric%2520septic%2520shock" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/digg.png" alt="Digg logo"></a> <a data-log-redirect-set="true" href="http://physiolgenomics.physiology.org/highwire_log/share/reddit?link=http%3A%2F%2Freddit.com%2Fsubmit%3Furl%3Dhttp%253A%2F%2Fphysiolgenomics.physiology.org%2Fcontent%2F34%2F1%2F127%26title%3DValidating%2520the%2520genomic%2520signature%2520of%2520pediatric%2520septic%2520shock" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/reddit.png" alt="Reddit logo"></a> <a data-log-redirect-set="true" href="http://physiolgenomics.physiology.org/highwire_log/share/technorati?link=http%3A%2F%2Ftechnorati.com%2Fsearch%2Fhttp%3A%2F%2Fphysiolgenomics.physiology.org%2Fcontent%2F34%2F1%2F127" id="technorati" title="Search Technorati for links to this post" class="service-links-technorati" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/technorati.png" alt="Technorati logo"></a> <a data-log-redirect-set="true" href="http://physiolgenomics.physiology.org/highwire_log/share/twitter?link=http%3A%2F%2Ftwitter.com%2Fshare%3Furl%3Dhttp%253A%2F%2Fphysiolgenomics.physiology.org%2Fcontent%2F34%2F1%2F127%26text%3DValidating%2520the%2520genomic%2520signature%2520of%2520pediatric%2520septic%2520shock" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/twitter.png" alt="Twitter logo"></a> <a data-log-redirect-set="true" href="http://physiolgenomics.physiology.org/highwire_log/share/citeulike?link=http%3A%2F%2Fwww.citeulike.org%2Fposturl%3Furl%3Dhttp%253A%2F%2Fphysiolgenomics.physiology.org%2Fcontent%2F34%2F1%2F127%26title%3DValidating%2520the%2520genomic%2520signature%2520of%2520pediatric%2520septic%2520shock" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/citeyoulike.png" alt="CiteULike logo"></a> <a data-log-redirect-set="true" href="http://physiolgenomics.physiology.org/highwire_log/share/connotea?link=http%3A%2F%2Fwww.connotea.org%2Fadd%3Furi%3Dhttp%253A%2F%2Fphysiolgenomics.physiology.org%2Fcontent%2F34%2F1%2F127%26title%3DValidating%2520the%2520genomic%2520signature%2520of%2520pediatric%2520septic%2520shock" id="connotea" title="Share on connotea" class="service-links-connotea" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/connotea.png" alt="Connotea logo"></a> <a data-log-redirect-set="true" href="http://physiolgenomics.physiology.org/highwire_log/share/facebook?link=http%3A%2F%2Fwww.facebook.com%2Fsharer.php%3Fu%3Dhttp%253A%2F%2Fphysiolgenomics.physiology.org%2Fcontent%2F34%2F1%2F127%26t%3DValidating%2520the%2520genomic%2520signature%2520of%2520pediatric%2520septic%2520shock" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/fb-blue.png" alt="Facebook logo"></a> <a data-log-redirect-set="true" href="http://physiolgenomics.physiology.org/highwire_log/share/google?link=http%3A%2F%2Fwww.google.com%2Fbookmarks%2Fmark%3Fop%3Dadd%26bkmk%3Dhttp%253A%2F%2Fphysiolgenomics.physiology.org%2Fcontent%2F34%2F1%2F127%26title%3DValidating%2520the%2520genomic%2520signature%2520of%2520pediatric%2520septic%2520shock" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/google-32.png" alt="Google logo"></a> <a data-log-redirect-set="true" href="http://physiolgenomics.physiology.org/highwire_log/share/mendeley?link=http%3A%2F%2Fwww.mendeley.com%2Fimport%2F%3Furl%3Dhttp%253A%2F%2Fphysiolgenomics.physiology.org%2Fcontent%2F34%2F1%2F127%26title%3DValidating%2520the%2520genomic%2520signature%2520of%2520pediatric%2520septic%2520shock" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/mendeley.png" alt="Mendeley logo"></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-custom pane-1">
  
      
  
  <div class="pane-content">
    <a href="http://physiolgenomics.physiology.org/content/34/1/127.full.pdf" target="_blank"><i class="icon-external-link-sign"></i> View Full Page PDF</a>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-physiolgenomics-accordion">
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-physiolgenomics_accordion">
  <div class="panel-panel panel-col">
    <div><div role="tablist" class="ui-accordion ui-widget ui-helper-reset" id="highwire_article_accordion_container"><h3 tabindex="0" aria-selected="true" aria-controls="ui-accordion-highwire_article_accordion_container-panel-0" id="ui-accordion-highwire_article_accordion_container-header-0" role="tab" class="ui-accordion-header ui-helper-reset ui-state-default ui-accordion-header-active ui-state-active ui-corner-top ui-accordion-icons"><span class="ui-accordion-header-icon ui-icon ui-icon-triangle-1-s"></span><i class="icon-list"></i>More in this TOC Section</h3><div aria-hidden="false" aria-expanded="true" role="tabpanel" aria-labelledby="ui-accordion-highwire_article_accordion_container-header-0" id="ui-accordion-highwire_article_accordion_container-panel-0" style="display: block;" class="ui-accordion-content ui-helper-reset ui-widget-content ui-corner-bottom ui-accordion-content-active"><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="23760" id="node23760" data-pisa="physiolgenomics;physiolgenomics.00086.2015v1" data-pisa-master="physiolgenomics;physiolgenomics.00086.2015" data-apath="/physiolgenomics/early/2015/11/11/physiolgenomics.00086.2015.atom"><cite class="highwire-cite highwire-citation-jcore-title-only">
  <div class="highwire-cite-title"><a href="http://physiolgenomics.physiology.org/content/early/2015/11/11/physiolgenomics.00086.2015" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><div xmlns="http://www.w3.org/1999/xhtml">Peripheral vascular reactivity and serum <em>BDNF</em> responses to aerobic training are impaired by the <em>BDNF</em> Val<em>66</em>Met polymorphism</div></span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="23758" id="node23758" data-pisa="physiolgenomics;physiolgenomics.00078.2015v1" data-pisa-master="physiolgenomics;physiolgenomics.00078.2015" data-apath="/physiolgenomics/early/2015/11/11/physiolgenomics.00078.2015.atom"><cite class="highwire-cite highwire-citation-jcore-title-only">
  <div class="highwire-cite-title"><a href="http://physiolgenomics.physiology.org/content/early/2015/11/11/physiolgenomics.00078.2015" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><div xmlns="http://www.w3.org/1999/xhtml">AMPK-agonist AICAR delays the initial decline in lifetime-apex VO<sub>2peak</sub> while voluntary wheel running fails to delay its initial decline in female rats</div></span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="23759" id="node23759" data-pisa="physiolgenomics;physiolgenomics.00072.2015v1" data-pisa-master="physiolgenomics;physiolgenomics.00072.2015" data-apath="/physiolgenomics/early/2015/11/11/physiolgenomics.00072.2015.atom"><cite class="highwire-cite highwire-citation-jcore-title-only">
  <div class="highwire-cite-title"><a href="http://physiolgenomics.physiology.org/content/early/2015/11/11/physiolgenomics.00072.2015" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">PATTERNS OF GENE EXPRESSION AMONG MURINE MODELS OF HEMORRHAGIC SHOCK/TRAUMA AND SEPSIS</span></a></div>
</cite>
</div></li></ul></div><div class="highwire-list-footer"><a href="http://physiolgenomics.physiology.org/content/by/section/Research%20Articles" class="" data-icon-position="" data-hide-link-title="0">Show more Research Articles</a></div></div><h3 tabindex="-1" aria-selected="false" aria-controls="ui-accordion-highwire_article_accordion_container-panel-1" id="ui-accordion-highwire_article_accordion_container-header-1" role="tab" class="ui-accordion-header ui-helper-reset ui-state-default ui-corner-all ui-accordion-icons"><span class="ui-accordion-header-icon ui-icon ui-icon-triangle-1-e"></span><i class="icon-file"></i> Related Articles</h3><div aria-hidden="true" aria-expanded="false" role="tabpanel" aria-labelledby="ui-accordion-highwire_article_accordion_container-header-1" id="ui-accordion-highwire_article_accordion_container-panel-1" style="display: none;" class="ui-accordion-content ui-helper-reset ui-widget-content ui-corner-bottom"><div class="highwire-list-footer"><div class="highwire-related-articles-footer"><ul class="links inline"><li class="related-isi first"><a href="http://physiolgenomics.physiology.org/lookup/external-ref?access_num=physiolgenomics%3B34/1/127&amp;link_type=ISI_CITING&amp;external_url=http%3A//gateway.webofknowledge.com/gateway/Gateway.cgi%3FGWVersion%3D2%26SrcApp%3DPARTNER_APP%26SrcAuth%3DLinksAMR%26KeyUT%3DWOS%3A000256819400014%26DestLinkType%3DRelatedRecords%26DestApp%3DALL_WOS%26UsrCustomerID%3D269fc60adb004b0b719031a97aedf5e9" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Web of Science</a></li><li class="related-pubmed"><a href="http://physiolgenomics.physiology.org/lookup/external-ref?link_type=MED_NBRS&amp;access_num=18460642" target="_blank" class="" data-icon-position="" data-hide-link-title="0">PubMed</a></li><li class="related-google-scholar last"><a href="http://physiolgenomics.physiology.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=related&amp;author[0]=Natalie%2BCvijanovich&amp;author[1]=Thomas%2BP.%2BShanley&amp;author[2]=Richard%2BLin&amp;author[3]=Geoffrey%2BL.%2BAllen&amp;author[4]=Neal%2BJ.%2BThomas&amp;author[5]=Paul%2BChecchia&amp;author[6]=Nick%2BAnas&amp;author[7]=Robert%2BJ.%2BFreishtat&amp;author[8]=Marie%2BMonaco&amp;author[9]=Kelli%2BOdoms&amp;author[10]=Bhuvaneswari%2BSakthivel&amp;author[11]=Hector%2BR.%2BWong&amp;author[12]=%2B&amp;title=Validating%2Bthe%2Bgenomic%2Bsignature%2Bof%2Bpediatric%2Bseptic%2Bshock&amp;publication_year=2008&amp;path=content/34/1/127" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Google Scholar</a></li></ul></div></div></div><h3 tabindex="-1" aria-selected="false" aria-controls="ui-accordion-highwire_article_accordion_container-panel-2" id="ui-accordion-highwire_article_accordion_container-header-2" role="tab" class="ui-accordion-header ui-helper-reset ui-state-default ui-corner-all ui-accordion-icons"><span class="ui-accordion-header-icon ui-icon ui-icon-triangle-1-e"></span><i class="icon-retweet"></i> Cited By...</h3><div aria-hidden="true" aria-expanded="false" role="tabpanel" aria-labelledby="ui-accordion-highwire_article_accordion_container-header-2" id="ui-accordion-highwire_article_accordion_container-panel-2" style="display: none;" class="ui-accordion-content ui-helper-reset ui-widget-content ui-corner-bottom"><div class="highwire-list highwire-cited-by"><ul><li class="first odd"><a href="http://stm.sciencemag.org/cgi/content/full/7/287/287ra71" class="" data-icon-position="" data-hide-link-title="0"><div class="highwire-cite-title">A comprehensive time-course-based multicohort analysis of sepsis and sterile inflammation reveals a robust diagnostic gene set</div></a><div class="variant-links"><ul class="links inline"><li class="variant-abstract first"><a href="http://stm.sciencemag.org/cgi/content/abstract/7/287/287ra71" class="" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="variant-fulltext"><a href="http://stm.sciencemag.org/cgi/content/full/7/287/287ra71" class="" data-icon-position="" data-hide-link-title="0">Fulltext</a></li><li class="variant-pdf last"><a href="http://stm.sciencemag.org/cgi/reprint/7/287/287ra71" class="" data-icon-position="" data-hide-link-title="0">PDF</a></li></ul></div></li><li class="even"><a href="http://jes.ecsdl.org/cgi/content/full/161/2/B3160" class="" data-icon-position="" data-hide-link-title="0"><div class="highwire-cite-title">Improving
 Reproducibility of Lab-on-a-Chip Sensor with Bismuth Working Electrode 
for Determining Zn in Serum by Anodic Stripping Voltammetry</div></a><div class="variant-links"><ul class="links inline"><li class="variant-abstract first"><a href="http://jes.ecsdl.org/cgi/content/abstract/161/2/B3160" class="" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="variant-fulltext"><a href="http://jes.ecsdl.org/cgi/content/full/161/2/B3160" class="" data-icon-position="" data-hide-link-title="0">Fulltext</a></li><li class="variant-pdf last"><a href="http://jes.ecsdl.org/cgi/reprint/161/2/B3160" class="" data-icon-position="" data-hide-link-title="0">PDF</a></li></ul></div></li><li class="odd"><a href="http://database.oxfordjournals.org/cgi/content/full/2013/0/bat010" class="" data-icon-position="" data-hide-link-title="0"><div class="highwire-cite-title">Uncovering hidden duplicated content in public transcriptomics data</div></a><div class="variant-links"><ul class="links inline"><li class="variant-abstract first"><a href="http://database.oxfordjournals.org/cgi/content/abstract/2013/0/bat010" class="" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="variant-fulltext"><a href="http://database.oxfordjournals.org/cgi/content/full/2013/0/bat010" class="" data-icon-position="" data-hide-link-title="0">Fulltext</a></li><li class="variant-pdf last"><a href="http://database.oxfordjournals.org/cgi/reprint/2013/0/bat010" class="" data-icon-position="" data-hide-link-title="0">PDF</a></li></ul></div></li><li class="even"><a href="http://bloodjournal.hematologylibrary.org/cgi/content/full/117/15/e151" class="" data-icon-position="" data-hide-link-title="0"><div class="highwire-cite-title">Gene expression profiling of peripheral blood mononuclear cells from children with active hemophagocytic lymphohistiocytosis</div></a><div class="variant-links"><ul class="links inline"><li class="variant-abstract first"><a href="http://bloodjournal.hematologylibrary.org/cgi/content/abstract/117/15/e151" class="" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="variant-fulltext"><a href="http://bloodjournal.hematologylibrary.org/cgi/content/full/117/15/e151" class="" data-icon-position="" data-hide-link-title="0">Fulltext</a></li><li class="variant-pdf last"><a href="http://bloodjournal.hematologylibrary.org/cgi/reprint/117/15/e151" class="" data-icon-position="" data-hide-link-title="0">PDF</a></li></ul></div></li><li class="last odd"><a href="http://pediatrics.aappublications.org/cgi/content/full/125/6/1248" class="" data-icon-position="" data-hide-link-title="0"><div class="highwire-cite-title">Mechanisms and Regulation of the Gene-Expression Response to Sepsis</div></a><div class="variant-links"><ul class="links inline"><li class="variant-abstract first"><a href="http://pediatrics.aappublications.org/cgi/content/abstract/125/6/1248" class="" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="variant-fulltext"><a href="http://pediatrics.aappublications.org/cgi/content/full/125/6/1248" class="" data-icon-position="" data-hide-link-title="0">Fulltext</a></li><li class="variant-pdf last"><a href="http://pediatrics.aappublications.org/cgi/reprint/125/6/1248" class="" data-icon-position="" data-hide-link-title="0">PDF</a></li></ul></div></li></ul></div><div class="highwire-list-footer"><ul class="links inline"><li class="citing-isi first"><a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000256819400014&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=269fc60adb004b0b719031a97aedf5e9" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Web of Science <span class="counter">(39)</span></a></li><li class="citing-google-scholar last"><a href="http://physiolgenomics.physiology.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=citedby&amp;path=content/34/1/127" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Google Scholar</a></li></ul></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-service-links pane-style-alt-content">
  
      
  
  <div class="pane-content">
    <div class="service-links"><div class="item-list"><ul><li class="first"><iframe data-url="http://physiolgenomics.physiology.org/content/34/1/127" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/tweet_button.html" title="Twitter Tweet Button" style="position: static; visibility: visible; width: 60px; height: 20px;" class="twitter-share-button twitter-share-button-rendered twitter-tweet-button" allowtransparency="true" scrolling="no" id="twitter-widget-0" frameborder="0"></iframe></li><li><iframe class="service-links-facebook-like" style="border: medium none; overflow: hidden; width: 100px; height: 21px;" allowtransparency="true" scrolling="no" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/like.html" frameborder="0"></iframe></li><li><div id="___plusone_0" style="text-indent: 0px; margin: 0px; padding: 0px; background: none repeat scroll 0% 0% transparent; border-style: none; float: none; line-height: normal; font-size: 1px; vertical-align: baseline; display: inline-block; width: 106px; height: 24px;"><iframe title="+1" data-gapiattached="true" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/fastbutton.html" name="I0_1449526045846" id="I0_1449526045846" vspace="0" tabindex="0" style="position: static; top: 0px; width: 106px; margin: 0px; border-style: none; left: 0px; visibility: visible; height: 24px;" scrolling="no" marginwidth="0" marginheight="0" hspace="0" frameborder="0" width="100%"></iframe></div></li><li><a data-log-redirect-set="true" href="http://physiolgenomics.physiology.org/highwire_log/share/reddit?link=http%3A%2F%2Freddit.com%2Fsubmit%3Furl%3Dhttp%253A%2F%2Fphysiolgenomics.physiology.org%2Fcontent%2F34%2F1%2F127%26title%3DValidating%2520the%2520genomic%2520signature%2520of%2520pediatric%2520septic%2520shock" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/reddit.png" alt="Reddit logo"> Reddit</a></li><li><a data-log-redirect-set="true" href="http://physiolgenomics.physiology.org/highwire_log/share/citeulike?link=http%3A%2F%2Fwww.citeulike.org%2Fposturl%3Furl%3Dhttp%253A%2F%2Fphysiolgenomics.physiology.org%2Fcontent%2F34%2F1%2F127%26title%3DValidating%2520the%2520genomic%2520signature%2520of%2520pediatric%2520septic%2520shock" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/citeyoulike.png" alt="CiteULike logo"> CiteULike</a></li><li><a data-log-redirect-set="true" href="http://physiolgenomics.physiology.org/highwire_log/share/mendeley?link=http%3A%2F%2Fwww.mendeley.com%2Fimport%2F%3Furl%3Dhttp%253A%2F%2Fphysiolgenomics.physiology.org%2Fcontent%2F34%2F1%2F127%26title%3DValidating%2520the%2520genomic%2520signature%2520of%2520pediatric%2520septic%2520shock" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/mendeley.png" alt="Mendeley logo"> Mendeley</a></li><li class="last"><a data-log-redirect-set="true" href="http://physiolgenomics.physiology.org/highwire_log/share/stumbleupon?link=http%3A%2F%2Fwww.stumbleupon.com%2Fsubmit%3Furl%3Dhttp%253A%2F%2Fphysiolgenomics.physiology.org%2Fcontent%2F34%2F1%2F127%26title%3DValidating%2520the%2520genomic%2520signature%2520of%2520pediatric%2520septic%2520shock" id="stumbleupon" title="Thumb this up at StumbleUpon" class="service-links-stumbleupon" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/stumbleit.png" alt="StumbleUpon logo"> StumbleUpon</a></li></ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-ajax-tab-tabs panels-ajax-tabs-once-processed">
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first active"><a style="cursor: pointer;" href="http://physiolgenomics.physiology.org/content/34/1/127" class="panels-ajax-tab-tab panels-ajax-tabs-once-processed panels-ajax-tabs-first-loaded" data-panel-name="physiolgenomics_call_for_papers" data-target-id="jnl_physiolgenomics_most_read_n_most_cited" data-entity-context="none" data-trigger="" data-url-enabled="1">Call For Papers</a><a href="http://physiolgenomics.physiology.org/panels_ajax_tab/physiolgenomics_call_for_papers/none/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a style="cursor: pointer;" href="http://physiolgenomics.physiology.org/content/34/1/127.most-cited" class="panels-ajax-tab-tab panels-ajax-tabs-once-processed" data-panel-name="physiolgenomics_most_cited" data-target-id="jnl_physiolgenomics_most_read_n_most_cited" data-entity-context="none" data-trigger="most-cited" data-url-enabled="1">Most Cited</a><a href="http://physiolgenomics.physiology.org/panels_ajax_tab/physiolgenomics_most_cited/none/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a style="cursor: pointer;" href="http://physiolgenomics.physiology.org/content/34/1/127.most-read" class="panels-ajax-tab-tab panels-ajax-tabs-once-processed" data-panel-name="physiolgenomics_most_read" data-target-id="jnl_physiolgenomics_most_read_n_most_cited" data-entity-context="none" data-trigger="most-read" data-url-enabled="1">Most Read</a><a href="http://physiolgenomics.physiology.org/panels_ajax_tab/physiolgenomics_most_read/none/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-ajax-tab-container">
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="physiolgenomics_call_for_papers" id="panels-ajax-tab-container-jnl_physiolgenomics_most_read_n_most_cited" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style="display:none"><img class="loading" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/loading.gif" alt="Loading" title="Loading"></div><div class="panels-ajax-tab-wrap-physiolgenomics_call_for_papers"><div class="panel-display panel-1col clearfix">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet">
  
      
  
  <div class="pane-content">
    <div class="snippet physiolgenomics-call-for-papers-snippet" id="physiolgenomics-call-for-papers-snippet">
  
      
  <div class="snippet-content">
    <div class="view-content">
	<div class="views-row">
		<a href="http://www.the-aps.org/mm/Publications/Journals/Physiol-Genomics/Special-Calls/Gut-Microbiota-in-Health-and-Disease.html" target="_blank">Gut Microbiota in Health and Disease</a></div>
	<div class="views-row">
		<a href="http://www.the-aps.org/mm/Publications/Journals/Physiol-Genomics/Special-Calls/Systems-Biology-and-Polygenic-Traits.html" target="_blank">Systems Biology and Polygenic Traits</a></div>
	<div class="views-row views-row-last">
		&nbsp;</div>
</div>
<p>&nbsp;</p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-ajax-tab-tabs panels-ajax-tabs-once-processed">
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first last active"><a style="cursor: pointer;" href="http://physiolgenomics.physiology.org/content/34/1/127" class="panels-ajax-tab-tab panels-ajax-tabs-once-processed panels-ajax-tabs-first-loaded" data-panel-name="featured_articles" data-target-id="aps_article_page_feat_articles" data-entity-context="none" data-trigger="" data-url-enabled="1">Featured Articles</a><a href="http://physiolgenomics.physiology.org/panels_ajax_tab/featured_articles/none/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-ajax-tab-container">
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="featured_articles" id="panels-ajax-tab-container-aps_article_page_feat_articles" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style="display:none"><img class="loading" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/loading.gif" alt="Loading" title="Loading"></div><div class="panels-ajax-tab-wrap-featured_articles"><div class="panel-display panel-1col clearfix">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-views pane-nodequeue-1">
  
      
  
  <div class="pane-content">
    <div class="view view-nodequeue-1 view-id-nodequeue_1 view-display-id-block view-dom-id-7f5ad7dbd16ef045cfb6e86935cad6ad">
        
  
  
      <div class="view-content">
      <div class="item-list">    <ul>          <li class="views-row views-row-1 views-row-odd views-row-first">  
  <div class="views-field views-field-title">        <span class="field-content"><a href="http://physiolgenomics.physiology.org/content/47/9/365" class="" data-icon-position="" data-hide-link-title="0">Genome-wide association studies and contribution to cardiovascular physiology</a></span>  </div></li>
          <li class="views-row views-row-2 views-row-even">  
  <div class="views-field views-field-title">        <span class="field-content"><a href="http://physiolgenomics.physiology.org/content/47/9/386" class="" data-icon-position="" data-hide-link-title="0">Brain-stem microRNAs implicated in hypertension</a></span>  </div></li>
          <li class="views-row views-row-3 views-row-odd">  
  <div class="views-field views-field-title">        <span class="field-content"><a href="http://physiolgenomics.physiology.org/content/47/9/388" class="" data-icon-position="" data-hide-link-title="0">MicroRNA network changes in the brain stem underlie the development of hypertension</a></span>  </div></li>
          <li class="views-row views-row-4 views-row-even views-row-last">  
  <div class="views-field views-field-title">        <span class="field-content"><a href="http://physiolgenomics.physiology.org/content/early/2015/07/16/physiolgenomics.00063.2015" class="" data-icon-position="" data-hide-link-title="0">Historical perspective: Gut dysbiosis and hypertension</a></span>  </div></li>
      </ul></div>    </div>
  
  
  
  
  
  
</div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-google-admanager-ef2ca450fab91a94bdc0546a26e072d2 pane-google-admanager">
  
      
  
  <div class="pane-content">
    <div id="gam-holder-physiolgenomics_tower_right_160x600" class="gam-holder"><script type="text/javascript">GA_googleFillSlot("physiolgenomics_tower_right_160x600");</script><script src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/ads_002"></script><div id="google_ads_div_physiolgenomics_tower_right_160x600_ad_wrapper">
<div id="google_ads_div_physiolgenomics_tower_right_160x600_ad_container" style="display:inline-block;">
<ins style="width: 160px; height: 600px; display: inline-table; position: relative; border: 0px none;"><ins style="width: 160px; height: 600px; display: block; position: relative; border: 0px none;"><iframe style="border: 0px none; position: absolute; top: 0px; left: 0px;" marginheight="0" marginwidth="0" scrolling="no" name="google_ads_iframe_physiolgenomics_tower_right_160x600" id="google_ads_iframe_physiolgenomics_tower_right_160x600" frameborder="0" height="600" width="160"></iframe></ins></ins></div>
</div>
<script>GA_googleCreateDomIframe("google_ads_div_physiolgenomics_tower_right_160x600_ad_container" ,"physiolgenomics_tower_right_160x600");</script></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-2 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <section class="block block-menu block-menu-navigate block-menu-menu-navigate odd" id="block-menu-menu-navigate">
  <div class="block-inner clearfix">
              <h2 class="block-title">Navigate</h2>
            
    <div class="content clearfix">
      <nav role="menubar" class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="http://physiolgenomics.physiology.org/content/current" title="" class="" data-icon-position="" data-hide-link-title="0">Current Issue</a></li>
<li class="leaf" role="menuitem"><a href="http://physiolgenomics.physiology.org/content/early/recent" title="Route the user to a particular piece of content based on cpath or cpathalias" class="" data-icon-position="" data-hide-link-title="0">Articles in Press</a></li>
<li class="leaf" role="menuitem"><a href="http://physiolgenomics.physiology.org/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archives</a></li>
<li class="leaf" role="menuitem"><a href="http://physiolgenomics.physiology.org/content/feedback" title="" class="" data-icon-position="" data-hide-link-title="0">Feedback</a></li>
<li class="leaf" role="menuitem"><a href="http://pg.msubmit.net/cgi-bin/main.plex" title="" class="" data-icon-position="" data-hide-link-title="0">Submit</a></li>
<li class="leaf" role="menuitem"><a href="http://www.the-aps.org/mm/Publications/Purchasing" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Subscribe</a></li>
<li class="last leaf" role="menuitem"><a href="http://physiolgenomics.physiology.org/alerts" title="" class="" data-icon-position="" data-hide-link-title="0">Personal Alerts</a></li>
</ul></nav>    </div>
  </div>
</section>  </div>
</div><div class="grid-6 suffix-2 prefix-1 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <section class="block block-menu block-menu-more-information block-menu-menu-more-information odd" id="block-menu-menu-more-information">
  <div class="block-inner clearfix">
              <h2 class="block-title">More Information</h2>
            
    <div class="content clearfix">
      <nav role="menubar" class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="http://physiolgenomics.physiology.org/about" class="" data-icon-position="" data-hide-link-title="0">About this Journal</a></li>
<li class="leaf" role="menuitem"><a href="http://www.the-aps.org/mm/Publications/Info-For-Authors" class="" data-icon-position="" data-hide-link-title="0">Information for Authors</a></li>
<li class="leaf" role="menuitem"><a href="http://pg.msubmit.net/cgi-bin/main.plex" class="" data-icon-position="" data-hide-link-title="0">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="http://www.the-aps.org/mm/hp/Audiences/Public-Press" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Press</a></li>
<li class="leaf" role="menuitem"><a href="http://physiolgenomics.physiology.org/site/misc/addir" class="" data-icon-position="" data-hide-link-title="0">Advertising</a></li>
<li class="leaf" role="menuitem"><a href="http://www.the-aps.org/authorchoice/" class="" data-icon-position="" data-hide-link-title="0">AuthorChoice</a></li>
<li class="leaf" role="menuitem"><a href="http://www.the-aps.org/specialcalls/pg" class="" data-icon-position="" data-hide-link-title="0">Calls for Papers</a></li>
<li class="leaf" role="menuitem"><a href="http://www.the-aps.org/publications/journals/apsethic.htm" class="" data-icon-position="" data-hide-link-title="0">Ethical Policies</a></li>
<li class="leaf" role="menuitem"><a href="http://www.the-aps.org/publications/journals/public_access_policy.htm" class="" data-icon-position="" data-hide-link-title="0">PubMed Central Policy</a></li>
<li class="leaf" role="menuitem"><a href="http://www.the-aps.org/publications/authorinfo/copyright.htm" class="" data-icon-position="" data-hide-link-title="0">Reprints and Permissions</a></li>
<li class="leaf" role="menuitem"><a href="http://physiolgenomics.physiology.org/help/subscriptions/institutional-admin" class="" data-icon-position="" data-hide-link-title="0">Institutional Administrators</a></li>
<li class="leaf" role="menuitem"><a href="http://www.the-aps.org/mm/Publications/Publications-News" data-hide-link-title="0" target="_blank" class="" data-icon-position="">APS Publications News</a></li>
<li class="last leaf" role="menuitem"><a href="http://www.twitter.com/APSPublications" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Follow APS Publications on Twitter</a></li>
</ul></nav>    </div>
  </div>
</section>  </div>
</div><div class="grid-10 prefix-2 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <section class="block block-menu-block block-1 block-menu-block-1 odd" id="block-menu-block-1">
  <div class="block-inner clearfix">
              <h2 class="block-title">American Physiological Society Journals</h2>
            
    <div class="content clearfix">
      <div class="menu-block-wrapper menu-block-1 menu-name-menu-american-physiological-soci parent-mlid-634 menu-level-1">
  <nav role="menubar" class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-635" role="menuitem"><a href="http://ajpcell.physiology.org/" title="" class="" data-icon-position="" data-hide-link-title="0">Cell Physiology</a></li>
<li class="leaf menu-mlid-636" role="menuitem"><a href="http://advan.physiology.org/" class="" data-icon-position="" data-hide-link-title="0">Advances in Physiology Education</a></li>
<li class="leaf menu-mlid-637" role="menuitem"><a href="http://www.comprehensivephysiology.com/" class="" data-icon-position="" data-hide-link-title="0">Comprehensive Physiology</a></li>
<li class="leaf menu-mlid-638" role="menuitem"><a href="http://ajpendo.physiology.org/" class="" data-icon-position="" data-hide-link-title="0">Endocrinology and Metabolism</a></li>
<li class="leaf menu-mlid-639" role="menuitem"><a href="http://ajpgi.physiology.org/" class="" data-icon-position="" data-hide-link-title="0">Gastrointestinal and Liver Physiology</a></li>
<li class="leaf menu-mlid-640" role="menuitem"><a href="http://ajpheart.physiology.org/" class="" data-icon-position="" data-hide-link-title="0">Heart and Circulatory Physiology</a></li>
<li class="leaf menu-mlid-641" role="menuitem"><a href="http://jap.physiology.org/" class="" data-icon-position="" data-hide-link-title="0">Journal of Applied Physiology</a></li>
<li class="leaf menu-mlid-642" role="menuitem"><a href="http://jn.physiology.org/" class="" data-icon-position="" data-hide-link-title="0">Journal of Neurophysiology</a></li>
<li class="leaf menu-mlid-643" role="menuitem"><a href="http://ajplung.physiology.org/" class="" data-icon-position="" data-hide-link-title="0">Lung Cellular and Molecular Physiology</a></li>
<li class="leaf menu-mlid-644" role="menuitem"><a href="http://physiolgenomics.physiology.org/" class="" data-icon-position="" data-hide-link-title="0">Physiological Genomics</a></li>
<li class="leaf menu-mlid-645" role="menuitem"><a href="http://physrev.physiology.org/" class="" data-icon-position="" data-hide-link-title="0">Physiological Reviews</a></li>
<li class="leaf menu-mlid-646" role="menuitem"><a href="http://physiologyonline.physiology.org/" class="" data-icon-position="" data-hide-link-title="0">Physiology</a></li>
<li class="leaf menu-mlid-647" role="menuitem"><a href="http://ajpregu.physiology.org/" class="" data-icon-position="" data-hide-link-title="0">Regulatory, Integrative and Comparative Physiology</a></li>
<li class="leaf menu-mlid-648" role="menuitem"><a href="http://ajprenal.physiology.org/" class="" data-icon-position="" data-hide-link-title="0">Renal Physiology</a></li>
<li class="leaf menu-mlid-649" role="menuitem"><a href="http://www.physiologicalreports.org/" class="" data-icon-position="" data-hide-link-title="0">Physiological Reports</a></li>
<li class="leaf menu-mlid-724" role="menuitem"><a href="http://ajplegacy.physiology.org/" title="" class="" data-icon-position="" data-hide-link-title="0">Legacy Content</a></li>
<li class="leaf menu-mlid-725" role="menuitem"><a href="http://apsselect.physiology.org/" title="" class="" data-icon-position="" data-hide-link-title="0">APS Select</a></li>
<li class="last leaf menu-mlid-737" role="menuitem"><a href="http://www.physiology.org/" title="" class="" data-icon-position="" data-hide-link-title="0">www.physiology.org</a></li>
</ul></nav></div>
    </div>
  </div>
</section>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-first" id="region-footer-first">
  <div class="region-inner region-footer-first-inner">
    <div class="block block-block block-2 block-block-2 odd block-without-title" id="block-block-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <p style="font-size: 12.222222328186035px;">Copyright Â© 2014 The American Physiological Society | Print ISSN: <span class="issn">1094-8341</span> | Online ISSN: <span class="issn">1531-2267</span></p>
<p style="font-size: 12.222222328186035px;"><a href="http://highwire.org/" target="_blank"><img alt="" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/HW_white_trans_sm.png" style="width: 133px; height: 23px;"></a></p>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>  <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/highwire_openurl.js"></script>
<script type="text/javascript" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/highwire_tables.js"></script>
<script gapi_processed="true" type="text/javascript" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/highwire_figures.js"></script>


<div style="display: none;" id="cboxOverlay"></div><div style="display: none;" tabindex="-1" role="dialog" class="" id="colorbox"><div id="cboxWrapper"><div><div style="float: left;" id="cboxTopLeft"></div><div style="float: left;" id="cboxTopCenter"></div><div style="float: left;" id="cboxTopRight"></div></div><div style="clear: left;"><div style="float: left;" id="cboxMiddleLeft"></div><div style="float: left;" id="cboxContent"><div style="float: left;" id="cboxTitle"></div><div style="float: left;" id="cboxCurrent"></div><button id="cboxPrevious" type="button"></button><button id="cboxNext" type="button"></button><button id="cboxSlideshow"></button><div style="float: left;" id="cboxLoadingOverlay"></div><div style="float: left;" id="cboxLoadingGraphic"></div></div><div style="float: left;" id="cboxMiddleRight"></div></div><div style="clear: left;"><div style="float: left;" id="cboxBottomLeft"></div><div style="float: left;" id="cboxBottomCenter"></div><div style="float: left;" id="cboxBottomRight"></div></div></div><div style="position: absolute; width: 9999px; visibility: hidden; display: none; max-width: none;"></div></div><div style="position: absolute; display: none;" id="cluetip-waitimage"></div><div style="position: absolute; display: none;" id="cluetip"><div style="position: relative; z-index: 97;" class="cluetip-outer"><h3 class="cluetip-title ui-widget-header ui-cluetip-header"></h3><div class="cluetip-inner ui-widget-content ui-cluetip-content"></div></div><div class="cluetip-extra"></div><div class="cluetip-arrows ui-state-default"></div></div><div aria-labelledby="ui-id-2" aria-describedby="ui-id-1" role="dialog" tabindex="-1" style="display: none;" class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span class="ui-dialog-title" id="ui-id-2">&nbsp;</span><button title="close" aria-disabled="false" role="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">close</span></button></div><div id="ui-id-1" class="minipanel-dialog-link-mini ui-dialog-content ui-widget-content" style=""><div class="empty_placeholder ui-helper-hidden-accessible"><input type="text"></div>
<div class="panel-display panel-1col clearfix" id="mini-panel-physiolgenomics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward">
  
      
  
  <div class="pane-content">
    <form action="/content/34/1/127.long" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for your interest in spreading the word on Physiological Genomics.</p><p>NOTE:
 We only request your email address so that the person you are 
recommending the page to knows that you wanted them to see it, and that 
it is not junk mail. We do not capture any email address.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input id="edit-email" name="email" size="58" maxlength="256" class="form-text required" type="text">
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input id="edit-name" name="name" size="58" maxlength="256" class="form-text required" type="text">
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable textarea-processed resizable-textarea"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea><div class="grippie"></div></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="http://physiolgenomics.physiology.org/content/34/1/127" class="active" data-icon-position="" data-hide-link-title="0">Validating the genomic signature of pediatric septic shock</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has sent you a message from Physiological Genomics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would like to see the Physiological Genomics web site.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable textarea-processed resizable-textarea"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea><div class="grippie"></div></div>
</div>
<input name="path" value="node/4156" type="hidden">
<input name="path_cid" value="" type="hidden">
<input name="forward_footer" value="" type="hidden">
<input name="form_build_id" value="form-LtPYAHmwnksxlpT-tZSawAk3d21CweuFgTNVMGl-O5o" type="hidden">
<input name="form_id" value="forward_form" type="hidden">
<div class="form-actions form-wrapper" id="edit-actions"><input id="edit-submit" name="op" value="Send Message" class="form-submit" type="submit"></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div><div aria-labelledby="ui-id-4" aria-describedby="ui-id-3" role="dialog" tabindex="-1" style="display: none;" class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span class="ui-dialog-title" id="ui-id-4">&nbsp;</span><button title="close" aria-disabled="false" role="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">close</span></button></div><div id="ui-id-3" class="minipanel-dialog-link-mini ui-dialog-content ui-widget-content" style=""><div class="empty_placeholder ui-helper-hidden-accessible"><input type="text"></div>
<div class="panel-display panel-1col clearfix" id="mini-panel-physiolgenomics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export">
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="4156" id="node4156" data-pisa="physiolgenomics;34/1/127" data-pisa-master="physiolgenomics;00025.2008" data-apath="/physiolgenomics/34/1/127.atom"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-jcore-standard clearfix">
    
  
  
      <div class="highwire-cite-title">Validating the genomic signature of pediatric septic shock</div>  
    	<div class="highwire-cite-authors"><span class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Natalie</span> <span class="nlm-surname">Cvijanovich</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Thomas P.</span> <span class="nlm-surname">Shanley</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Richard</span> <span class="nlm-surname">Lin</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Geoffrey L.</span> <span class="nlm-surname">Allen</span></span>, <span class="highwire-citation-author" data-delta="4"><span class="nlm-given-names">Neal J.</span> <span class="nlm-surname">Thomas</span></span>, <span class="highwire-citation-author" data-delta="5"><span class="nlm-given-names">Paul</span> <span class="nlm-surname">Checchia</span></span>, <span class="highwire-citation-author" data-delta="6"><span class="nlm-given-names">Nick</span> <span class="nlm-surname">Anas</span></span>, <span class="highwire-citation-author" data-delta="7"><span class="nlm-given-names">Robert J.</span> <span class="nlm-surname">Freishtat</span></span>, <span class="highwire-citation-author" data-delta="8"><span class="nlm-given-names">Marie</span> <span class="nlm-surname">Monaco</span></span>, <span class="highwire-citation-author" data-delta="9"><span class="nlm-given-names">Kelli</span> <span class="nlm-surname">Odoms</span></span>, <span class="highwire-citation-author" data-delta="10"><span class="nlm-given-names">Bhuvaneswari</span> <span class="nlm-surname">Sakthivel</span></span>, <span class="highwire-citation-author" data-delta="11"><span class="nlm-given-names">Hector R.</span> <span class="nlm-surname">Wong</span></span></span></div>
  
    	<div class="highwire-cite-metadata"><span class="highwire-cite-metadata-journal highwire-cite-metadata">Physiological Genomics </span><span class="highwire-cite-metadata-date highwire-cite-metadata">Jun 2008, </span><span class="highwire-cite-metadata-volume highwire-cite-metadata">34 </span><span class="highwire-cite-metadata-issue highwire-cite-metadata">(1) </span><span class="highwire-cite-metadata-pages highwire-cite-metadata">127-134; </span><span class="highwire-cite-metadata-doi highwire-cite-metadata"><span class="label">DOI:</span> 10.1152/physiolgenomics.00025.2008 </span></div>
  
  
  
</div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=127&amp;rft.epage=134&amp;rft.atitle=Validating%20the%20genomic%20signature%20of%20pediatric%20septic%20shock&amp;rft.volume=34&amp;rft.issue=1&amp;rft.date=2008-06-12%2019%3A51%3A28&amp;rft.stitle&amp;rft.jtitle=Physiological%20Genomics&amp;rft.au=Cvijanovich%2C+Natalie&amp;rft.au=Shanley%2C+Thomas+P.&amp;rft.au=Lin%2C+Richard&amp;rft.au=Allen%2C+Geoffrey+L.&amp;rft.au=Thomas%2C+Neal+J.&amp;rft.au=Checchia%2C+Paul&amp;rft.au=Anas%2C+Nick&amp;rft.au=Freishtat%2C+Robert+J.&amp;rft.au=Monaco%2C+Marie&amp;rft.au=Odoms%2C+Kelli&amp;rft.au=Sakthivel%2C+Bhuvaneswari&amp;rft.au=Wong%2C+Hector+R."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="http://physiolgenomics.physiology.org/highwire/citation/4156/bibtex" class="" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="http://physiolgenomics.physiology.org/highwire/citation/4156/bookends" class="" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="http://physiolgenomics.physiology.org/highwire/citation/4156/easybib" class="" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="http://physiolgenomics.physiology.org/highwire/citation/4156/tagged" class="" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="http://physiolgenomics.physiology.org/highwire/citation/4156/endnote8" class="" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="http://physiolgenomics.physiology.org/highwire/citation/4156/medlars" class="" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="http://physiolgenomics.physiology.org/highwire/citation/4156/mendeley" class="" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="http://physiolgenomics.physiology.org/highwire/citation/4156/papers" class="" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="http://physiolgenomics.physiology.org/highwire/citation/4156/refworks" class="" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="http://physiolgenomics.physiology.org/highwire/citation/4156/reference-manager" class="" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="http://physiolgenomics.physiology.org/highwire/citation/4156/ris" class="" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="http://physiolgenomics.physiology.org/highwire/citation/4156/zotero" class="" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div><iframe tabindex="-1" style="width: 1px; height: 1px; position: absolute; top: -100px;" src="9692%20TABLE1%20with%201%20sp%20less%20than%20GEO_files/postmessageRelay.html" id="oauth2relay311042228" name="oauth2relay311042228"></iframe><iframe style="display: none; width: 0px; height: 0px;" name="google_osd_static_frame" id="google_osd_static_frame_4909634061001"></iframe></body></html>